<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html id="nojs" xmlns="http://www.w3.org/1999/xhtml" xmlns:m="http://www.w3.org/1998/Math/MathML" xmlns:og="http://ogp.me/ns#" xml:lang="en-GB" lang="en-GB">

<head>

 
    
    
                                                                                                                                                                                                                                
                                                                                                                                                            
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
        <title>BMC Genomics | Full text | Yellow lupin (Lupinus luteus L.) transcriptome sequencing: molecular marker development and comparative studies</title>
<link rel="shortcut icon" type="image/x-icon" href="/images/icon/1008/favicon.ico" />
<meta http-equiv="X-UA-Compatible" content="IE=7" />


                




		
		
		
<!-- <?xml version="1.0" encoding="UTF-8"?>
<rdf:RDF xmlns:cc="http://web.resource.org/cc/" xmlns="http://purl.org/rss/1.0/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:prism="http://prismstandard.org/namespaces/1.2/basic/">
<cc:Work rdf:about="http://www.biomedcentral.com/1471-2164/13/425">
<cc:license rdf:resource="http://creativecommons.org/licenses/by/2.0/"/>
</cc:Work>
<cc:License rdf:about="http://creativecommons.org/licenses/by/2.0/">
<cc:permits rdf:resource="http://web.resource.org/cc/Reproduction"/>
<cc:permits rdf:resource="http://web.resource.org/cc/Distribution"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Notice"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Attribution"/>
<cc:permits rdf:resource="http://web.resource.org/cc/DerivativeWorks"/>
</cc:License>
<item rdf:about="http://www.biomedcentral.com/1471-2164/13/425">
<title>Yellow lupin (Lupinus luteus L.) transcriptome sequencing: molecular marker development and comparative studies</title>
<dc:title>Yellow lupin (Lupinus luteus L.) transcriptome sequencing: molecular marker development and comparative studies</dc:title>
<dc:creator>Parra-González, Lorena B</dc:creator>
<dc:creator>Aravena-Abarzúa, Gabriela A</dc:creator>
<dc:creator>Navarro-Navarro, Cristell S</dc:creator>
<dc:creator>Udall, Joshua</dc:creator>
<dc:creator>Maughan, Jeff</dc:creator>
<dc:creator>Peterson, Louis M</dc:creator>
<dc:creator>Salvo-Garrido, Haroldo E</dc:creator>
<dc:creator>Maureira-Butler, Iván J</dc:creator>
<dc:identifier>info:doi/10.1186/1471-2164-13-425</dc:identifier>
<dc:identifier>info:pmid/22920992</dc:identifier>
<dc:source>BMC Genomics 2012, 13:425</dc:source>
<dc:date>2012-08-24</dc:date>

<prism:publicationName>BMC Genomics</prism:publicationName>
<prism:publicationDate>2012-08-24</prism:publicationDate>
<prism:volume>13</prism:volume>
<prism:number>1</prism:number>
<prism:section>Research article</prism:section>
<prism:startingPage>425</prism:startingPage>
<prism:copyright>2012 Parra-González et al.; licensee BioMed Central Ltd.</prism:copyright>
</item>
</rdf:RDF>
 -->
 
 
<meta name="citation_journal_title" content="BMC Genomics" />
<meta name="citation_publisher" content="BioMed Central Ltd" />





   <meta name="citation_author" content="Lorena B Parra-González" />

              <meta name="citation_author_institution" content="
        Agriaquaculture Nutritional Genomic Center, CGNA, Genomics and Bioinformatics Unit, Km 10 Camino Cajón-Vilcún, INIA, Temuco, Chile
        " />
   
 
   <meta name="citation_author" content="Gabriela A Aravena-Abarzúa" />

              <meta name="citation_author_institution" content="
        Agriaquaculture Nutritional Genomic Center, CGNA, Genomics and Bioinformatics Unit, Km 10 Camino Cajón-Vilcún, INIA, Temuco, Chile
        " />
   
 
   <meta name="citation_author" content="Cristell S Navarro-Navarro" />

              <meta name="citation_author_institution" content="
        Agriaquaculture Nutritional Genomic Center, CGNA, Genomics and Bioinformatics Unit, Km 10 Camino Cajón-Vilcún, INIA, Temuco, Chile
        " />
   
 
   <meta name="citation_author" content="Joshua Udall" />

              <meta name="citation_author_institution" content="
        Plant and Wildlife Science Department, Brigham Young University, 150 E 1230 North, Provo, UT, 84602, USA
        " />
   
 
   <meta name="citation_author" content="Jeff Maughan" />

              <meta name="citation_author_institution" content="
        Plant and Wildlife Science Department, Brigham Young University, 150 E 1230 North, Provo, UT, 84602, USA
        " />
   
 
   <meta name="citation_author" content="Louis M Peterson" />

              <meta name="citation_author_institution" content="
        Plant and Wildlife Science Department, Brigham Young University, 150 E 1230 North, Provo, UT, 84602, USA
        " />
   
 
   <meta name="citation_author" content="Haroldo E Salvo-Garrido" />

              <meta name="citation_author_institution" content="
        Institute of Agricultural Research (INIA), P.O. Box 58-D, Temuco, Chile
        " />
              <meta name="citation_author_institution" content="
        Agriaquaculture Nutritional Genomic Center, CGNA, Genomics and Bioinformatics Unit, Km 10 Camino Cajón-Vilcún, INIA, Temuco, Chile
        " />
   
 
   <meta name="citation_author" content="Iván J Maureira-Butler" />

              <meta name="citation_author_institution" content="
        Agriaquaculture Nutritional Genomic Center, CGNA, Genomics and Bioinformatics Unit, Km 10 Camino Cajón-Vilcún, INIA, Temuco, Chile
        " />
   
  <meta name="citation_issn" content="1471-2164" />
<meta name="citation_title" content="Yellow lupin (Lupinus luteus L.) transcriptome sequencing: molecular marker development and comparative studies" />
<meta name="citation_volume" content="13" />
<meta name="citation_issue" content="1" />
<meta name="citation_date" content="2012-08-24" />
<meta name="citation_firstpage" content="425" />
<meta name="citation_doi" content="10.1186/1471-2164-13-425" />
<meta name="title" content="Yellow lupin (Lupinus luteus L.) transcriptome sequencing: molecular marker development and comparative studies" />
<meta name="description" content="Yellow lupin (Lupinus luteus L.) is a minor legume crop characterized by its high seed protein content. Although grown in several temperate countries, its orphan condition has limited the generation of genomic tools to aid breeding efforts to improve yield and nutritional quality. In this study, we report the construction of 454-expresed sequence tag (EST) libraries, carried out comparative studies between L. luteus and model legume species, developed a comprehensive set of EST-simple sequence repeat (SSR) markers, and validated their utility on diversity studies and transferability to related species." />

<meta name="dc.title" content="Yellow lupin (Lupinus luteus L.) transcriptome sequencing: molecular marker development and comparative studies" />
<meta name="dc.creator" content="Parra-González, Lorena B" />
<meta name="dc.creator" content="Aravena-Abarzúa, Gabriela A" />
<meta name="dc.creator" content="Navarro-Navarro, Cristell S" />
<meta name="dc.creator" content="Udall, Joshua" />
<meta name="dc.creator" content="Maughan, Jeff" />
<meta name="dc.creator" content="Peterson, Louis M" />
<meta name="dc.creator" content="Salvo-Garrido, Haroldo E" />
<meta name="dc.creator" content="Maureira-Butler, Iván J" />
<meta name="dc.description" content="Yellow lupin (Lupinus luteus L.) is a minor legume crop characterized by its high seed protein content. Although grown in several temperate countries, its orphan condition has limited the generation of genomic tools to aid breeding efforts to improve yield and nutritional quality. In this study, we report the construction of 454-expresed sequence tag (EST) libraries, carried out comparative studies between L. luteus and model legume species, developed a comprehensive set of EST-simple sequence repeat (SSR) markers, and validated their utility on diversity studies and transferability to related species." />
<meta name="dc.source" content="BMC Genomics 2012 13:425" />
<meta name="dc.format" content="text/html" />
<meta name="dc.publisher" content="BioMed Central Ltd" />
<meta name="dc.date" content="2012-08-24" />
<meta name="dc.type" content="Research article" />
<meta name="dc.identifier" content="10.1186/1471-2164-13-425" />
<meta name="dc.language" content="en" />
<meta name="dc.copyright" content="2012 Parra-González et al.; licensee BioMed Central Ltd." />
<meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0/" />
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com" />

<meta name="prism.issn" content="1471-2164" />
<meta name="prism.publicationName" content="BMC Genomics" />
<meta name="prism.publicationDate" content="2012-08-24" />
<meta name="prism.volume" content="13" />
<meta name="prism.number" content="1" />
<meta name="prism.section" content="Research article" />
<meta name="prism.startingPage" content="425" />
<meta name="prism.copyright" content="2012 Parra-González et al.; licensee BioMed Central Ltd." />
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com" />




<link rel="alternate" type="application/pdf" title="PDF" href="http://www.biomedcentral.com/bmcgenomics/content/pdf/1471-2164-13-425.pdf" />
<meta name="citation_pdf_url" content="http://www.biomedcentral.com/bmcgenomics/content/pdf/1471-2164-13-425.pdf" />

<meta name="citation_abstract_html_url" content="http://www.biomedcentral.com/1471-2164/13/425/abstract" />
<meta name="citation_fulltext_html_url" content="http://www.biomedcentral.com/1471-2164/13/425" />
<meta name="citation_pdf_url" content="http://www.biomedcentral.com/bmcgenomics/content/pdf/1471-2164-13-425.pdf" />
<link rel="alternate" type="application/pdf" title="PDF" href="http://www.biomedcentral.com/bmcgenomics/content/pdf/1471-2164-13-425.pdf" />

<link rel='alternate' type='text/xml' title='XML version' href='http://www.biomedcentral.com/bmcgenomics/content/download/xml/1471-2164-13-425.xml' />

<link rel="image_src" href="http://www.biomedcentral.com/content/figures/1471-2164-13-425-toc.gif" />



    <link rel="section" href="http://www.biomedcentral.com/content/download/supplementary/1471-2164-13-425-s1.pdf" title="Additional file 1" />
<link rel="section" href="http://www.biomedcentral.com/content/download/supplementary/1471-2164-13-425-s2.pdf" title="Additional file 2" />

    
    
                    
    
<script type="text/javascript">
// must run before css
document.documentElement.id = "js";
</script>






	                <link rel="stylesheet" type="text/css" href="/css/main-0.css" media="screen, print"/>
    
                <link rel="stylesheet" type="text/css" href="/css/plugins-0.css" media="screen, print"/>
    



    
			                    <link rel="stylesheet" type="text/css" href="/css/articles-0.css" media="screen, print"/>
    			


    

    <link rel="stylesheet" type="text/css" href="/bmcgenomics/css/themes/1008.css?1330424572279" media="screen, print"/>


                <link rel="stylesheet" type="text/css" href="/css/media/print-0.css" media="print"/>
    


<!--[if IE]>
                    <link rel="stylesheet" type="text/css" href="/css/hacks/ie-0.css"/>
    <![endif]-->

<!--[if IE 7]>
                    <link rel="stylesheet" type="text/css" href="/css/hacks/ie7-0.css"/>
    <![endif]-->


<script type="text/javascript">
    //configuration
    OAS_url = 'http://oas.biomedcentral.com/RealMedia/ads/';

	// control of advert's position on page [AMC]
    
    	
	
    OAS_sitepage = "biomedcentral.com/bmcgenomics/article/10.1186/1471/2164/13/425";
    	
		OAS_listpos = 'Top,x96,Bottom';
	
	
			
		OAS_query = '';
		
	

    OAS_target = '_top';
    //end of configuration
    OAS_version = 10;
    OAS_rn = '001234567890'; OAS_rns = '1234567890';
    OAS_rn = new String (Math.random()); OAS_rns = OAS_rn.substring (2, 11);
    function OAS_NORMAL(pos) {
		document.write('<a href="' + OAS_url + 'click_nx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '!' + pos + '?' + OAS_query + '" target="' + OAS_target + '">');
		document.write('<img src="' + OAS_url + 'adstream_nx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '!' + pos + '?' + OAS_query + '" border="0"/></a>');
    }
    
	function refererTerms(referer) {

	if(referer.match("q=") != null) {
		referer = referer.split("q=");
		referer = referer[1].split("&");
		referer = referer[0];
	} else {
		referer = referer;
	}

		if(referer.match("%20") != null && referer.match("%20").length >= 1) {
			referer =  referer.replace(/%20/gi,"+");
			return referer;
		} else { 
			return referer; 
		}
	}
 </script>

 
 <div id="oas-campaign" style="display:none">biomedcentral.com/bmcgenomics/article/10.1186/1471/2164/13/425</div>
 <div id="oas-positions" style="display:none">Bottom,Top</div> 


 <script type="text/javascript">
	// Set the version of JavaScript to 10 for a Browser 'Mozilla/3' on webTV 
    OAS_version = 11;
    if ((navigator.userAgent.indexOf('Mozilla/3') != -1) || (navigator.userAgent.indexOf('Mozilla/4.0 WebTV') != -1)) {
     OAS_version = 10; }
    if (OAS_version >= 11) {
    document.write('<scr' + 'ipt type="text/javascript" src="' + OAS_url + 'adstream_mjx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '?' + OAS_query + '"><\/script>'); }
</script>
	
<script type="text/javascript">
    document.write('');
	
    function OAS_AD(pos) {
        if (OAS_version >= 11)
         OAS_RICH(pos);
        else
         OAS_NORMAL(pos);
    }
 </script>




<script type="text/javascript">
    jsonLoaders = function(){
        var _loaders = [];
        return {
            register: function(config){
                config.prefix = config.prefix || '/webapi/1.0/blogs/';
                config.feedLength = config.feedLength || 3;

                _loaders.push(config);
            },
            list: function(){return _loaders}
        };
    }();

    function totext() {
	    document.write('<a href="javascript:returnToText();">Return to text</a>');
    }

    function retrieve_doi(){
        var metas = document.getElementsByTagName('meta');
        var i;
        for (i = 0; i < metas.length; i++)
            if (metas[i].getAttribute('name') == "citation_doi")
                break;
        var citation_doi = metas[i].getAttribute('content');
        return citation_doi;
    }

</script>


        <link title="Latest articles" rel="alternate" href="http://www.biomedcentral.com/bmcgenomics/latest/rss" type="application/rss+xml" exposeToHeader="true"/>
    <link title="Most viewed" rel="alternate" href="http://www.biomedcentral.com/bmcgenomics/mostviewed/rss/" type="application/rss+xml" exposeToHeader="true"/>
    <link title="Latest comments" rel="alternate" href="http://www.biomedcentral.com/bmcgenomics/latestcomments/rss" type="application/rss+xml" exposeToHeader="true"/>
    


	<script type="text/javascript">
	var _gaq = _gaq || [];
	_gaq.push(['_setAccount', 'UA-72326-1']);
	_gaq.push(['_gat._anonymizeIp']);
	_gaq.push(['_setLocalRemoteServerMode']);

		_gaq.push(['_trackPageview']); 

	(function() {
		var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
		ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
		var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
	})();
	</script>

	
        </head>


<body class=" theme    ">





 
    
    
                                                                                                                                                                                                                                
                                                                                                                                                            
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
    
<div id="branding">
            			
		<dl class="google-ad wide ">
	<dt class="hide">
					<a class="banner-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising"></a>
				
			</dt>
		<dd>

		<!-- OAS AD 'Top' begin -->
		<script type="text/javascript">
		OAS_AD('Top');

		</script>
		<!-- OAS AD 'Top' end -->
		</dd>

		</dl>

		<noscript>
			<dl class="google-ad wide noscript">  
    <dt class="hide"></dt>
    
    <dd>
        <a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcgenomics/article/10.1186/1471/2164/13/425/18635758938@Top?">
            <img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/biomedcentral.com/bmcgenomics/article/10.1186/1471/2164/13/425/12004528008@Top?" alt="advert"/>
        </a>
    </dd>
			</dl>
		</noscript>
				    <div class="sup-panel-outer">
        <div class="sup-panel-inner">
            <ul id="login" class="login">
        <li class="greeting"><strong>Welcome <span id="username">Cameron Neylon</span></strong></li>
            <li id="loginLink"><a href="/bmcgenomics/logoff">Log off</a></li>
    </ul>            <ul class="nav-sup">
    
        <li id="BMC" class="current">
            <a href="http://www.biomedcentral.com/"><span class="img-title">&nbsp;</span>
            <span class="text-title">BioMed Central</span></a>
        </li>
		
        		<li><a href="http://www.biomedcentral.com/journals"><span>Journals</span></a></li>
		
								<li>
				<a href="http://www.biomedcentral.com/gateways"><span>Gateways</span></a>
			</li>
					    	
</ul>
        </div>
    </div>
    <div class="branding-inner">
					<div class="logo">
			<a href="/bmcgenomics/"><img alt="BMC Genomics" title="BMC Genomics Logo" src="/sites/1008/images/logo.gif" /></a>			
			</div>		
				
                    

        <div id="impact-factor" class="official">
    <img src="/images/branding/official.gif" alt="official impact factor" title="4.07"/>
    <span id="impact-factor-value">4.07</span>
</div>
    
                            
<div class="module search gray small  ">
    <div class="module-inner padded">
        <form method="get" class="searchForm"
              action=" /quickSearchRedirectURL "
              onsubmit="window.location.href=this['url'].value + '?terms=' + encodeURIComponent(this['terms'].value); return false;">
            <fieldset class="search">
			                <span>
                    Search <select name="url">
                                <option value="http://www.biomedcentral.com/bmcgenomics/search/results" selected>BMC Genomics</option>
                                <option value="http://www.biomedcentral.com/search/results">BioMed Central</option>
                           </select> for
                </span>
                            <input id="searchTerms" name="terms" type="text" alt="Search terms" class="text " />
                 <button class="w37" value="Go" type="submit">Go</button>
            </fieldset>
        </form>
    </div>
	<a class="advanced-search" href="/bmcgenomics/search">Advanced search</a>
</div> 

                        <div id="mobile-nav-list" >
	<ul class="primary-nav">
		<li     ><a href="/bmcgenomics/"><span id="home-tab">Home</span></a></li>
														<li             class="current"
    ><a href="/bmcgenomics/content"><span id="articles-tab">Articles</span></a></li>
										
	

	
            <li     ><a href="/bmcgenomics/authors/instructions"><span id="authors-tab">Authors</span></a></li>
        <li     ><a href="/bmcgenomics/about/reviewers"><span id="reviewers-tab">Reviewers</span></a></li>
    		<li     ><a href="/bmcgenomics/about"><span id='about-tab'>About this journal</span></a></li>
	<li     ><a href="/bmcgenomics/my"><span id="my-tab">My BMC Genomics</span></a></li>
		
    		</ul>
</div>
            </div>
</div>

<hr class="hide" />
<div class="container">
	<div id="content"  class="overflow-visible" >
			    
 	

<!--div class="branding-mobile">
		<h1 class="logo">
	    <a href="/bmcgenomics/">
		    <img alt="BMC Genomics" title="BMC Genomics Logo" src="/sites/1008/images/logo.gif" />
		</a>
		</h1>
	</div><br/-->



			
	



<div class="outline-wrapper block mobile-hidden" id="left-article-box">
<ul id="box-outline" class="box">
<ul id="outline" class="outline">
   <li>
      <a href="#">Top</a>
   </li>
   <li>
      <a href="#abs">Abstract</a>
   </li>
   <li>
      <a href="#sec1">Background</a>
   </li>
   <li>
      <a href="#sec2">Methods</a>
   </li>
   <li>
      <a href="#sec3">Results</a>
   </li>
   <li>
      <a href="#sec4">Discussion</a>
   </li>
   <li>
      <a href="#sec5">Conclusion</a>
   </li>
   <li>
      <a href="#sec6">Competing interests</a>
   </li>
   <li>
      <a href="#sec7">Authors’ contributions</a>
   </li>
   <li>
      <a href="#ack">Acknowledgments</a>
   </li>
   <li class="lowest">
      <a href="#refs">References</a>
   </li>
</ul>
</ul>
<dl id="box-outline1">
</dl>
</div>

<div class="mobile-hidden" id="right-panel">

<div id="mobile-sidebar">
	<div id="mobile-sidebar-tab">
		<span>
			<a href="#" id="mobile-sidebar-tab-link"><img style="opacity: 0.5;" src="/images/sidebar-hidden.png"><img style="display:none;" src="/images/sidebar-shown.png"></a>
		</span>
	</div>	<div id="mobile-sidebar-list">

<div id="article-navigation-bar">
    <div class="issue-information block">
        <h5><a href="http://www.biomedcentral.com/bmcgenomics"><strong>BMC Genomics</strong></a></h5>
        <ul class="square normal">
            <li><a href="http://www.biomedcentral.com/bmcgenomics/content/13">Volume 13</a></li>
				
                    </ul>
    </div>

    <div class="article-information block" id="article-info">
        


<div id="viewing-options-links">
	<h5>Viewing options</h5>
	<ul class="square normal">
        
            <li>
                             <a href="/1471-2164/13/425/abstract">Abstract</a>
                        </li>
        				<li>
							<strong>Full text</strong>
					</li>
				
		
		
		<li class="pdfFileSize">			<a href="/content/pdf/1471-2164-13-425.pdf">View PDF</a>
			<span>(3.4MB)</span>		</li>
		
		<li>
                    </li>
		
				<li>
						<a href="/1471-2164/13/425/additional">Additional files</a>
					</li>
			</ul>
</div>
		
<div id="associated-material-links">
    <h5>Associated material</h5>
    <ul class="square normal">
                        <li>
            <a href="/pubmed/22920992">PubMed record</a>
        </li>
                            <li><a href="/1471-2164/13/425/about">About this article</a></li>
                <li>
                        <a href="/1471-2164/13/425/comments">Readers' comments </a>
                    </li>
        <li>
                    </li>
    </ul>
</div>
		




<div id="related-literature-links">
    <h5>Related literature</h5>
    <ul class="normal">
                    <li> <a href="http://www.biomedcentral.com/1471-2164/13/425/about#citations"><h6>Cited by</h6></a></li>
                <li id="google-blog-search"><a href="http://www.google.com/search?tbm=blg&hl=en&amp;source=hp&amp;biw=1280&amp;bih=899&amp;q=Yellow+lupin+%28Lupinus+luteus+L.%29+transcriptome+sequencing%3A+molecular+marker+development+and+comparative+studies&amp;btnG=Search">on Google blog search</a></li>
        <h6>Other articles by authors</h6>
        <li>
            <a class="collapser"><i>&nbsp;</i>on Google Scholar</a>
<ul id="authg" class="hidebeforeload">                         <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22LB+Parra-González%22">Parra-González LB</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22GA+Aravena-Abarzúa%22">Aravena-Abarzúa GA</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22CS+Navarro-Navarro%22">Navarro-Navarro CS</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22J+Udall%22">Udall J</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22J+Maughan%22">Maughan J</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22LM+Peterson%22">Peterson LM</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22HE+Salvo-Garrido%22">Salvo-Garrido HE</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22IJ+Maureira-Butler%22">Maureira-Butler IJ</a>
            </li>
            </ul>          </li>
                <li>
            		<a class="collapser"><i>&nbsp;</i>on PubMed</a>
	<ul id="authpm" class="hidebeforeload"> 									<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Parra-González_LB%20[Author]">Parra-González LB</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Aravena-Abarzúa_GA%20[Author]">Aravena-Abarzúa GA</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Navarro-Navarro_CS%20[Author]">Navarro-Navarro CS</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Udall_J%20[Author]">Udall J</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Maughan_J%20[Author]">Maughan J</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Peterson_LM%20[Author]">Peterson LM</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Salvo-Garrido_HE%20[Author]">Salvo-Garrido HE</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Maureira-Butler_IJ%20[Author]">Maureira-Butler IJ</a>
				</li>
						</ul > 	        </li>
    </ul>     <h6>Related articles/pages</h6>
    <ul class="square normal">
        <li><a href="http://www.google.co.uk/search?hl=en&amp;q=related:http://www.biomedcentral.com/1471-2164/13/425">on Google</a></li>
        <li><a href="http://scholar.google.com/scholar?q=related:http://www.biomedcentral.com/1471-2164/13/425">on Google Scholar</a></li>
                    <li><a href="/pubmed/related/22920992">on PubMed</a></li>
            </ul>

</div>		
        
    
    


    	





<h5>Tools</h5>
	<ul class="square normal">
									<li>
						<a href="/1471-2164/13/425/citation">Download references</a>
					</li>
				<li><a href="/content/download/xml/1471-2164-13-425.xml">Download XML</a></li>
				<li>
						<a href="/1471-2164/13/425/email">Email to a friend</a>
					</li>
		<li><a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=478&doi=10.1186/1471-2164-13-425&volume=13&issue=1&title=Yellow+lupin+%28Lupinus+luteus+L.%29+transcriptome+sequencing%3A+molecular+marker+development+and+comparative+studies&author_name=Lorena B Parra-González&start_page=1&end_page=15">Order reprints</a></li>
		<li>
						<a href="/1471-2164/13/425/postcomment">Post a comment</a>
					</li>
	</ul>
		



<h5>Share this article</h5>
<ul id="social-networking-links">
	<li>
	<div class="fb-like" data-href="http://www.biomedcentral.com/1471-2164/13/425" data-send="false" data-layout="button_count" data-width="100" data-show-faces="false" data-action="recommend"></div>
	</li>
	
	<li>
		<a href="http://twitter.com/share" data-url="http://www.biomedcentral.com/1471-2164/13/425" data-text="#bmcgenomics
 Yellow lupin (Lupinus luteus L.) transcriptome sequencing: molecular marker development and comparative studies" class="twitter-share-button">Tweet</a>
	</li>
	<li>
			<g:plusone size="medium" width="100"></g:plusone>
	</li>
	<li>
		<a class="sharethis_title btn" href="#"><span class="icon">&nbsp;</span>More options...</a>
	<ul class="iconlist hidebeforeload" id="sharethis">
			<li>
				<a onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false" href="http://www.citeulike.org/posturl?url=http://www.biomedcentral.com/1471-2164/13/425"><span class="share-icons citeulike"></span>Citeulike</a>
			</li>
					<li>
					<a href="http://www.connotea.org/addpopup?continue=confirm&amp;uri=http://dx.doi.org/doi:10.1186/1471-2164-13-425" onclick="largepopup(this.href,'Connotea',800,600);return false"><span class="share-icons connotea"></span>Connotea</a>
	</li>
				<li>
				<a onclick="largepopup(this.href,'Del.icio.us',800,600);return false" href="http://del.icio.us/post?url=http://www.biomedcentral.com/1471-2164/13/425&title=Yellow lupin (Lupinus luteus L.) transcriptome sequencing: molecular marker development and comparative studies"><span class="share-icons delicious"></span>Del.icio.us</a>
	</li>
			<li>
				<a href="/1471-2164/13/425/email"><span class="share-icons email"></span>Email</a>
	</li>
			<li>
				<a onclick="largepopup(this.href,'facebook',800,600);return false" href="http://www.facebook.com/sharer.php?u=http://www.biomedcentral.com/1471-2164/13/425"><span class="share-icons facebook"></span>Facebook</a>
	</li>
	<li class="gp">
			<div class="googlehider mobile-hide"> 
				<div class="g-plusone" data-annotation="none"></div>
			</div>
				<span class="share-icons googleplus mobile-hide"></span>
				<div class="googlehider2 mobile-hide"> 
				<div class="g-plusone mobile-hide" data-annotation="none"></div>
			</div>
				<span class="share-icons googleplus"></span>
				<span class="gp-link"><a href="#">Google+</a></span>
	</li>
			<li>
				<a onclick="largepopup(this.href,'mendeley',800,600);return false" href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1471-2164/13/425"><span class="share-icons mendeley"></span>Mendeley</a>
			</li>
			<li>
				<a onclick="largepopup(this.href,'twitter',800,600);return false" href="http://twitter.com/?status=%23bmcgenomics%0D%0AYellow%20lupin%20%28Lupinus%20luteus%20L.%29%20transcriptome%20sequencing%3A%20molecular%20marker%20development%20and%20comparative%20studies%20http%3A%2F%2Fwww.biomedcentral.com%2F1471-2164%2F13%2F425"><span class="share-icons twitter"></span>Twitter</a>  
			</li>
		</ul>
</ul>

    </div>

    	
	
    		









		
    
        
        
    
        
    
        
        
    
		
	
	
</div>
		<div class="mobile-sidebar-gradient top"></div>
		<div class="mobile-sidebar-gradient bottom"></div>
	</div>

    

</div>





</div>
<div class="rounded custom white article full-text">
<div class="wrap-inner content">
    <div class="padded-inner">                


    
 
    
    
                                                                                                                                                                                                                                
                                                                                                                                                            
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
    
<div id="topmatter">
	
	   








				<a href="/about/access"><img alt="Open Access" src="/images/articles/openaccess-large.png" class="access mr15"/></a>
			<a href="http://www.biomedcentral.com/about/mostviewed/"><img alt="Highly Accessed" src="/images/articles/highlyaccessed-large.png" class="access mr15"/></a>
				<span class="articletype">Research article</span>
<h1>Yellow lupin (<em>Lupinus luteus</em> L.) transcriptome sequencing: molecular marker development and comparative studies</h1>

        
			<div class="multipleins">
		
	<p class="authors">
        																																																																																																																																																																																																																																																																			        <strong>Lorena B Parra-González</strong><sup><a href="/1471-2164/13/425/#ins1">1</a></sup>, <strong>Gabriela A Aravena-Abarzúa</strong><sup><a href="/1471-2164/13/425/#ins1">1</a></sup>, <strong>Cristell S Navarro-Navarro</strong><sup><a href="/1471-2164/13/425/#ins1">1</a></sup>, <strong>Joshua Udall</strong><sup><a href="/1471-2164/13/425/#ins2">2</a></sup>, <strong>Jeff Maughan</strong><sup><a href="/1471-2164/13/425/#ins2">2</a></sup>, <strong>Louis M Peterson</strong><sup><a href="/1471-2164/13/425/#ins2">2</a></sup>, <strong>Haroldo E Salvo-Garrido</strong><sup><a href="/1471-2164/13/425/#ins1">1</a>,</sup><sup><a href="/1471-2164/13/425/#ins3">3</a></sup> and <strong>Iván J Maureira-Butler</strong><sup><a href="/1471-2164/13/425/#ins1">1</a></sup><sup>&#42;</sup>	</p>

			<div id="affiliations">
		<div class="module gray  inner">
<div class="module-inner padded-inner">
		<ul>
			<li>                                                                                                            
                            		<p class="authors" >
							<span>*</span>
									 Corresponding author:										            Iván J Maureira-Butler <a href='mailto:ivan.maureira@cgna.cl'>ivan.maureira@cgna.cl</a>    					</p>
      
</li>
			
                                                                                                                                                                                                                                                		</ul>

		
		
            <section> 
						<div class="collapsible-content">
							 <p class="options"><a class="affiliations-toggle" href="#"><i class="arrow"></i>Author Affiliations</a></p>				<div id="ins_container" style="display: block;">
										    						    <p >
						        <sup id="ins1">1</sup>                                Agriaquaculture Nutritional Genomic Center, CGNA, Genomics and Bioinformatics Unit, Km 10 Camino Cajón-Vilcún, INIA, Temuco, Chile
                            </p>
																    						    <p >
						        <sup id="ins2">2</sup>                                Plant and Wildlife Science Department, Brigham Young University, 150 E 1230 North, Provo, UT, 84602, USA
                            </p>
																    						    <p >
						        <sup id="ins3">3</sup>                                Institute of Agricultural Research (INIA), P.O. Box 58-D, Temuco, Chile
                            </p>
																
				</div>
										<p id="authoremails">            Email:             Lorena B Parra-González <a href='mailto:lorena.parra@cgna.cl'>lorena.parra@cgna.cl</a> -             Gabriela A Aravena-Abarzúa <a href='mailto:gabriela.aravena@cgna.cl'>gabriela.aravena@cgna.cl</a> -             Cristell S Navarro-Navarro <a href='mailto:cristell.navarro@cgna.cl'>cristell.navarro@cgna.cl</a> -             Joshua Udall <a href='mailto:jaudall@gmail.com'>jaudall@gmail.com</a> -             Jeff Maughan <a href='mailto:jeff_maughan@byu.edu'>jeff_maughan@byu.edu</a> -             Louis M Peterson <a href='mailto:louis98405@gmail.com'>louis98405@gmail.com</a> -             Haroldo E Salvo-Garrido <a href='mailto:haroldo.salvo@cgna.cl'>haroldo.salvo@cgna.cl</a> -             Iván J Maureira-Butler <a href='mailto:ivan.maureira@cgna.cl'>ivan.maureira@cgna.cl</a>        </p>
						</div>
			</section>
				</div>
</div>
		</div>
	  </div>

	<section class="cit">	
				<div class="collapsible-content">
    <p><em>BMC Genomics</em> 2012, <strong>13</strong>:425&nbsp;<span class="pseudotab">doi:10.1186/1471-2164-13-425</span></p>	
		
    
    		
        		<br/>
		<p>The electronic version of this article is the complete one and can be found online at: <a href="http://www.biomedcentral.com/1471-2164/13/425">http://www.biomedcentral.com/1471-2164/13/425</a></p>
		<br/>
                    <table cellspacing="0" cellpadding="0">
        <tbody>
        <tr><td>Received:</td><td>9 May 2012</td></tr>                <tr><td>Accepted:</td><td>13 August 2012</td></tr>        <tr><td>Published:</td><td>24 August 2012</td></tr>        </tbody>
        </table>
    		<br/>
        <p>
            &copy; 2012 Parra-González et al.; licensee BioMed Central Ltd. <br />
        </p>
					<p>
				This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href='http://creativecommons.org/licenses/by/2.0'>http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.
			</p>
		    	</div>
	</section>

</div>
    	<div id="article-body">		
    <section><a name="abs"></a><h3>Abstract</h3>
   <div class="collapsible-content">
      <h4>Background</h4>
      <p style="line-height:160%">Yellow lupin (<em>Lupinus luteus</em> L.) is a minor legume crop characterized by its high seed protein content. Although
         grown in several temperate countries, its orphan condition has limited the generation
         of genomic tools to aid breeding efforts to improve yield and nutritional quality.
         In this study, we report the construction of 454-expresed sequence tag (EST) libraries,
         carried out comparative studies between <em>L. luteus</em> and model legume species, developed a comprehensive set of EST-simple sequence repeat
         (SSR) markers, and validated their utility on diversity studies and transferability
         to related species.
      </p>
      <h4>Results</h4>
      <p style="line-height:160%">Two runs of 454 pyrosequencing yielded 205 Mb and 530 Mb of sequence data for L1 (young
         leaves, buds and flowers) and L2 (immature seeds) EST- libraries. A combined assembly
         (L1L2) yielded 71,655 contigs with an average contig length of 632 nucleotides. L1L2
         contigs were clustered into 55,309 isotigs. 38,200 isotigs translated into proteins
         and 8,741 of them were full length. Around 57% of <em>L. luteus</em> sequences had significant similarity with at least one sequence of <em>Medicago</em>, <em>Lotus</em>, <em>Arabidopsis</em>, or <em>Glycine</em>, and 40.17% showed positive matches with all of these species. <em>L. luteus</em> isotigs were also screened for the presence of SSR sequences. A total of 2,572 isotigs
         contained at least one EST-SSR, with a frequency of one SSR per 17.75 kbp. Empirical
         evaluation of the EST-SSR candidate markers resulted in 222 polymorphic EST-SSRs.
         Two hundred and fifty four (65.7%) and 113 (30%) SSR primer pairs were able to amplify
         fragments from <em>L. hispanicus</em> and <em>L. mutabilis</em> DNA, respectively. Fifty polymorphic EST-SSRs were used to genotype a sample of 64 <em>L. luteus</em> accessions. Neighbor-joining distance analysis detected the existence of several
         clusters among <em>L. luteus</em> accessions, strongly suggesting the existence of population subdivisions. However,
         no clear clustering patterns followed the accession’s origin.
      </p>
      <h4>Conclusion</h4>
      <p style="line-height:160%"><em>L. luteus</em> deep transcriptome sequencing will facilitate the further development of genomic
         tools and lupin germplasm. Massive sequencing of cDNA libraries will continue to produce
         raw materials for gene discovery, identification of polymorphisms (SNPs, EST-SSRs,
         INDELs, etc.) for marker development, anchoring sequences for genome comparisons and
         putative gene candidates for QTL detection.
      </p>
   </div>
</section><span id="keywords">
   <h5 class="inline">Keywords: </h5><em>Lupinus luteus</em>; EST-SSR; Orphan crop; Microsynteny</span><section><a name="sec1"></a><h3>Background</h3>
   <div class="collapsible-content">
      <p style="line-height:160%"><em>L. luteus</em> is a member of the genistoid clade of the Fabaceae family (2n = 52), which is the
         third largest flowering plant family with over 700 genera and 20,000 species <a name="d47075e246"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>]. The genus <em>Lupinus</em> comprises more than 200 annual and perennial herbaceous species of which several
         are cultivated and used as human food or animal feed <a name="d47075e253"></a>[<a onclick="LoadInParent('#B2'); return false;" href="#B2">2</a>]. Some of them show high levels of tolerance to biotic and abiotic stresses. For instance,
         <em>L. hispanicus,</em> a wild relative of <em>L. luteus</em>, has high tolerance to diseases and good adaptation to poor soils, but high levels
         of bitter alkaloids and low agronomic yields <a name="d47075e264"></a>[<a onclick="LoadInParent('#B3'); return false;" href="#B3">3</a>]. Lupins are considered to be of polyploid origin which probably played a crucial
         role in the evolution of their ancestral genomes <a name="d47075e268"></a><a name="d47075e270"></a>[<a onclick="LoadInParent('#B4'); return false;" href="#B4">4</a></a>,<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a>]. The major cultivated species are the old world lupin <em>L. albus</em> (white lupin), <em>L. angustifolius</em> (narrow-leafed lupin), <em>L. luteus</em> (yellow lupin), and the new world species <em>L. mutabilis</em> (pearl lupin or tarwii) <a name="d47075e287"></a>[<a onclick="LoadInParent('#B6'); return false;" href="#B6">6</a>].
      </p>
      <p style="line-height:160%"><em>L. luteus</em> is widely distributed across the Mediterranean region, has shallow soil requirements,
         and cultivated accessions have variable seed yields in Mediterranean environments
         <a name="d47075e295"></a>[<a onclick="LoadInParent('#B7'); return false;" href="#B7">7</a>]. In addition, yellow lupin seeds have the highest protein content and twice the cysteine
         and methionine content of most lupins <a name="d47075e299"></a><a name="d47075e301"></a>[<a onclick="LoadInParent('#B8'); return false;" href="#B8">8</a></a>,<a onclick="LoadInParent('#B9'); return false;" href="#B9">9</a>]. However, despite its highly nutritional qualities, there is a lack of genetic and
         molecular tools to aid the genetic breeding of this species.
      </p>
      <p style="line-height:160%">EST sequencing has accelerated gene discovery when genome sequences are not available,
         facilitating gene family identification and development of molecular markers. Next-generation
         sequencing has generated enormous amount of expressed sequence data for a wide number
         of plant species, specially minor or orphan crops <a name="d47075e307"></a>[<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a>]. For example, EST and genome sequencing of lentil and chickpea would not have been
         feasible without next-generation sequencing <a name="d47075e311"></a><a name="d47075e313"></a>[<a onclick="LoadInParent('#B11'); return false;" href="#B11">11</a></a>,<a onclick="LoadInParent('#B12'); return false;" href="#B12">12</a>]. The lower cost and greater sequence yield has allowed the identification of candidate
         genes, even when they are expressed at low levels <a name="d47075e317"></a><a name="d47075e319"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a></a>,<a onclick="LoadInParent('#B14'); return false;" href="#B14">14</a>].
      </p>
      <p style="line-height:160%">Research on plants, animals and fungi has shown that sequences of expressed genes
         are often widely transferable among species, and even genera, allowing wide genome
         comparative mapping studies <a name="d47075e325"></a><a name="d47075e327"></a>[<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a></a>,<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>]. For instance, the combination of orphan crop EST sequences with model plant genetic
         and genomic resources, such as <em>Lotus japonicus</em> (Japanese trefoil) and <em>Medicago truncatula</em> (barrel medic), has identified macro- and micro-scale synteny, discovered new genes
         and alleles, and provided insights into genome evolution and duplication <a name="d47075e337"></a><a name="d47075e339"></a>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a></a>,<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>]. Comparisons between ESTs and gene sequences among several legume species have allowed
         comparative genome studies between <em>L. albus</em> and <em>M. truncatula</em><a name="d47075e349"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>], and <em>L. angustifolius</em> and <em>Lotus japonicus</em><a name="d47075e358"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>].
      </p>
      <p style="line-height:160%">Several molecular markers have been developed for <em>Lupinus</em> species, including RFLPs, ITAPs (Intron targeted amplified polymorphic sequences),
         and AFLPs, which have been used to build genetic linkage maps in <em>L. albus</em><a name="d47075e369"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>] and <em>L. angustifolius</em><a name="d47075e375"></a><a name="d47075e377"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a></a>,<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a>]. So far, a limited number of SSRs have been developed for <em>Lupinus</em> species, and very few of these are EST-SSRs <em>i.e.</em> SSRs that are found in expressed sequences <a name="d47075e388"></a><a name="d47075e390"></a><a name="d47075e392"></a>[<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a>-<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>]. Genomic and EST-SSRs have been widely used for the improvement of major crop plants,
         but their initial development with traditional methods requires significant research
         investment. Now, an almost unlimited number of genomic and EST-SSRs can be readily
         developed from next-generation sequencing approaches within most crop species, including
         orphan crops such as lupin <a name="d47075e396"></a><a name="d47075e398"></a><a name="d47075e400"></a><a name="d47075e402"></a><a name="d47075e404"></a>[<a onclick="LoadInParent('#B24'); return false;" href="#B24">24</a>-<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>]. The expressed nature of EST-SSRs allows the annotation of these markers with putative
         functions by sequence homology and potentially reduces the genetic distance between
         marker and causal gene to 0 cM. <a name="d47075e408"></a><a name="d47075e410"></a>[<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a></a>,<a onclick="LoadInParent('#B30'); return false;" href="#B30">30</a>]. For instance, the length of a dinucleotide SSR at the 5’ UTR of a waxy gene has
         been associated with amylase content in rice <a name="d47075e414"></a><a name="d47075e416"></a>[<a onclick="LoadInParent('#B31'); return false;" href="#B31">31</a></a>,<a onclick="LoadInParent('#B32'); return false;" href="#B32">32</a>]. EST-SSRs have also been associated with several disease resistant genes in wheat
         and rice <a name="d47075e420"></a><a name="d47075e422"></a>[<a onclick="LoadInParent('#B33'); return false;" href="#B33">33</a></a>,<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>] and a number of agronomically important traits in cotton, maize and narrow-leafed
         lupin <a name="d47075e427"></a><a name="d47075e429"></a><a name="d47075e431"></a>[<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>-<a onclick="LoadInParent('#B37'); return false;" href="#B37">37</a>].
      </p>
      <p style="line-height:160%">In this study, we constructed 454-EST libraries, carried out comparative studies between
         <em>L. luteus</em> and model legume species, and mapped <em>L. luteus</em> expressed sequences on the <em>M. truncatula</em> chromosomes. Alignments between our putative <em>L. luteus</em> genes and their homologs in <em>M. truncatula</em>, coupled with amplifications of intergenic regions provided evidence of microscale
         synteny between both species. In addition, we developed EST-SSR markers and illustrated
         their utility within diverse accessions of yellow lupin. Finally, because these EST-SSR
         markers are gene-based, they are also likely conserved among different species of
         lupin. We evaluated EST-SSR utility in the other <em>Lupinus</em> species, <em>L. mutabilis</em> and <em>L. hispanicus.</em></p>
   </div>
</section>
<section><a name="sec2"></a><h3>Methods</h3>
   <div class="collapsible-content">
      <h4>Library construction and 454 sequencing</h4>
      <p style="line-height:160%">cDNA libraries were constructed from mRNA isolated from two tissue pools. Pool 1 (L1)
         included young leaves, buds and flowers, and pool 2 (L2), seeds in different developmental
         stages. RNA from pool 1 and 2 was isolated separately according to the guanidine hydrochloride
         method <a name="d47075e471"></a>[<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a>]. Both RNAs were assessed for quality by inspecting rRNA bands on an Agilent Bioanalyzer
         (Agilent Technologies, CA, USA).
      </p>
      <p style="line-height:160%">cDNAs libraries were normalized and prepared using procedures for Roche 454 Titanium
         sequencing (Roche, Branford, CT, USA). cDNAs from L1 and L2 were synthesized using
         the stratagene AccuScript High Fidelity RT-PCR System (Agilent Technologies, CA, USA)
         and 5’ specific adaptors from Clontech. A cDNA normalization was used to improve coding
         sequence coverage, avoid AT homopolymer artifacts, and reduce excessive 3’ end transcript
         sequence <a name="d47075e477"></a>[<a onclick="LoadInParent('#B39'); return false;" href="#B39">39</a>]. cDNAs from both libraries were amplified using the Clontech Advantage HF system
         (Clontech Laboratories, Inc) and normalized utilizing the Evrogen Trimmer cDNA Normalization
         kit (Axxora, LLC). These un-cloned, normalized cDNA libraries were prepared for pyrosequencing
         according to the manufacturers specifications. One 454 run of sequencing was performed
         for each EST library (454 Life Sciences, Roche).
      </p>
      <p style="line-height:160%">Separate transcriptome assemblies of L1 and L2 libraries were created using Newbler
         (<em>de novo</em> sequence assembly software of Roche 454 Life Sciences) and the cDNA option. A third
         assembly (L1L2) was completed using the reads from both libraries to avoid sequence
         redundancy when developing SSR markers. Reads were initially assembled into contigs
         and contigs into isotigs, which are equivalent to splice transcriptional variants.
         Sequence read EST data for L1 and L2 are available through the Sequence Read Archive
         (SRA055806).
      </p>
      <h4>EST annotation, function and comparative genomics to other species</h4>
      <p style="line-height:160%">Comparing isotigs from the combined assembly (L1L2) to the curated non-redundant protein
         database (nr, <a href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a> <a href="http://www.webcitation.org/query.php?url=http://www.ncbi.nlm.nih.gov&amp;refdoi=10.1186/1471-2164-13-425" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>; blastx, e value ≤ 1e<sup>-10</sup>) provided a functional annotation for each isotig. Alignments of translated-isotigs
         and proteins with an e-value ≤ 1e<sup>-40</sup> were considered to have significant homology. Annotations of the aligned proteins
         were extrapolated to annotate our putative isotig sequence using Blast2GO (<a href="http://www.blast2go.org">http://www.blast2go.org</a> <a href="http://www.webcitation.org/query.php?url=http://www.blast2go.org&amp;refdoi=10.1186/1471-2164-13-425" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>). To directly compare the lupin isotigs to the genes of other crops, blast searches
         were also used to compare isotig translations to <em>Arabidopsis thaliana</em>, <em>Glycine max</em>, <em>Medicago truncatula</em> and <em>Lotus japonicus</em> Gene Indices (tblastx, e-value ≤ 1e<sup>-10</sup>). Isotigs were also annotated using Gene Ontology (GO) annotations from InterProScan
         (<a href="http://www.ebi.ac.uk">http://www.ebi.ac.uk</a> <a href="http://www.webcitation.org/query.php?url=http://www.ebi.ac.uk&amp;refdoi=10.1186/1471-2164-13-425" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>).
      </p>
      <h4><em>In silico</em> lupin EST mapping and microsynteny
      </h4>
      <p style="line-height:160%">Blast was used to compare lupin EST isotigs to the Medicago genome 3.0 release (≤
         1e<sup>-20</sup>, HSP identity 60% and HSP length &gt; 50 bp.) The Blast results were visualized using
         GBrowse where positive matches were displayed as featured tracks on GBrowse 2.13 <a name="d47075e534"></a>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a>]. The presence of microsynteny was evaluated by PCR amplification of putatively conserved
         chromosome blocks between <em>L. luteus</em> and <em>M. truncatula</em>. Where alignments between yellow lupin and <em>M. truncatula</em> were identified, specific primer pairs were designed to amplify intergenic regions
         (Additional file <a name="d47075e547"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/suppl/S1">1</a>). These targeted, intergenic regions were PCR amplified from two <em>L. luteus</em> and one <em>L. hispanicus</em> accessions using 100 ng of genomic DNA in 20 ul reactions containing 100 ng of genomic
         DNA, 0.2 mM dNTPs, 2 mM MgCl2, 1X PCR buffer, 2.5% DMSO, 1 U taq polymerase (Agilent
         Technologies, Santa Clara, CA) and 5 pmoles of each forward-reverse primer pair. PCR
         reactions were carried out following a touchdown protocol on a peltier thermalcycler
         (MJ Research, Inc.) 94°C for 5 min; 5 cycles of 1 min at 94°C, 1 min at 55-65°C decreasing
         1°C per cycle, 2 min at 72°C followed by 35 cycles of 1 min at 94°C, 1 min at 50-60°C
         and 2 min at 72°C. Amplicons were purified from agarose gels and sequenced. These
         amplified, intergenic sequences were mapped onto the <em>M. truncatula</em> genome and visualized within a local implementation of GBrowse (Additional file <a name="d47075e559"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/suppl/S1">1</a>). Positive PCR microsynteny set of primers were additionally tested against a screening
         panel consisting of six diverse accessions of <em>L. luteus</em> to search for polymorphisms among yellow lupin genotypes (Additional file <a name="d47075e566"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/suppl/S2','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/suppl/S2">2</a>).
      </p><div class="figs">
      <div class="addfile "><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/suppl/S1','S1',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/suppl/S1"></a><strong>Additional file 1.</strong> <strong> Table S1.</strong> Characteristics of 33 Conserved Microsynteny (CMS) markers developed in <em>L. luteus.</em> Shown for each primer pair are the Medicago chromosome library specificity, l1l2
         isotigs where CMS forward and reverse primers were anchored, forward and reverse sequence,
         expected Medicago amplicon size (bp), <em>L. luteus</em> CMS amplicon size (bp), amplification in other Lupin species (<em>L. hispanicus</em>), and the level of polymorphism on the <em>L. luteus</em> screening panel.</p><p>
         	Format: PDF 
         	Size: 164KB <a href="/content/supplementary/1471-2164-13-425-s1.pdf">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.adobe.com/products/acrobat/readstep.html','Help');return false;" href="http://www.adobe.com/products/acrobat/readstep.html">Adobe Acrobat Reader</a><span class="open-data-icon"><a href="/about/access/#opendata"><img title="Open Data" alt="Open Data" src="/bmcimages/article/opendata.gif"></a></span></div>
      <div class="addfile "><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/suppl/S2','S2',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/suppl/S2"></a><strong>Additional file 2.</strong> <strong> Table S2. </strong><em>Lupinus luteus</em>, <em>L. hispanicus</em> and <em>L. mutabilis</em> accessions included in the study.</p><p>
         	Format: PDF 
         	Size: 80KB <a href="/content/supplementary/1471-2164-13-425-s2.pdf">Download file</a></p><p>This file can be viewed with: <a onclick="largepopup('http://www.adobe.com/products/acrobat/readstep.html','Help');return false;" href="http://www.adobe.com/products/acrobat/readstep.html">Adobe Acrobat Reader</a><span class="open-data-icon"><a href="/about/access/#opendata"><img title="Open Data" alt="Open Data" src="/bmcimages/article/opendata.gif"></a></span></div></div>
      <h4>Identification of EST-SSRs</h4>
      <p style="line-height:160%">SSR containing lupin isotigs were identified using the software MISA (MIcroSAtellite,
         <a href="http://www.pgrc.ipk-gatersleben.de/misa">http://www.pgrc.ipk-gatersleben.de/misa</a> <a href="http://www.webcitation.org/query.php?url=http://www.pgrc.ipk-gatersleben.de/misa&amp;refdoi=10.1186/1471-2164-13-425" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>). SSR search criteria changed according to repeat types. Di-, and tri-repeats were
         selected with a minimum length of 12 and 15 nucleotides, respectively. For tetra-,
         penta- and hexa-repeats, the minimum length was 20 nucleotides. Mononucleotide repeats
         were not considered due to the possibility of 454 homopolymer sequencing errors associated
         with this technology. To estimate the amount of SSRs included in coding regions, L1L2
         sequences were analyzed using ESTScan (<a href="http://www.ch.embnet.org/software/ESTScan.html">http://www.ch.embnet.org/software/ESTScan.html</a> <a href="http://www.webcitation.org/query.php?url=http://www.ch.embnet.org/software/ESTScan.html&amp;refdoi=10.1186/1471-2164-13-425" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>). ORFs discovery was carried out using default parameters and putative cd sequences
         scanned for SSR motifs using MISA.
      </p>
      <p style="line-height:160%">From all selected-SSR containing isotigs, only sequences with a motif of at least
         7 repeat units were considered for primer design. Flanking primer pairs were designed
         using the Primer3 software available at NCBI v.3.12 with expected amplicon lengths
         between 150 - 500 bp. Oligonucleotides were synthesized by IDT (Integrated DNA Technologies,
         Inc.).
      </p>
      <h4>Evaluation and utility of EST-SSRs</h4>
      <p style="line-height:160%">EST-SSR polymorphisms and transferability were evaluated on the germplasm screening
         panel previously mentioned<em>,</em> and one accession each of <em>L. hispanicus</em> and <em>L. mutabilis</em>.
      </p>
      <p style="line-height:160%">DNAs were extracted following standard procedures <a name="d47075e645"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>], quantified using a synergy HT Multimode Microplate Reader (Biotek Instruments, Winooski,
         VT), and diluted to 50 ng/ul in TE buffer (10 Mm TRIS, 1 mM EDTA pH 7.5). DNA amplification
         was carried out in 20ul PCR reactions as described above.
      </p>
      <p style="line-height:160%">PCR products were separated on 6% denaturing polyacrylamide gels, run in TBE buffer
         at 60 watts for 3–4 hours and visualized using silver stain procedures. DNA amplicons
         of six EST-SSR primer-pairs used in the polymorphism screening were purified from
         agarose gels and sequenced in an Applied Biosystems 3730xl DNA Analyzer sequencer
         (Applied Biosystems, Carlsbad, CA). Amplicon sequences from each EST-SSR primer-pairs
         were aligned using Geneious version 5.5.3.0 (Biomatters Ltd., using default parameters).
      </p>
      <h4>Genetic diversity</h4>
      <p style="line-height:160%">The polymorphic EST-SSRs were evaluated in sixty-four <em>L. luteus</em> accessions from several origins (Poland, Ukraine, the former Soviet Union, Spain,
         Germany, Morocco, Belarus, Portugal, Netherlands, Israel, Hungary, and Chile; Additional
         file <a name="d47075e659"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/suppl/S2','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/suppl/S2">2</a>). Polish accessions were kindly provided by W.K. Swiecicki, Institute of Plant Genetics,
         Polish Academy of Sciences, Poznan. Our collection of Chilean accessions is composed
         of improved breeding lines that are adapted to the Chilean environment. This Chilean
         germplasm originated from breeding and selection of old European varieties for Southern
         Chilean environmental conditions. The rest were obtained from the western Regional
         PI Station, USDA, ARS, WRPIS, Washington State University, Regional Plant Introduction
         Station, Pullman, Washington, USA. A sample of 50 polymorphic EST-SSRs was used to
         genotype the sixty-four <em>L. luteus</em> accessions (Table <a name="d47075e665"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/table/T1">1</a>). Eighteen EST-SSRs were identified from isotigs specific to L2, 25 isotigs specific
         to L1, and seven were common to both L1 and L2 libraries. EST-SSR fragments with different
         sizes were scored as different alleles and coded with alphabetical letters for each
         primer set. Genetic relationships among <em>L. luteus</em> accessions were evaluated using the neighbor-joining algorithm implemented in PAUP*
         (v4.01b10). A distance tree was built and branch support estimated by 10,000 bootstraps.
      </p><div class="figs">
      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/table/T1','T1',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/table/T1"><strong>Table 1.</strong></a> <strong>Characteristics of 50 EST-SSR primers developed in</strong><strong><em> L. luteus.</em></strong><strong> Shown for each primer pair are the library specificity, repeat motif, forward and
            reverse sequence, allele range size (bp), number of alleles, amplification in other
            Lupin species, and annotation</strong></div></div>
   </div>
</section>
<section><a name="sec3"></a><h3>Results</h3>
   <div class="collapsible-content">
      <h4>Seed and leaf-flower EST libraries</h4>
      <p style="line-height:160%">Two runs of 454 pyrosequencing yielded 205 Mb and 530 Mb of sequence data for L1 and
         L2 EST libraries, respectively (Table <a name="d47075e2106"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/table/T2">2</a>). L1 produced 604,869 usable reads that assembled into 26,975 contigs with an average
         length of 468 nucleotides. L2 generated 1,345,892 usable reads that assembled into
         43,674 contigs with an average length of 800 nucleotides. Careful inspection of the
         L1 contigs found lower percentages of coding regions, higher A/T content, and 2x more
         A/T homopolymers than L2 contigs. A combined assembly (L1L2) was created to identify
         the genes that were common in both tissues. 1,964,517 reads were used in the L1L2
         assembly and they formed 71,655 contigs with an average contig length of 632 nucleotides.
         To reduce sequence redundancy due to transcript and alternative splice variants, L1L2
         contigs were clustered into 55,309 isotigs, of which 38,200 isotigs translated into
         proteins and 8,741 of them were full length.
      </p><div class="figs">
      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/table/T2','T2',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/table/T2"><strong>Table 2.</strong></a> <strong>cDNA 454 assembly statistics of L1, L2 and L1L2</strong><strong><em> L. luteus</em></strong><strong> libraries</strong></div></div>
      <h4>Functional classification and <em>in silico</em> comparative genomics
      </h4>
      <p style="line-height:160%">The assembled 454 isotigs represented putative transcriptional products <em>i.e.</em> functional genes. Blastx was used to annotate the L1L2 putative genes (<em>i.e.</em> isotigs). A total of 32,862 (59.5%) putative genes showed matches with other species
         (≤1e<sup>-10</sup>). Of these sequences, 20,169 (36.5%) showed high similarity to other plant species
         genes (≤1e<sup>-40</sup>). GO annotations were grouped under three categories: molecular function, biological
         processes, and cellular components (Figure <a name="d47075e2350"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/figure/F1">1</a>). At least 31,142 isotigs were annotated with one molecular function, 11,894 with
         a cellular component and 22,842 with biological process.
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/figure/F1','F1',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/figure/F1"><img align="top" src="/content/figures/1471-2164-13-425-1.gif" alt="thumbnail" class="thumbnail"><strong>Figure 1 .</strong></a> <strong>GO term annotations for L1L2. Isotigs were grouped under three categories: (a) molecular
            function, (b) biological processes, and (c) cellular components.</strong> Numbers between parentheses indicate the number of positive matches for each function.
      </div></div>
      <p style="line-height:160%">Blast was used to compare L1L2 to several model species (tblastx; ≤ 1e<sup>-10</sup>; Figure <a name="d47075e2371"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/figure/F2">2</a>). Around 57% (31,520) of <em>L. luteus</em> sequences had significant similarity with at least one sequence of <em>Medicago</em>, <em>Lotus</em>, <em>Arabidopsis</em>, or <em>Glycine</em>, and 40.17% showed positive matches with all of these species.
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/figure/F2','F2',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/figure/F2"><img align="top" src="/content/figures/1471-2164-13-425-2.gif" alt="thumbnail" class="thumbnail"><strong>Figure 2 .</strong></a> <strong>Venn diagram summarizing the distribution of tBlastX matches between </strong><strong><em>L. luteus </em></strong><strong>and four model species (</strong><strong><em>A. thaliana, M. truncatula, L. japonicus and G. max</em></strong><strong>).</strong> Numbers following the model species correspond to the size of the respective data
         base. Numbers within the Venn diagram indicate the number of sequences sharing similarity
         using tBLASTx. Numbers within parenthesis indicate the percentage of matches in terms
         of the total number of <em>L. luteus</em> sequences.
      </div></div>
      <h4><em>In silico</em> mapping of lupin ESTs on <em>M. Truncatula</em> chromosomes
      </h4>
      <p style="line-height:160%">Alignment of <em>L. luteus</em> isotig sequences to the <em>M. truncatula</em> genome (Blastn; ≤1e<sup>-20</sup>; MT3) was used to identify local genomic variability between our ESTs and a related,
         well-annotated reference genome sequence. The alignments were visualized using GBrowse
         (v. 2.13) with the Blast matches displayed as feature tracks. A total of 25,400 sequences
         (46%) from L1L2 had a positive match with MT3 and were distributed heterogeneously
         on the <em>M. truncatula</em> chromosomes. Chromosomes 3 and 1 had the highest (34,636) and lowest (16,055) number
         of matches, respectively. Each <em>L. luteus</em> sequence was mapped to an average of 3.7 positions on the <em>Medicago</em> genome.
      </p>
      <p style="line-height:160%">Occasionally, independent alignments of lupin genes with the <em>M. truncatula</em> genome were found relatively close to each other that primers could be designed to
         hybridize conserved exons, allowing the amplification of intergenic sequences in between
         lupin and <em>M. truncatula</em> coding sequences (Figure <a name="d47075e2460"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/figure/F3">3</a>). Positive PCR amplification of intergenic regions using <em>L. luteus</em> genomic DNA and primers anchored on conserved exonic regions of adjacent <em>M. truncatula</em> genes suggested the occurrence of microsynteny (<em>i.e.</em> conserved gene order) between yellow lupin and <em>Medicago.</em> Thirty-three out of 79 (42%) primer pairs amplified clear PCR products. 16 pairs
         showed expected sizes based on <em>Medicago</em> genomic regions. The remainder primer pairs amplified shorter or longer lupin fragments
         than the fragments amplified in <em>M. truncatula</em>. Amplicon sequence data for <em>L. luteus</em> containing intergenic DNA sequence were mapped onto the <em>Medicago</em> genome using blast (Figure <a name="d47075e2489"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/figure/F3">3</a>). The alignments between <em>L. luteus</em> and <em>Medicago</em> showed high levels of conservation in the coding regions, but little sequence similarity
         in the intergenic regions. When <em>L. hispanicus</em> DNA was included as PCR template, only 23 primer pairs amplified. Variable amplification
         was likely due to localized sequence polymorphism within the primer binding site (<em>i.e.</em> small indels) and not the lack of microsynteny. This ratio (23/33) is similar to
         the number of EST-SSRs that were found to amplify fragments in both species. Alignments
         among <em>L. luteus</em> and <em>L. hispanicus</em> were possible at intergenic regions but sequences were clearly less similar than
         coding regions.
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/figure/F3','F3',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/figure/F3"><img align="top" src="/content/figures/1471-2164-13-425-3.gif" alt="thumbnail" class="thumbnail"><strong>Figure 3 .</strong></a> <strong>Microsyntenic </strong><strong><em>L. luteus </em></strong><strong>DNA fragments mapped on the Medicago genome using a GBrowse platform.</strong> (<strong>a</strong>) <em>L. luteus</em> microsyntenic region 13 on <em>M. truncatula</em> chromosome 1; (<strong>b</strong>) <em>L. luteus</em> microsyntenic region 5 on <em>M. truncatula</em> chromosome 1; (<strong>c</strong>) <em>L. luteus</em> microsyntenic region 11 on <em>M. truncatula</em> chromosome 2.
      </div></div>
      <p style="line-height:160%">When these markers were evaluated on the screening panel of diverse germplasm accessions,
         10 had length polymorphism for these intergenic regions (Additional file <a name="d47075e2562"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/suppl/S1">1</a>). In addition to EST-SSRs, this new Conserved Microsynteny (CMS) marker could be
         valuable resource for crop improvement with molecular markers.
      </p>
      <h4>Identification of EST-SSRs</h4>
      <p style="line-height:160%">A total of 2,572 isotig sequences contained at least one EST-SSR, with a frequency
         of one SSR per 17.75 kilobases (Table <a name="d47075e2571"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/table/T3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/table/T3">3</a>). The observed frequencies for di-, tri-, tetra-, penta-, and hexa-repeats were 30.4%,
         52.7%, 2.4%, 7.5% and 6.2%, respectively (Table <a name="d47075e2574"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/table/T4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/table/T4">4</a>). Among the di-nucleotide repeats, the AT/TA motif was the most frequently observed
         (49%) followed by GA/CT (45%). The AC/GT motif was found in low frequency (6%) and
         there were no CG/GC motifs in the <em>Lupinus</em> sequences. Tri-nucleotide repeats, predominantly A/T-rich motifs (74.5%), were the
         most frequent tri-nucleotide repeat found in the <em>Lupinus</em> transcriptome. These tri-nucleotide repeats were often found within the coding sequence
         of putative genes (77.2%). GAA/CTT motif was the most frequent tri-nucleotide repeat
         (31%).
      </p><div class="figs">
      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/table/T3','T3',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/table/T3"><strong>Table 3.</strong></a> <strong>Features of EST-SSRs identified in assembled L1L2</strong><strong><em> L. luteus</em></strong><strong> library</strong></div>
      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/table/T4','T4',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/table/T4"><strong>Table 4.</strong></a> <strong>Distribution of repeat types and number of repeats within the L1L2</strong><strong><em> L. luteus</em></strong><strong> library</strong></div></div>
      <h4>Evaluation of EST-SSRs within yellow lupin and other lupin species</h4>
      <p style="line-height:160%">Studies involving repeat sizes and level of polymorphism have suggested a positive
         correlation between repeat number and rates of polymorphisms, especially in dimeric
         microsatellites <a name="d47075e2862"></a><a name="d47075e2864"></a>[<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>,<a onclick="LoadInParent('#B42'); return false;" href="#B42">42</a>]. Thus, only EST-SSRs containing at least 7 repeat units were selected for validation
         to increase the likelihood of finding markers polymorphic between lupin accessions.
         A total of 783 EST-SSR candidate loci had sufficient repeat units, but only 375 had
         enough repeat flanking sequence to be suitable for primer design. PCR amplification
         of these markers resulted in 222 EST-SSRs (59%) that were polymorphic among the six
         diverse <em>L. luteus</em> included in screening panel. 130 EST-SSRs were monomorphic and 23 primer-pairs failed
         to amplify. A small number (6) of EST-SSRs were validated by Sanger sequencing. The
         amplicon sequences from four different <em>L. luteus</em> genotypes and from <em>L. hispanicus</em> and <em>L. mutabilis</em> confirmed the existence of SSR motifs and their length variability between lupin
         accessions (Figure <a name="d47075e2880"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/figure/F4">4</a>). EST-SSR amplicons showed high conservation at the flanking SSR regions of both
         <em>Lupinus</em> species when compared with <em>L. luteus</em>. However, several indels were observed in adjacent regions and within the SSR motif,
         especially in <em>L. mutabilis</em>. 
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/figure/F4','F4',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/figure/F4"><img align="top" src="/content/figures/1471-2164-13-425-4.gif" alt="thumbnail" class="thumbnail"><strong>Figure 4 .</strong></a> <strong>Alignment of </strong><strong><em>L. luteus</em></strong><strong>, </strong><strong><em>L. hispanicus</em></strong><strong> and </strong><strong><em>L. mutabilis </em></strong><strong>containing several repeat motifs.</strong> (<strong>a</strong>) isotig03739 with GA and AGA motifs; (<strong>b</strong>) isotig16318 with a TAA motif; and (<strong>c</strong>) isotig21236 with a GAA motif.
      </div></div>
      <p style="line-height:160%">Fifty polymorphic EST-SSRs were used to genotype a sample of 64 <em>L. luteus</em> accessions (Table <a name="d47075e2944"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/table/T1">1</a> and Additional file <a name="d47075e2947"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/suppl/S2','',630,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/suppl/S2">2</a>). Twenty-four of these selected markers were specific to L1 (leaf-flower EST library),
         20 EST-SSRs were specific to L2 (seed EST library), and 6 were present in both libraries.
         Neighbor-joining distance analysis detected several clusters among <em>L. luteus</em> accessions, strongly suggesting the existence of population subdivisions (Figure
         <a name="d47075e2953"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/figure/F5">5</a>). However, no clear geographical patterns (country of origin) were observed among
         lupin accessions. Interestingly, Chilean accessions were distributed in most clusters,
         probably reflecting the breeding history of these genotypes. Two hundred and fifty
         four (65.7%) and 113 (30%) SSR primer pairs were able to amplify fragments from <em>L. hispanicus</em> and <em>L. mutabilis</em> DNA, respectively.
      </p><div class="figs">
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/figure/F5','F5',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/figure/F5"><img align="top" src="/content/figures/1471-2164-13-425-5.gif" alt="thumbnail" class="thumbnail"><strong>Figure 5 .</strong></a> <strong>Neighbour Joining tree relating the 64</strong>  <strong><em>L. luteus </em></strong><strong>accessions included in the diversity study.</strong> Numbers above branches correspond to bootstrap values. Accessions are identified
         by a letter L followed by numbers. Letters around accessions identify country of origin
         based on seed bank or breeding histories (RUS: Russia, ISRL: Israel, HUNG: Hungary,
         CHIL: Chile, GER: Germany, SPN: Spain, PORT: Portugal, MORO: Morocco, POL: Poland,
         BYS: Belarus, UKR: Ukraine). The scale is in distance units.
      </div></div>
   </div>
</section>
<section><a name="sec4"></a><h3>Discussion</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">Next-generation sequencing has reduced the existing gap between major crop genomic
         platforms and the limited resources that are currently available for orphan crops
         <a name="d47075e2992"></a>[<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a>]. Complete transcriptome sequencing has generated species specific molecular markers,
         <em>in silico</em> expression analyses, gene discovery, and phylogenetic relationships <a name="d47075e2999"></a><a name="d47075e3001"></a>[<a onclick="LoadInParent('#B43'); return false;" href="#B43">43</a></a>,<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>].
      </p>
      <p style="line-height:160%">In this research, we used 454 cDNA sequences to assemble transcriptomes of two tissues
         (L1 and L2) of yellow lupin. We recovered a large number of previously unknown and
         uncharacterized yellow lupin gene sequences (Table <a name="d47075e3006"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/table/T2">2</a>). The total number of sequences for the combined library was mostly additive from
         L1 and L2. The L1 library favored the inclusion of longer 3’UTR regions, and thus,
         reducing the amount of coding sequences needed to assemble longer combined contigs
         (L1L2). As a consequence, two or more sequences belonging to the same transcript may
         not be assembled together, causing an overestimation of expressed sequences. The larger
         amount of 3’UTR regions for L1 is also in agreement with the lower GC content, condition
         typically associated with untranslated regions <a name="d47075e3010"></a><a name="d47075e3012"></a>[<a onclick="LoadInParent('#B45'); return false;" href="#B45">45</a></a>,<a onclick="LoadInParent('#B46'); return false;" href="#B46">46</a>]. Undoubtedly, a number of expressed sequences are tissue specific and will not assemble
         into combined contigs. For instance, several genes related to seed dormancy and germination
         are not expressed in vegetative and floral tissues <a name="d47075e3016"></a><a name="d47075e3018"></a>[<a onclick="LoadInParent('#B47'); return false;" href="#B47">47</a></a>,<a onclick="LoadInParent('#B48'); return false;" href="#B48">48</a>]. The same specificity was observed in a number of tissues and plant species <a name="d47075e3022"></a><a name="d47075e3024"></a><a name="d47075e3026"></a>[<a onclick="LoadInParent('#B49'); return false;" href="#B49">49</a>-<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>]. The assembly of L1L2 generated 55,309 isotigs of which 30,811 had similarity to
         putative proteins found in other plant species. Comparative studies carried out against
         <em>L. japonicus, M. truncatula</em> and <em>G. max</em> showed a total of 31,520 lupin sequences similar to at least one of the model legume
         databases and 22,219 were similar to all of them. <em>Lotus</em> and <em>Medicago</em> belong to the Galegoid subclade, which includes mostly temperate legume species <a name="d47075e3043"></a>[<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>]. <em>Glycine</em> is a member of the Phaseoloid subclade which comprises mostly tropical species <a name="d47075e3050"></a>[<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>]. Lupins belong to the Genistoid subclade, which is sister (and distant) to most of
         the described Papilionoid subclades; especially those containing most domesticated
         species <a name="d47075e3055"></a>[<a onclick="LoadInParent('#B53'); return false;" href="#B53">53</a>].
      </p>
      <p style="line-height:160%">Although micro-repeat motifs are frequent in plant genomes and their respective transcriptomes,
         the frequency of SSR discovery depends on the search criteria <a name="d47075e3061"></a><a name="d47075e3063"></a><a name="d47075e3065"></a><a name="d47075e3067"></a>[<a onclick="LoadInParent('#B42'); return false;" href="#B42">42</a>,<a onclick="LoadInParent('#B54'); return false;" href="#B54">54</a>-<a onclick="LoadInParent('#B56'); return false;" href="#B56">56</a>]. We analyzed 55,309 lupin isotig sequences using MISA and identified 2,796 SSR motifs
         with an average frequency of one SSR per 17.75 kbp. Tri-nucleotide repeats were the
         motifs most frequently found in <em>L. luteus</em> expressed sequences. Similar results have been reported in numerous plant species
         <a name="d47075e3074"></a><a name="d47075e3076"></a><a name="d47075e3078"></a><a name="d47075e3080"></a><a name="d47075e3082"></a>[<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a>,<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>,<a onclick="LoadInParent('#B54'); return false;" href="#B54">54</a></a>,<a onclick="LoadInParent('#B55'); return false;" href="#B55">55</a>,<a onclick="LoadInParent('#B57'); return false;" href="#B57">57</a>]. The abundance of trimeric EST-SSRs has been attributed to the absence of frameshift
         mutations when there is length variation in these SSRs <a name="d47075e3086"></a>[<a onclick="LoadInParent('#B58'); return false;" href="#B58">58</a>]. Indeed, 1,435 EST-SSRs were discovered within coding regions of the gene. Among
         tri-nucleotide repeats, AT-rich motifs were the most predominant ones (74.5%), which
         have also been observed in soybean, Citrus and Arabidopsis <a name="d47075e3090"></a><a name="d47075e3092"></a>[<a onclick="LoadInParent('#B54'); return false;" href="#B54">54</a>,<a onclick="LoadInParent('#B57'); return false;" href="#B57">57</a>]. For di-nucleotide repeats, AT was the most frequently observed motif, contrasting
         with results from Arabidopsis, soybean, maize, rice, wheat and barley where AC/GT
         were the most frequent repeats <a name="d47075e3097"></a><a name="d47075e3099"></a><a name="d47075e3101"></a><a name="d47075e3103"></a><a name="d47075e3105"></a>[<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a>,<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>,<a onclick="LoadInParent('#B54'); return false;" href="#B54">54</a></a>,<a onclick="LoadInParent('#B55'); return false;" href="#B55">55</a>,<a onclick="LoadInParent('#B57'); return false;" href="#B57">57</a>]. The high proportion of untranslated sequences (specifically 3’UTR), mainly contributed
         from the L1, could explain the bias toward A/T-rich repeat sequences observed in yellow
         lupin. There were no CG repeats in the lupin sequences, similar to results obtained
         in barrel medic <a name="d47075e3109"></a>[<a onclick="LoadInParent('#B24'); return false;" href="#B24">24</a>], rice, corn, soybean <a name="d47075e3113"></a>[<a onclick="LoadInParent('#B57'); return false;" href="#B57">57</a>], wheat <a name="d47075e3117"></a>[<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>], Sorghum <a name="d47075e3121"></a>[<a onclick="LoadInParent('#B25'); return false;" href="#B25">25</a>], Arabidopsis, apricot and peach <a name="d47075e3125"></a>[<a onclick="LoadInParent('#B59'); return false;" href="#B59">59</a>].
      </p>
      <p style="line-height:160%">We used GBrowse to visualize lupin ESTs aligned to the <em>M. truncatula</em> chromosomes (Figure <a name="d47075e3133"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/figure/F3">3</a>). This approach potentially identifies paralogs sequences and allows color-coded
         alignment by BLAST significance <a name="d47075e3137"></a>[<a onclick="LoadInParent('#B60'); return false;" href="#B60">60</a>]. A total of 25,400 <em>L. luteus</em> contigs were localized and found to be distributed across the entire Medicago genome
         with chromosomes Mt1 and Mt3 having the highest number of gene matches. Each yellow
         lupin sequence was mapped to an average of 3.7 locations, which may correspond in
         part to rounds of genome duplications previously described for the Medicago genome
         <a name="d47075e3144"></a>[<a onclick="LoadInParent('#B61'); return false;" href="#B61">61</a>]. Understanding syntenic relationships among species is essential to exploit the available
         tools developed for comparative genomic analysis. Using this approach, we created
         a new method of developing molecular markers, markers that are based on conserved
         microsynteny (CMS) between orphan and model species. Genome comparisons among <em>M. truncatula</em>, <em>G. max</em> and <em>L. japonicus</em> have shown that, in general, most genes in Papilionoid legume species are likely
         to be found within a relatively long syntenic region of any other Papilioniod species
         <a name="d47075e3158"></a>[<a onclick="LoadInParent('#B62'); return false;" href="#B62">62</a>]. Positive amplification and sequencing of <em>L. luteus</em> intergenic regions, based on PCR primers located on <em>M.</em> t<em>runcatula</em> adjacent genes, suggested the existence of microscale synteny between these legume
         species. Roughly 40% of the targeted intergenic <em>L. luteus</em> regions amplified, points out the usefulness of conserved legume chromosome blocks
         for genomic studies of orphan crops. Although some primer pairs failed to amplify,
         poor amplification could be a consequence of non-synteny, but also other technical
         limitations could also explain negative PCR results. For instance it is known that
         non-coding DNA regions are highly variable among species <a name="d47075e3175"></a><a name="d47075e3177"></a>[<a onclick="LoadInParent('#B63'); return false;" href="#B63">63</a></a>,<a onclick="LoadInParent('#B64'); return false;" href="#B64">64</a>], and negative PCR amplifications could easily due to excessively long <em>L. luteus</em> intergenic regions.
      </p>
      <p style="line-height:160%">Few studies have reported the use of EST-SSRs in <em>Lupinus</em> species <a name="d47075e3189"></a><a name="d47075e3191"></a><a name="d47075e3193"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a></a>,<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>]. Most efforts have focused on genetic linkage mapping and in diversity studies in
         <em>L. angustifolius</em><a name="d47075e3199"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>], <em>L. albus</em><a name="d47075e3205"></a>[<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a>] and <em>L. luteus</em><a name="d47075e3212"></a>[<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>]<em>.</em> To validate our <em>L. luteus</em> polymorphic markers we tested 50 EST-SSRs on a population of 64 genotypes of <em>L. luteus</em>. An analysis of genotypic diversity illustrated the existence of several clusters
         within <em>L. luteus</em> germplasm. The lack of a clear pattern following the geographical accession origin
         (country) could be explained by three reasons. 1) The number of accessions may not
         have been large enough to allow a clear pattern to emerge. 2) <em>L. luteus</em> is widely distributed across the Mediterranean region, mainly due to human introductions
         <a name="d47075e3231"></a>[<a onclick="LoadInParent('#B6'); return false;" href="#B6">6</a>]. This situation could have homogenized natural genetic distinctiveness, leaving mostly
         population subdivisions based on breeding histories. 3) Finally, it is possible some
         accessions could have been misclassified; and thus, obscuring an existing geographical
         clustering pattern.
      </p>
      <p style="line-height:160%">We observed that a number of high yellow lupin EST-SSR amplified fragments in two
         other lupin species, <em>L. hispanicus</em> and <em>L. mutabilis</em> (Table <a name="d47075e3242"></a><a onclick="popup('http://www.biomedcentral.com/1471-2164/13/425/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2164/13/425/table/T1">1</a>). The high number of transferable markers between <em>L. luteus</em> and <em>L. hispanicus</em> confirmed their closer genetic relationship <a name="d47075e3253"></a><a name="d47075e3255"></a>[<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a>,<a onclick="LoadInParent('#B65'); return false;" href="#B65">65</a>] than <em>L. luteus and L. mutabilis</em>. The two closely related species have the same chromosome number (2n = 52) and are
         still interfertile, generating a natural hybrid called <em>hispanicoluteus</em><a name="d47075e3264"></a>[<a onclick="LoadInParent('#B66'); return false;" href="#B66">66</a>]. Phylogenetic studies have placed new and old world lupins into two different clades
         <a name="d47075e3268"></a><a name="d47075e3270"></a><a name="d47075e3272"></a>[<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a>,<a onclick="LoadInParent('#B65'); return false;" href="#B65">65</a>,<a onclick="LoadInParent('#B67'); return false;" href="#B67">67</a>]. Thus, most EST-SSRs amplified in <em>L. mutabilis</em> (2n = 48), the only cultivated new world lupin <a name="d47075e3280"></a>[<a onclick="LoadInParent('#B65'); return false;" href="#B65">65</a>], should have high transferability rates to other lupin species, such as <em>L. albus</em> and <em>L. angustifolius.</em> The understanding of the genetic diversity among other close relative lupin species
         will facilitate the transfer of favorable variation into cultivated species. For instance,
         <em>L. hispanicus</em> has been suggested as a reservoir of favorable variation for a number of biotic and
         abiotic stresses currently affecting <em>L. luteus</em><a name="d47075e3295"></a><a name="d47075e3297"></a>[<a onclick="LoadInParent('#B68'); return false;" href="#B68">68</a></a>,<a onclick="LoadInParent('#B69'); return false;" href="#B69">69</a>].
      </p>
   </div>
</section>
<section><a name="sec5"></a><h3>Conclusion</h3>
   <div class="collapsible-content">
      <p style="line-height:160%"><em>L. luteus</em> deep transcriptome sequencing will facilitate the further development of genomic
         tools and lupin germplasm. Massive sequencing of cDNA libraries will continue to produce
         raw materials for gene discoveries, identification of polymorphisms (SNPs, EST-SSRs,
         INDELs, etc.) for marker development, anchoring sequences for genome comparison studies
         and putative gene candidates for QTL detection. We are also exploiting the microsyntenic
         regions observed among <em>L. luteus</em> and legume model species to saturate yellow lupin linkage maps by amplifying conserved
         regions across legume species. The utilization of these tools will allow transforming
         <em>L. luteus</em> into a valid temperate legume crop alternative.
      </p>
   </div>
</section>
<section><a name="sec6"></a><h3>Competing interests</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">The authors declare that they have no competing interests.</p>
   </div>
</section>
<section><a name="sec7"></a><h3>Authors’ contributions</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">LBP collected the tissues and extracted the RNAs. JU, LBP and JM constructed the EST
         libraries. JU and JM supervised the 454 sequencing of the libraries. LBP and JU conducted
         the SSR search and primer design. LBP conducted the SSR polymorphism tests and transferability
         studies. LMP sequenced the amplicons of SSR and intergenic blocks. GAA grew the plants
         for the diversity study, extracted the DNAs, PCR amplified and conducted the genotyping
         of the population. IMB drafted the experimental design of all the studies carried
         out in this work and conducted the genetic analysis for the diversity study. CSN conducted
         annotations and <em>in silico</em> mapping of the sequences. LBP conducted the microscale synteny studies. HS and IMB
         conceived the study. LBP drafted the manuscript with the support of IMB. All the authors
         read and approved the final manuscript.
      </p>
   </div>
</section>
<section><a name="ack"></a><h3>Acknowledgments</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">This research was funded by the National Commission for Scientific &amp; Technological
         Research (FONDECYT Project No.1090759) and CONICYT Regional/GORE Araucanía/CGNA/R10C1001,
         Chile. We thank Héctor Urbina for his assistance on <em>L. luteus</em> sequence assemblies.
      </p>
   </div>
</section>
<section><a name="refs"></a><h3>References</h3>
   <div id="article-references" class="collapsible-content">
      <ol id="references">
         <li id="B1">
            <p><a name="B1"></a> Mace ES,  Varshney RK,  Mahalakshmi V,  Seetha K,  Gafoor A,  Leeladevi Y,  Crouch JH: <strong> In silico development of simple sequence repeat markers within the aeschynomenoid/dalbergoid
                  and genistoid clades of the Leguminosae family and their transferability to Arachis
                  hypogaea, groundnut. </strong><p><em>Plant Sci</em> 2008,  <strong>174</strong><strong>:</strong>51-60.   <a target="_blank" href="http://dx.doi.org/10.1016/j.plantsci.2007.09.014">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B1" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B1','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B2">
            <p><a name="B2"></a> Gladstones JS: <strong> Distribution, origin, taxonomy, history and importance. </strong>In <em>Lupins as crop plants: biology, production and utilization</em>.  Edited by Gladstones JS, Atkins CA, Hamblin J.  CAB International, Cambridge, United Kingdom;  1998:1-37.  <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B2" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B2','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B3">
            <p><a name="B3"></a> Múzquiz M,  Burbano C,  Gorospe MJ,  Ródenas I: <strong> A chemical study of Lupinus hispanicus seed—toxic and antinutritional components. </strong><p><em>J Scie Food Agr</em> 1989,  <strong>47</strong><strong>:</strong>205-214.   <a target="_blank" href="http://dx.doi.org/10.1002/jsfa.2740470208">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B3" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B3','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B4">
            <p><a name="B4"></a> Wolko B,  Weeden NF: <strong> Estimation of Lupinus genome polyploidy on the basis of isozymic loci number. </strong><p><em>Genet Pol</em> 1989,  <strong>30</strong><strong>:</strong>165-170.  <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B4" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B4','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B5">
            <p><a name="B5"></a> Naganowska B,  Wolko B,  Sliwinska E,  Kaczmarek Z: <strong> Nuclear DNA content variation and species relationships in the genus Lupinus (Fabaceae). </strong><p><em>Ann Bot</em> 2003,  <strong>92</strong><strong>:</strong>349-355.  <a target="_blank" href="/pubmed/12853281">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12853281">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B5" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B5','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B6">
            <p><a name="B6"></a> Petterson DS: <strong> Composition and food uses of lupin. </strong>In <em>Lupins as a crop plants: biology, production and utilization</em>.  Edited by Gladstones JS, Atkins CA, Hamblin J.  CAB International, Wallingford, UK;  1998:353-384.  <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B6" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B6','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B7">
            <p><a name="B7"></a> Gladstones JS: <em>Lupins of the Mediterranean Region and Africa. Technical Bulletin No. 26</em>.  Western Australian Department of Agriculture, Western Australia;  1974:43-48.  <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B7" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B7','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B8">
            <p><a name="B8"></a> Glencross BD,  Carter CG,  Duijster N,  Evans DR,  Dods K,  McCafferty P,  Hawkins WE,  Maas R,  Sipsas S: <strong> A comparison of the digestibility of a range of lupin and soybean protein products
                  when fed to either Atlantic salmon (Salmo salar) or rainbow trout (Oncorhynchus mykiss). </strong><p><em>Aquaculture</em> 2004,  <strong>237</strong><strong>:</strong>333-346.   <a target="_blank" href="http://dx.doi.org/10.1016/j.aquaculture.2004.03.023">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B8" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B8','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B9">
            <p><a name="B9"></a> Berville ABS,  Heinanen J,  Kartuzova LT,  Bernatskaya ML,  Chmeleva ZV: <strong> Diversity of lupin (Lupinus L.) based on biochemical composition. </strong><p><em>Plant Genet Res Newsletter</em> 2003,  <strong>134</strong><strong>:</strong>42-57.  <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B9" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B9','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B10">
            <p><a name="B10"></a> Varshney RK,  Close TJ,  Singh NK,  Hoisington DA,  Cook DR: <strong> Orphan legume crops enter the genomics era! </strong><p><em>Curr Opin Plant Biol</em> 2009,  <strong>12</strong><strong>:</strong>202-210.  <a target="_blank" href="/pubmed/19157958">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19157958">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B10" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B10','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B11">
            <p><a name="B11"></a> Kaur S,  Cogan N,  Pembleton LW,  Shinozuka M,  Savin KW,  Materne M,  Forster JW: <strong> Transcriptome sequencing of lentil based on second-generation technology permits large-scale
                  unigene assembly and SSR marker discovery. </strong><p><em>BMC Genomics</em> 2011,  <strong>12</strong><strong>:</strong>265.  <a target="_blank" href="/pubmed/21609489">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2164-12-265">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21609489">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B11" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B11','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B12">
            <p><a name="B12"></a> Hiremath PJ,  Farmer A,  Cannon SB,  Woodward J,  Kudapa H,  Tuteja R,  Kumar A,  Bhanuprakash A,  Mulaosmanovic B,  Gujaria N,  Krishnamurthy L,  Gaur PM,  Kavikishor PB,  Shah T,  Srinivasan R,  Lohse M,  Xiao Y,  Town CD,  Cook DR,  May GD,  Varshney RK: <strong> Large-scale transcriptome analysis in chickpea (Cicer arietinum L.), an orphan legume
                  crop of the semi-arid tropics of Asia and Africa. </strong><p><em>Plant Biotechnol J</em> 2011,  <strong>9</strong><strong>:</strong>922-931.  <a target="_blank" href="/pubmed/21615673">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21615673">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21615673">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B12" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B12','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B13">
            <p><a name="B13"></a> Cheung F,  Haas BJ,  Goldberg SMD,  May GD,  Xiao Y,  Town CD: <strong> Sequencing Medicago truncatula expressed tags using 454 Life Science technology. </strong><p><em>BMC Genomics</em> 2006,  <strong>7</strong><strong>:</strong>272.  <a target="_blank" href="/pubmed/17062153">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2164-7-272">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17062153">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B13" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B13','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B14">
            <p><a name="B14"></a> Weber APM,  Weber KL,  Carr K,  Wilkerson C,  Ohlrogge JB: <strong> Sampling the Arabidopsis transcriptome with massively parallel pyrosequencing. </strong><p><em>Plant Physiol</em> 2007,  <strong>144</strong><strong>:</strong>32-42.  <a target="_blank" href="/pubmed/17351049">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17351049">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17351049">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B14" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B14','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B15">
            <p><a name="B15"></a> Yu JK,  La Rota M,  Kantety RV,  Sorrells ME: <strong> EST derived SSR markers for comparative mapping in wheat and rice. </strong><p><em>Mol Genet Genomics</em> 2004,  <strong>271</strong><strong>:</strong>742-751.  <a target="_blank" href="/pubmed/15197579">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15197579">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B15" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B15','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B16">
            <p><a name="B16"></a> Rexroad CE III,  Rodriguez MF,  Coulibaly I,  Gharbi K,  Danzmann RG,  DeKoning J,  Phillips R,  Palti Y: <strong> Comparative mapping of expressed sequence tags containing microsatellites in rainbow
                  trout (Oncorhynchus mykiss). </strong><p><em>BMC Genomics</em> 2005,  <strong>6</strong><strong>:</strong>54.  <a target="_blank" href="/pubmed/15836796">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2164-6-54">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15836796">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B16" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B16','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B17">
            <p><a name="B17"></a> Young ND,  Udvardi M: <strong> Translating Medicago truncatula genomics to crop legumes. </strong><p><em>Curr Opin Plant Biol</em> 2009,  <strong>12</strong><strong>:</strong>193-201.  <a target="_blank" href="/pubmed/19162532">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19162532">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B17" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B17','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B18">
            <p><a name="B18"></a> Aubert G,  Morin J,  Jacquin F,  Loridon K,  Quillet MC,  Petit A,  Rameau C,  Lejeune-Henaut I,  Huguet T,  Bursting J: <strong> Functional mapping in pea, as an aid to the candidate gene selection and for investigating
                  synteny with the model legume Medicago truncatula. </strong><p><em>Theor Appl Genet</em> 2006,  <strong>112</strong><strong>:</strong>1024-1041.  <a target="_blank" href="/pubmed/16416153">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16416153">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B18" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B18','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B19">
            <p><a name="B19"></a> Phan HT,  Ellwood SR,  Adhikari K,  Nelson MN,  Oliver RP,  Phan HT,  Ellwood SR,  Adhikari K,  Nelson MN,  Oliver RP: <strong> The first genetic and comparative map of white lupin (Lupinus albus L.): Identification
                  of QTLs for anthracnose resistance and flowering time, and a locus for alkaloid content. </strong><p><em>DNA Res</em> 2007,  <strong>14</strong><strong>:</strong>59-70.  <a target="_blank" href="/pubmed/17526914">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17526914">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17526914">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B19" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B19','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B20">
            <p><a name="B20"></a> Nelson MN,  Moolhuijzen PM,  Boersma JG,  Chudy M,  Lesniewska K,  Bellgard M,  Oliver RP,  Swiecicki W,  Wolko B,  Cowling WA,  Ellwood SR: <strong> Aligning a New Reference Genetic Map of Lupinus angustifolius with the Genome Sequence
                  of the Model Legume, Lotus japonicus. </strong><p><em>DNA Res</em> 2010,  <strong>17</strong><strong>:</strong>73-83.  <a target="_blank" href="/pubmed/20133394">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20133394">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=20133394">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B20" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B20','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B21">
            <p><a name="B21"></a> Nelson M,  Huyen P,  Ellwood S,  Moolhuijzen P,  Hane J,  Williams A,  O’Lone C,  Fosu-Nyarko J,  Scobie M,  Cakir M,  Jones M,  Bellgard M,  Ksiazkiewicz M,  Wolko B,  Barker S,  Oliver R,  Cowling W: <strong> The first gene-based map of Lupinus angustifolius L.-location of domestication genes
                  and conserved synteny with Medicago truncatula. </strong><p><em>Theor Appl Genet</em> 2006,  <strong>113</strong><strong>:</strong>225-238.  <a target="_blank" href="/pubmed/16791689">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16791689">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B21" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B21','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B22">
            <p><a name="B22"></a> Parra Gonzalez L,  Straub S,  Doyle J,  Mora Ortega P,  Salvo Garrido H,  Maureira Butler I: <strong> Development of microsatellites inLupinus luteus(Fabaceae) and cross-species amplification
                  in other lupine species. </strong><p><em>Am J Bot</em> 2010,  <strong>97</strong><strong>:</strong>e72-e74.  <a target="_blank" href="/pubmed/21616875">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21616875">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B22" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B22','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B23">
            <p><a name="B23"></a> Gao LL,  Hane JK,  Kamphuis LG,  Foley R,  Shi BJ,  Atkins CA,  Singh KB: <strong> Development of genomic resources for the narrow-leafed lupin (Lupinus angustifolius):
                  construction of a bacterial artificial chromosome (BAC) library and BAC-end sequencing. </strong><p><em>BMC Genomics</em> 2011,  <strong>12</strong><strong>:</strong>521.  <a target="_blank" href="/pubmed/22014081">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2164-12-521">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=22014081">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B23" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B23','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B24">
            <p><a name="B24"></a> Gupta S,  Prasad M: <strong> Development and characterization of genic SSR markers in Medicago truncatula and their
                  transferability in leguminous and non-leguminous species. </strong><p><em>Genome</em> 2009,  <strong>52</strong><strong>:</strong>761-771.  <a target="_blank" href="/pubmed/19935924">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19935924">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B24" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B24','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B25">
            <p><a name="B25"></a> Kantety RM,  Mathews DE,  Sorrels ME: <strong> Data mining for simple sequence repeats in expressed sequence tags from barley, maize,
                  rice, sorghum and wheat. </strong><p><em>Plant Mol Biol</em> 2002,  <strong>148</strong><strong>:</strong>501-510.  <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B25" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B25','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B26">
            <p><a name="B26"></a> La Rota RM,  Kantety RV,  Yu JK,  Sorrells ME: <strong> Nonrandom distribution and frequencies of genomic and EST-derived microsatellite markers
                  in rice, wheat, and barley. </strong><p><em>BMC Genomics</em> 2005,  <strong>6</strong><strong>:</strong>23-35.  <a target="_blank" href="/pubmed/15720707">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2164-6-23">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15720707">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B26" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B26','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B27">
            <p><a name="B27"></a> Nicot N,  Chiquet VB,  Amilhat L,  Legeai F,  Leroy P,  Bernard M,  Sourdille P: <strong> Study of simple sequence repeat (SSR) markers from wheat expressed sequence tags (ESTs). </strong><p><em>Theor Appl Genet</em> 2004,  <strong>109</strong><strong>:</strong>800-805.  <a target="_blank" href="/pubmed/15146317">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15146317">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B27" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B27','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B28">
            <p><a name="B28"></a> Thiel T,  Michalek W,  Varshney RK,  Graner A: <strong> Exploiting EST databases for the development and characterization of gene-derived
                  SSR-markers in barley (Hordeum vulgare L.). </strong><p><em>Theor Appl Genet</em> 2003,  <strong>106</strong><strong>:</strong>411-422.  <a target="_blank" href="/pubmed/12589540">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12589540">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B28" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B28','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B29">
            <p><a name="B29"></a> Domon E,  Fuijita M,  Ishikawa N: <strong> The insertion/deletion polymorphisms in the waxy gene of barley genetic resources
                  from East Asia. </strong><p><em>Theor Appl Genet</em> 2002,  <strong>104</strong><strong>:</strong>132-138.  <a target="_blank" href="/pubmed/12579438">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12579438">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B29" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B29','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B30">
            <p><a name="B30"></a> Blair M,  McCouch SR: <strong> Microsatellite and sequence-tagged site markers diagnostic for the rice bacterial
                  leaf blight resistance gene xa-5. </strong><p><em>Theor Appl Genet</em> 1997,  <strong>95</strong><strong>:</strong>174-184.   <a target="_blank" href="http://dx.doi.org/10.1007/s001220050545">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B30" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B30','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B31">
            <p><a name="B31"></a> Ayres NM,  McClung AM,  Larkin PD,  Bligh HFJ,  Jones CA,  Park WD: <strong> Microsatellites and a single nucleotide polymorphism differenciate apparent amylase
                  classes in an extended pedigree of US rice germplasm. </strong><p><em>Theor Appl Genet</em> 1997,  <strong>94</strong><strong>:</strong>773-781.   <a target="_blank" href="http://dx.doi.org/10.1007/s001220050477">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B31" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B31','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B32">
            <p><a name="B32"></a> Prathepha P: <strong> Characterization of Waxy microsatellite classes that are closely linked to the rice
                  Waxy gene and amylose content in Thai rice germplasm Songklanakarin. </strong><p><em>J Sci Technol</em> 2003,  <strong>25</strong><strong>:</strong>1-8.  <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B32" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B32','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B33">
            <p><a name="B33"></a> Selvaraj I,  Nagarajan P,  Thiyagarajan K,  Bharathi M,  Rabindran R: <strong> Identification of Microsatellite (SSR) and RAPD Markers Linked to Rice Blast Disease
                  Resistance gene in Rice (Oryza sativa L.). </strong><p><em>African J Biotechnol</em> 2011,  <strong>10</strong><strong>:</strong>3301-3321.  <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B33" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B33','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B34">
            <p><a name="B34"></a> Zhou WC,  Kolb FL,  Bai GH,  Domier LL,  Boze LK,  Smith NJ: <strong> Validation of a major QTL for scab resistance with SSR markers and use of marker-assisted
                  selection in wheat. </strong><p><em>Plant Breeding</em> 2003,  <strong>122</strong><strong>:</strong>40-46.   <a target="_blank" href="http://dx.doi.org/10.1046/j.1439-0523.2003.00802.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B34" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B34','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B35">
            <p><a name="B35"></a> Abdurakhmonov IY,  Abdullaev AA,  Saha S,  Buriev ZT,  Arslanov D,  Kuryazov Z,  Mavlonov GT,  Rizaeva SM,  Reddy UK,  Jenkins JN,  Abdullaev A,  Abdukarimov A: <strong> Simple Sequence Repeat Marker Associated with a Natural Leaf Defoliation Trait in
                  Tetraploid Cotton. </strong><p><em>J Hered</em> 2005,  <strong>96</strong><strong>:</strong>644-653.  <a target="_blank" href="/pubmed/16159909">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16159909">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B35" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B35','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B36">
            <p><a name="B36"></a> Zeng L,  Meredith WR Jr,  Gutiérrez OA,  Boykin DL: <strong> Identification of associations between SSR markers and fiber traits in an exotic germplasm
                  derived from multiple crosses among Gossypium tetraploid species. </strong><p><em>Theor Appl Genet</em> 2009,  <strong>119</strong><strong>:</strong>93-103.  <a target="_blank" href="/pubmed/19360391">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19360391">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B36" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B36','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B37">
            <p><a name="B37"></a> Li X,  Yang H,  Buirchell B,  Yan G: <strong> Development of a DNA marker tightly linked to low-alkaloid gene iucundus in narrow-leafed
                  lupin (Lupinus angustifolius L.) for marker-assisted selection. </strong><p><em>Crop and Pasture Sci</em> 2011,  <strong>62</strong><strong>:</strong>218-224.   <a target="_blank" href="http://dx.doi.org/10.1071/CP10352">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B37" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B37','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B38">
            <p><a name="B38"></a> Logeman J,  Schell J,  Willmitzer L: <strong> Improved method for the isolation of RNA from plant tissues. </strong><p><em>Anal Biochem</em> 1987,  <strong>163</strong><strong>:</strong>16-20.  <a target="_blank" href="/pubmed/2441623">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=2441623">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B38" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B38','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B39">
            <p><a name="B39"></a> Meyer E,  Aglyamova GV,  Wang S,  Buchanan-Carter J,  Abrego D,  Colbourne JK,  Willis BL,  Matz MV: <strong> Sequencing and de novo analysis of a coral larval transcriptome using 454 GSFlx. </strong><p><em>BMC Genomics</em> 2009,  <strong>10</strong><strong>:</strong>219.  <a target="_blank" href="/pubmed/19435504">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2164-10-219">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19435504">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B39" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B39','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B40">
            <p><a name="B40"></a> Podicheti R,  Gollapudi R,  Dong Q: <strong> WebGBrowse: a web server for GBrowse. </strong><p><em>Bioinformatics</em> 2009,  <strong>25</strong><strong>:</strong>1550-1551.  <a target="_blank" href="/pubmed/19357095">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19357095">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B40" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B40','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B41">
            <p><a name="B41"></a> Doyle JJ,  Doyle JL: <strong> A rapid DNA isolation procedure for small quantities of fresh leaf tissue. </strong><p><em>Phytochem Bull</em> 1978,  <strong>19</strong><strong>:</strong>11-15.  <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B41" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B41','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B42">
            <p><a name="B42"></a> Temnykh S,  DeClerck G,  Lukashova A,  Lipovich L,  Cartinhour S,  McCouch S: <strong> Computational and experimental analysis of microsatellites in rice (Oryza sativaL.):
                  Frequency, length variation, transposon associations, and genetic marker potential. </strong><p><em>Genome Res</em> 2001,  <strong>11</strong><strong>:</strong>1441-1452.  <a target="_blank" href="/pubmed/11483586">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11483586">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11483586">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B42" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B42','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B43">
            <p><a name="B43"></a> Edwards D,  Batley J: <strong> Plant genome sequencing: applications for crop improvement. </strong><p><em>Plant Biotech J</em> 2010,  <strong>8</strong><strong>:</strong>2-9.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1467-7652.2009.00459.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B43" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B43','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B44">
            <p><a name="B44"></a> Lister R,  Gregory BD,  Ecker JR: <strong> Next is now: new technologies for sequencing of genomes, transcriptomes, and beyond. </strong><p><em>Curr Opin Plant Biol</em> 2009,  <strong>12</strong><strong>:</strong>107-118.  <a target="_blank" href="/pubmed/19157957">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19157957">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19157957">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B44" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B44','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B45">
            <p><a name="B45"></a> Zhang L,  Kasif S,  Cantor CR,  Broude NE: <strong> GC_AT-content spikes as genomic punctuation marks. </strong><p><em>Proc Natl Acad Sci</em> 2004,  <strong>101</strong><strong>:</strong>16855-16860.  <a target="_blank" href="/pubmed/15548610">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15548610">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15548610">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B45" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B45','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B46">
            <p><a name="B46"></a> Pesole G,  Liuni S,  Grillo G,  Licciulli F,  Mignone F,  Gissi C,  Saccone C: <strong> UTRdb and UTRsite: specialized database of sequences and functional elements of 5’
                  and 3’ unstranslated regions of eukaryotic mRNAs. Update 2002. </strong><p><em>Nucleic Acids Res</em> 2002,  <strong>30</strong><strong>:</strong>335-340.  <a target="_blank" href="/pubmed/11752330">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11752330">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11752330">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B46" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B46','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B47">
            <p><a name="B47"></a> Diaz I,  Vicente-Carbajosa J,  Abraham Z,  Martinez M,  Isabel-La Moneda I,  Carbonero P: <strong> The GAMYB protein from barley interacts with the DOF transcription factor BPBF and
                  activates endosperm-specific genes during seed development. </strong><p><em>Plant J</em> 2002,  <strong>29</strong><strong>:</strong>453-464.  <a target="_blank" href="/pubmed/11846878">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11846878">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B47" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B47','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B48">
            <p><a name="B48"></a> Koornneff M,  Bentsink L,  Hilhorst H: <strong> Seed dormancy and germination. </strong><p><em>Curr Opin Plant Biol</em> 2002,  <strong>5</strong><strong>:</strong>33-36.  <a target="_blank" href="/pubmed/11788305">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11788305">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B48" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B48','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B49">
            <p><a name="B49"></a> Tillett RL,  Ergül A,  Albion RL,  Schlauch KA,  Cramer GR,  Cushman JC: <strong> Identification of tissue-specific, abiotic stress responsive gene expression patterns
                  in wine grape (Vitis vinifera L.) based on curation and mining of large-scale EST
                  data sets. </strong><p><em>BMC Plant Biology</em> 2011,  <strong>11</strong><strong>:</strong>86.  <a target="_blank" href="/pubmed/21592389">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2229-11-86">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21592389">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B49" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B49','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B50">
            <p><a name="B50"></a> Covitz PA,  Smith LS,  Long SR: <strong> Expressed Sequence Tags from a Root-Hair-Enriched Medicago truncatula cDNA Library. </strong><p><em>Plant Physiol</em> 1998,  <strong>117</strong><strong>:</strong>1325-1332.  <a target="_blank" href="/pubmed/9701588">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9701588">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=9701588">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B50" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B50','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B51">
            <p><a name="B51"></a> Yamada S,  Katsuhara M,  Kelly WB,  Michalowski CB,  Bohnert HJ: <strong> A family of transcripts encoding water channel proteins: Tissue-specific expression
                  in the common ice plant. </strong><p><em>Plant Cell</em> 1995,  <strong>7</strong><strong>:</strong>1129-1142.  <a target="_blank" href="/pubmed/7549476">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=7549476">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=7549476">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B51" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B51','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B52">
            <p><a name="B52"></a> Bertioli DJ,  Moretzsohn MC,  Madsen LH,  Sandal N,  Leal-Bertioli S,  Guimarães PM,  Hougaard BK,  Fredslund J,  Schauser L,  Nielsen AM,  Sato S,  Tabata S,  Cannon SB,  Stougaard J: <strong> An analysis of synteny of Arachis with Lotus and Medicago sheds new light on the structure,
                  stability and evolution of legume genomes. </strong><p><em>BMC Genomics</em> 2009,  <strong>10</strong><strong>:</strong>45.  <a target="_blank" href="/pubmed/19166586">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2164-10-45">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19166586">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B52" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B52','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B53">
            <p><a name="B53"></a> Wojciechowski MF,  Lavin M,  Sanderson MJ: <strong> A phylogeny of legumes (leguminosae) based on analysis of the plastid MATK gene resolves
                  many well-supported subclades within the family. </strong><p><em>Am J Bot</em> 2004,  <strong>91</strong><strong>:</strong>1846-1862.  <a target="_blank" href="/pubmed/21652332">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21652332">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B53" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B53','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B54">
            <p><a name="B54"></a> Cardle I,  Ramsay L,  Milbourne D,  Macaulay M,  Marshall D,  Waugh R: <strong> Computational and experimental characterization of physically clustered simple sequence
                  repeats in plants. </strong><p><em>Genetics</em> 2000,  <strong>156</strong><strong>:</strong>847-854.  <a target="_blank" href="/pubmed/11014830">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11014830">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11014830">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B54" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B54','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B55">
            <p><a name="B55"></a> Varshney RK,  Thiel T,  Stein N,  Langridge P,  Graner A: <strong> In silico analysis on frequency and distribution of microsatellites in ESTs of some
                  cereals species. </strong><p><em>Cell Mol Biol Lett</em> 2002,  <strong>7</strong><strong>:</strong>537-546.  <a target="_blank" href="/pubmed/12378259">PubMed&nbsp;Abstract</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B55" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B55','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B56">
            <p><a name="B56"></a> Varshney RK,  Graner A,  Sorrels ME: <strong> Genic microsatellite markers in plants: features an applications. </strong><p><em>Trends Biotechnol</em> 2005,  <strong>23</strong><strong>:</strong>48-55.  <a target="_blank" href="/pubmed/15629858">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15629858">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B56" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B56','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B57">
            <p><a name="B57"></a> Gao L,  Tang J,  Li H,  Jia J: <strong> Analysis of microsatellites in major crops assessed by computational and experimental
                  approaches. </strong><p><em>Mol Breeding</em> 2003,  <strong>12</strong><strong>:</strong>245-261.   <a target="_blank" href="http://dx.doi.org/10.1023/A:1026346121217">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B57" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B57','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B58">
            <p><a name="B58"></a> Metzgar D,  Bytof J,  Wills C: <strong> Selection against frameshift mutations limits expansion in coding DNA. </strong><p><em>Genome Res</em> 2000,  <strong>10</strong><strong>:</strong>72-80.  <a target="_blank" href="/pubmed/10645952">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10645952">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=10645952">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B58" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B58','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B59">
            <p><a name="B59"></a> Jung S,  Abbott A,  Jesudurai C,  Tomkins J,  Main D: <strong> Frequency, type, distribution and annotation of simple sequence repeats in Rosaceae
                  ESTs. </strong><p><em>Funct Integr Genomics</em> 2005,  <strong>5</strong><strong>:</strong>136-143.  <a target="_blank" href="/pubmed/15761705">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15761705">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B59" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B59','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B60">
            <p><a name="B60"></a> Beckett P,  Bancroft I,  Trick M: <strong> Computational tools for Brassica-Arabidopsis comparative genomics. </strong><p><em>Comp Funct Genom</em> 2005,  <strong>6</strong><strong>:</strong>147-152.   <a target="_blank" href="http://dx.doi.org/10.1002/cfg.463">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B60" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B60','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B61">
            <p><a name="B61"></a> Shoemaker RC,  Schlueter J,  Doyle JJ: <strong> Paleopolyploidy and gene duplication in soybean and other legumes. </strong><p><em>Curr Opin Plant Biol</em> 2006,  <strong>9</strong><strong>:</strong>104-109.  <a target="_blank" href="/pubmed/16458041">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16458041">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B61" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B61','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B62">
            <p><a name="B62"></a> Mudge J,  Cannon SB,  Kalo P,  Oldroyd GED,  Roe BA,  Town CD,  Young ND: <strong> Highly syntenic regions in the genomes of soybean,Medicago truncatula, andArabidopsis
                  thaliana. </strong><p><em>BMC Plant Biology</em> 2005,  <strong>5</strong><strong>:</strong>15.  <a target="_blank" href="/pubmed/16102170">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2229-5-15">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16102170">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B62" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B62','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B63">
            <p><a name="B63"></a> Grover CE,  Wendel JF: <strong> Recent insights into mechanisms of genome size change in plants. </strong><p><em>J Botany 2010</em> 2010,  <strong>Article ID 382732</strong><strong>:</strong>1-8.  <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B63" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B63','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B64">
            <p><a name="B64"></a> Bruggmann R,  Bharti AK,  Gundlach H,  Lai J,  Young S,  Pontaroli AC,  Wei F,  Haberer G,  Fuks G,  Du C,  Raymond C,  Estep MC,  Liu R,  Bennetzen JL,  Chan AP,  Rabinowicz PD,  Quackenbush J,  Barbazuk WB,  Wing RA,  Birren B,  Nusbaum C,  Rounsley S,  Mayer FX,  Messing J: <strong> Uneven chromosome contraction and expansion in the maize genome. </strong><p><em>Genome Res</em> 2006,  <strong>16</strong><strong>:</strong>1241-1251.  <a target="_blank" href="/pubmed/16902087">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16902087">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16902087">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B64" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B64','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B65">
            <p><a name="B65"></a> Ainouche AK,  Bayer R: <strong> Phylogenetic relationships in Lupinus (Fabaceae: Papilionoideae) based on internal
                  transcribed spacer sequences (ITS) of nuclear ribosomal DNA. </strong><p><em>Am J Bot</em> 1999,  <strong>86</strong><strong>:</strong>590-607.  <a target="_blank" href="/pubmed/10205079">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10205079">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B65" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B65','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B66">
            <p><a name="B66"></a> Swiecicki W,  Swiecicki WK,  Nijaki T: <strong> Lupinus x hispanicoluteus – An interspecific hybrid of Old world lupins. </strong><p><em>Acta Soc Bot Pol</em> 1999,  <strong>68</strong><strong>:</strong>217-220.  <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B66" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B66','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B67">
            <p><a name="B67"></a> Drummond CS: <strong> Diversification of Lupinus (Leguminosae) in the western New World: Derived evolution
                  of perennial life history and colonization of montane habitats. </strong><p><em>Mol Phylogenet Evol</em> 2008,  <strong>48</strong><strong>:</strong>408-421.  <a target="_blank" href="/pubmed/18534869">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18534869">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B67" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B67','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B68">
            <p><a name="B68"></a> Callow JA,  Ford-Lloyd BV,  Newbury HJ: <em>Biotechnology and plant genetic resources: conservation and use</em>.  CAB International, Wallingford, UK;  1997:49-76.  <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B68" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B68','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B69">
            <p><a name="B69"></a> Swanson T: <strong> Global values of Biological diversity: the public interest in the conservation of
                  plant genetic resources for agriculture. </strong><p><em>Plant Genet Res Newsletter</em> 1996,  <strong>105</strong><strong>:</strong>1-3.  <a href="/sfx_links?ui=1471-2164-13-425&amp;bibl=B69" onclick="popup('/sfx_links?ui=1471-2164-13-425&amp;bibl=B69','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
      </ol>
   </div>
</section>
	</div>
	    </div></div>
</div>


		</div>
</div>
    			
		<dl class="google-ad wide ">
	<dt class="hide">
					<a class="banner-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising"></a>
				
			</dt>
		<dd>

		<!-- OAS AD 'Bottom' begin -->
		<script type="text/javascript">
		OAS_AD('Bottom');

		</script>
		<!-- OAS AD 'Bottom' end -->
		</dd>

		</dl>

		<noscript>
			<dl class="google-ad  noscript">  
    <dt class="hide"><a href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</dt>
    
    <dd>
        <a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcgenomics/article/10.1186/1471/2164/13/425/18792631444@Bottom?">
            <img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/biomedcentral.com/bmcgenomics/article/10.1186/1471/2164/13/425/19246182553@Bottom?" alt="advert"/>
        </a>
    </dd>
			</dl>
		</noscript>
			    <hr class="hide" />
	
    



 
    
    
                                                                                                                                                                                                                                
                                                                                                                                                            
    
            

            
        <script>window.bmcIsMobile = "classic";</script>
        
        
    
<div id="footer">

									<span class="views"><a href="?fmt_view=mobile">Mobile view</a> | <strong>Desktop view</strong></span>
			
	<div class="content">
    <div class="rounded ">
<div class="wrap-inner ">
        	
    <ul class="desktop">
                                                    
				        <li class="noborder"><a href="http://www.biomedcentral.com/about/tandc">Terms and Conditions</a></li>
<li><a href="http://www.biomedcentral.com/about/privacy">Privacy statement</a></li>
<li><a href="http://www.biomedcentral.com/presscenter">Press</a></li>
	<li><a href="http://www.biomedcentral.com/advertisers">Information for advertisers</a></li>
<li><a href="http://www.biomedcentral.com/about/bmcjobs">Jobs at BMC</a></li>
<li><a href="http://www.biomedcentral.com/support">Support</a></li>
<li><a href="http://www.biomedcentral.com/about/contact">Contact us</a></li>


    
    	    </ul>
    <p id="copyright">&copy; 2013 
	BioMed Central Ltd unless otherwise stated. Part of Springer Science+Business Media.</p>
        </div>
</div>

	</div>

</div>
<div class="springer">
    <a href="http://www.springer.com/" target="blank"><img style="float:right; margin:0;" src="/images/Springer_Branding_footer_logo.png" /></a>
</div>
					    
<script src="//ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script>
<script type="text/javascript">
if (typeof jQuery == 'undefined') { document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.min.js' type='text/javascript'%3E%3C/script%3E")); }

document.documentElement.id = "js";



var site = {
			abbreviation: "/bmcgenomics",
	        type: "BMC_CORE",
        portalId: "9001",
                section: "browse",
                page: "fulltext",
    		
									homePortal: "biomedcentral.com",
		  

			siteShowAds: "true",
				showAds: "true",
				url: "biomedcentral.com/bmcgenomics",
			portal: false,
	        adpage_url: "biomedcentral.com/bmcgenomics/article/10.1186/1471/2164/13/425",
        id: "1008",
	name: "BMC Genomics",
	plugins: {
		scrollable: (document.getElementsByClassName("scrollable").length > 0),
		flowplayer: ((document.getElementById("flowplayer") != null) || (document.getElementsByClassName("myPlayer").length > 0) || (document.getElementById("videoarticle") != null) || (document.getElementsByClassName("flowplayer").length > 0) || (document.getElementById("audio") != null) || (document.getElementById("elifeplayer") != null)),
		lightbox: ((document.getElementById("image-highlight") != null) || (document.getElementsByClassName("lightbox").length > 0)),
		thickbox: (document.getElementsByClassName("thickbox").length > 0)
	}
	
};



if(typeof quoteSearchUri == "undefined") { quoteSearchUri=null; }
</script>


        <script type="text/javascript" src="/javascript/journals/behaviours-0.js"></script>








				        <script type="text/javascript" src="/javascript/plugins/mobile-0.js"></script>
	        <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.23/jquery-ui.min.js"></script>
	



            <script type="text/javascript" src="/javascript/articles/articles-0.js"></script>
		<div id="fb-root"></div>








<!--[if lte IE 6]>
            <script type="text/javascript" src="/javascript/plugins/DD_belatedPNG_0.0.8a-0.js"></script>
    <script type="text/javascript">
        DD_belatedPNG.fix('.png_bg, ul.primary-nav li a, ul.primary-nav li a span, ul.secondary-nav li a, ul.secondary-nav li a span, ul li i, .plus-button, .facebook, .twitter, .fancy-bg, #fancybox-close');
    </script>
	                <link rel="stylesheet" type="text/css" href="/css/hacks/ie6-0.css"/>
    <![endif]-->

<!-- Claudia Whitcombe wanted this comment here -->

       <div class="hide">
      <dl class="google-ad" id="x96banner">
        <dt class="hide" style="display:block;"></dt>
        <dd>
          <!-- OAS AD 'x96' begin -->
          <script type="text/javascript">
            OAS_AD('x96');
          </script>
        </dd>
      </dl>
    </div>  
    
</body>
</html>

