
			<!doctype html>
	<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" class="no-javascript">	<head>
		<meta charset="utf-8" />
				<title>
			Managing biological complexity across orthologs with a visual knowledgebase of documented biomolecular interactions :  Scientific Reports :  Nature Publishing Group
		</title>
		
		<script type="text/javascript"><!--//--><![CDATA[//><!--
		document.getElementsByTagName('html')[0].className = 'js-enabled';
		//--><!]]></script>

		<script type="text/javascript" src="//nexus.ensighten.com/npg/Bootstrap.js"></script>

										
												
					
													

									

								
		

	
																	

		





					




						

										<meta name="keywords" content="" />
															<meta name="description" content="The complexity of biomolecular interactions and influences is a major obstacle to their comprehension and elucidation. Visualizing knowledge of biomolecular interactions increases comprehension and facilitates the development of new hypotheses. The rapidly changing landscape of high-content experimental results also presents a challenge for the maintenance of comprehensive knowledgebases. Distributing the responsibility for maintenance of a knowledgebase to a community of subject matter experts is an effective strategy for large, complex and rapidly changing knowledgebases. Cognoscente serves these needs by building visualizations for queries of biomolecular interactions on demand, by managing the complexity of those visualizations, and by crowdsourcing to promote the incorporation of current knowledge from the literature." />
															<meta name="robots" content="noarchive" />
															<meta name="access" content="Yes" />
						
										<meta name="DC.publisher" content="Nature Publishing Group" />
															<meta name="DC.language" content="en" />
															<meta name="DC.rights" content="© 2012 Macmillan Publishers Limited. All rights reserved" />
															<meta name="DC.title" content="Managing biological complexity across orthologs with a visual knowledgebase of documented biomolecular interactions" />
															<meta name="DC.identifier" content="doi:10.1038/srep01011" />
															<meta name="DC.date" content="2012-12-20" />
																				<meta name="DC.creator" content="Vincent VanBuren" />
																<meta name="DC.creator" content="Hailin Chen" />
										
										<meta name="prism.copyright" content="© 2012 Macmillan Publishers Limited. All rights reserved" />
															<meta name="prism.rightsAgent" content="permissions@nature.com" />
															<meta name="prism.publicationDate" content="2012-12-20" />
															<meta name="prism.volume" content="2" />
															<meta name="prism.issue" content="" />
															<meta name="prism.section" content="Article" />
															<meta name="prism.startingPage" content="" />
															<meta name="prism.endingPage" content="" />
															<meta name="prism.publicationName" content="Scientific Reports" />
															<meta name="prism.issn" content="" />
															<meta name="prism.eIssn" content="2045-2322" />
						
										<meta name="citation_publisher" content="Nature Publishing Group" />
															<meta name="citation_title" content="Managing biological complexity across orthologs with a visual knowledgebase of documented biomolecular interactions" />
															<meta name="citation_date" content="2012-12-20" />
															<meta name="citation_volume" content="2" />
															<meta name="citation_issue" content="" />
															<meta name="citation_firstpage" content="" />
															<meta name="citation_doi" content="doi:10.1038/srep01011" />
															<meta name="citation_journal_title" content="Scientific Reports" />
															<meta name="citation_authors" content="Vincent VanBuren; Hailin Chen" />
						
										<meta name="WT.cg_n" content="Scientific Reports" />
															<meta name="WT.cg_s" content="Article" />
															<meta name="WT.template" content="C" />
															<meta name="WT.ec_id" content="NATURE-20130103" />
															<meta name="WT.site_id" content="542;696;8969;10742" />
															<meta name="WT.registrant_id" content="2796558" />
																	
												
					
	<link rel="shortcut icon" href="/favicon.ico" />

	<link rel="search" href="/search/executeSearch" />


	<link rel="search" type="application/opensearchdescription+xml" href="http://www.nature.com/opensearch/opensearch.xml" title="nature.com" />
	<link rel="search" type="application/sru+xml" href="http://www.nature.com/opensearch/request" title="nature.com" />
											
												
					
			
										
		
																																		
								<link href="/view/styles/global.yui.css" type="text/css" rel="stylesheet" />
																	
		
																																		
								<link href="/view/styles/global.layout.css" type="text/css" rel="stylesheet" />
																	
		
																																		
								<link href="/view/styles/global.shared.css" type="text/css" rel="stylesheet" />
																	
		
																																		
								<link href="/view/styles/global.sidebar.css" type="text/css" rel="stylesheet" />
																	
		
																																		
								<link href="/view/styles/global.article.css" type="text/css" rel="stylesheet" />
																	
		
																																		
										<link href="/view/styles/global.main.print.css" type="text/css" rel="stylesheet" media="print" />
																	
		
																																		
								<link href="/view/styles/global.header-v2.css" type="text/css" rel="stylesheet" />
																			
					<!--[if gte IE 8]><!--> 
		
																																		
								<link href="/view/styles/global.mobile.css" type="text/css" rel="stylesheet" />
											<!--<![endif]-->
											
		
																																		
								<link href="/view/srep/styles/site.css" type="text/css" rel="stylesheet" />
																			
					<!--[if gte IE 8]><!--> 
		
																																		
								<link href="/view/srep/styles/site.mobile.css" type="text/css" rel="stylesheet" />
											<!--<![endif]-->
						
		<!--[if IE 6]>
		<script type="text/javascript" src="/view/scripts/global.ie6.js"></script>
		<![endif]-->
	</head>
	
				
							
		
			<!--[if lt IE 6]><body class="www-nature-com article lt-ie6"><![endif]-->
	<!--[if IE 6]><body class="www-nature-com article ie6"><![endif]-->
	<!--[if IE 7]><body class="www-nature-com article ie7"><![endif]-->
	<!--[if IE 8]><body class="www-nature-com article ie8"><![endif]-->
	<!--[if gt IE 8]><body class="www-nature-com article gt-ie8"><![endif]-->
	<!--[if !IE]>--><body class="www-nature-com article"><!--<![endif]-->
		
		
		<div id="constrain">
			<div id="constrain-header" class="cleared">
								<header><div class="wrapper">
															
												
							
						







<div class="constrain cleared" id="foreword">

		
	<nav>		<h1 class="home" id="top"><a href="/">nature.com</a></h1>
		<p class="sitemap"><a href="/siteindex">Sitemap</a></p>
		<ul class="right">
			<li class="cart"><a href="/ecommerce/cart/list.action">Cart</a></li>
																		<li class="my-account">	
																<a href="/myaccount" class="my-account">My account</a></li>
																				<li class="logout">	
																<a href="/foxtrot/svc/logout" class="logout">Logout</a></li>
														</ul>
	</nav>

						<p class="welcome">Welcome back Cameron Neylon</p>
			</div>

<div class="constrain cleared" id="top-ads">
	
					
	
							
	
																		
						

		
		
				
			
		
		
			
						
							
																											
																		
																										
						
	
	
	
	
		
		<div class="ad adunit-leaderboard 
					leaderboard
		">
		
					<script type="text/javascript" src="//ad.doubleclick.net/N285/adj/scientific_reports/article;pos=top;artid=srep01011;type=af;submeta=%2F631%2F114%2F129;submeta=%2F631%2F553%2F2703;submeta=%2F631%2F114%2F2408;submeta=%2F631%2F208%2F200;abr=!webtv;ptile=4;sz=728x90;ord=4592047608988775?"> </script>
			<noscript>
					<a href="//ad.doubleclick.net/N285/jump/scientific_reports/article;pos=top;artid=srep01011;type=af;submeta=%2F631%2F114%2F129;submeta=%2F631%2F553%2F2703;submeta=%2F631%2F114%2F2408;submeta=%2F631%2F208%2F200;abr=!webtv;ptile=4;sz=728x90;ord=4592047608988775?"><img src="//ad.doubleclick.net/N285/ad/scientific_reports/article;pos=top;artid=srep01011;type=af;submeta=%2F631%2F114%2F129;submeta=%2F631%2F553%2F2703;submeta=%2F631%2F114%2F2408;submeta=%2F631%2F208%2F200;abr=!webtv;ptile=4;sz=728x90;ord=4592047608988775?" alt="Advertisment" /></a>
					</noscript>
			</div>
	</div>
<div id="header" class="constrain v2">
					
						<h1>
									<a href="/srep">Scientific Reports</a>
							</h1>
				
	
									
												
					




			<form method="get" action="/search/executeSearch" id="header-search" class="search-form" role="search">
			<label for="header-keyword">Search</label>
			<input type="hidden" name="sp-c" value="25"/>
			<input type="hidden" name="sp-s" value=""/>
															 									 						<input type="hidden" name="sp-q-1" value="SREP"/>
						 			 				
			<input type="search" id="header-keyword" name="sp-q" placeholder="Search" value="" class="txt" maxlength="200"/>
			<button type="submit" id="header-search-submit">Go</button>
							<a href="/search/adv_search?sp-q-1=srep">Advanced search</a>
					</form>
			
		
	<nav>
		<ul id="main-navigation" class="clear" role="navigation" >
							<li class="nav-home first"><a href="/srep/index.html"><span>Home</span></a></li>
							<li class="nav-search"><a href="/search/executeSearch?sp-advanced=true&amp;sp-m=0&amp;siteCode=srep&amp;sp-p=all&amp;sp-q-9[SREP]=1&amp;sp-p-2=all&amp;sp-p-3=all&amp;subject=&amp;sp-s=&amp;sp-c=25"><span>Search</span></a></li>
							<li class="nav-authors"><a href="/srep/authors/index.html"><span>For Authors</span></a></li>
							<li class="nav-referees"><a href="/srep/referees/index.html"><span>For Referees</span></a></li>
							<li class="nav-about"><a href="/srep/about/index.html"><span>About Scientific Reports</span></a></li>
					</ul>
	</nav>
</div>





	 		 		 		 	
<div id="breadcrumbs" class="constrain">
	<nav>
		<ul id="sub-navigation" class="clear" role="navigation" >
															<li class="first parent parent-1"><a href="/search/executeSearch?sp-advanced=true&amp;sp-m=0&amp;siteCode=srep&amp;sp-p=all&amp;sp-q-9[SREP]=1&amp;sp-p-2=all&amp;sp-p-3=all&amp;sp-s=date_descending&amp;sp-c=25">Search</a></li>
															<li class="parent parent-2"><a href="/search/executeSearch?sp-advanced=true&amp;sp-m=0&amp;siteCode=srep&amp;sp-p=all&amp;sp-q-9[SREP]=1&amp;sp-p-2=all&amp;sp-p-3=all&amp;pub-date-mode=between&amp;sp-start-month=01&amp;sp-start-year=2012&amp;sp-end-month=12&amp;sp-end-year=2012&amp;sp-s=date_descending&amp;sp-c=25">2012</a></li>
															<li class="parent parent-3"><a href="/search/executeSearch?sp-advanced=true&amp;siteCode=srep&amp;sp-p=all&amp;sp-q-9[SREP]=1&amp;sp-p-2=all&amp;sp-p-3=all&amp;pub-date-mode=between&amp;sp-start-month=12&amp;sp-start-year=2012&amp;sp-end-month=12&amp;sp-end-year=2012&amp;sp-s=date_descending&amp;sp-c=25">December</a></li>
																									<li class="current"><span>Article</span></li>
					</ul>
	</nav>
</div>


					</div></header>


			</div>
			
			<div id="constrain-content" class="cleared">
			






















































































	
 



																																																					




























										








														




















































									














 




















	
















																												
	
	
		









		







												
												
							



			




		







								




  						
<div id="content" class="col article-template" role="article">
											
												
					

			
	<article>										
												
								<header>			<div class="top-links cleared">
									<p class="article-type">
									<span class="journal-title">Scientific Reports</span><span class="divider"> | 
					</span>Article					<abbr title="open-access" class="open-access">Open</abbr>				</p>
			
																	
												
					
<div class="comm-tool-box" >
			<ul class="article-com-tools">
			<li class="print"><a href="javascript:window.print();" id="print-link" class="track" data-tooltip="Print">Print</a></li>
						<li class="sendtofriend"><a href="/srep/foxtrot/svc/mailform?doi=10.1038/srep01011&amp;file=/srep/2012/121220/srep01011/full/srep01011.html" class="tip" data-tooltip="Email">Email</a></li>
					<li class="bookmarking">
				<a id="toggle-bookmarking-links" href="javascript:;" class="tip" data-tooltip="Share/bookmark">Share/bookmark</a>
				<div id="bookmarking-links">
													
												
					
<ul class="bookmarking">
			<li class="citeulike"><a href="http://www.citeulike.org/posturl2?username=&amp;url=http://dx.doi.org/10.1038/srep01011&amp;title=Managing+biological+complexity+across+orthologs+with+a+visual+knowledgebase+of+documented+biomolecular+interactions&amp;rewrite=yes">Cite<span class="hidden"> </span>U<span class="hidden"> </span>Like</a></li>
		<li class="facebook"><a href="http://www.facebook.com/sharer.php?u=http://dx.doi.org/10.1038/srep01011&amp;t=Managing+biological+complexity+across+orthologs+with+a+visual+knowledgebase+of+documented+biomolecular+interactions">Facebook</a></li>
				<li class="twitter"><a href="http://twitter.com/home/?status=Managing+biological+complexity+across+orthologs+with+a+visual+knowledgebase+of+documented+biomolecular+interactions+http://go.nature.com/eGvVfC">Twitter</a></li>
	<li class="delicious"><a href="http://del.icio.us/post?url=http://dx.doi.org/10.1038/srep01011&amp;title=Managing+biological+complexity+across+orthologs+with+a+visual+knowledgebase+of+documented+biomolecular+interactions">Delicious</a></li>
	<li class="digg"><a href="http://digg.com/submit?phase=2&amp;url=http://dx.doi.org/10.1038/srep01011&amp;title=Managing+biological+complexity+across+orthologs+with+a+visual+knowledgebase+of+documented+biomolecular+interactions">Digg</a></li>
	<li class="google-plus"><a href="https://plusone.google.com/_/+1/confirm?hl=en&amp;url=http://dx.doi.org/10.1038/srep01011&amp;title=Managing+biological+complexity+across+orthologs+with+a+visual+knowledgebase+of+documented+biomolecular+interactions" >Google&#43;</a></li>
	<li class="linkedin"><a href="http://www.linkedin.com/cws/share?url=http://dx.doi.org/10.1038/srep01011&amp;title=Managing+biological+complexity+across+orthologs+with+a+visual+knowledgebase+of+documented+biomolecular+interactions">LinkedIn</a></li>
	<li class="reddit"><a href="http://www.reddit.com/s/http://dx.doi.org/10.1038/srep01011">Reddit</a></li>
	<li class="stumbleupon"><a href="http://www.stumbleupon.com/submit?url=http://dx.doi.org/10.1038/srep01011&amp;title=Managing+biological+complexity+across+orthologs+with+a+visual+knowledgebase+of+documented+biomolecular+interactions">StumbleUpon</a></li>
</ul>					</div>
			</li>
			</ul>

</div>


									
				
			</div>
			
						
																																					
			 			
			 							
						
									
																
							<h1 class="article-heading">Managing biological complexity across orthologs with a visual knowledgebase of documented biomolecular interactions</h1>
												
																		
												
																					
												
																					
												
					
		
					
												
								
										
		<ul class="authors citation-authors">
																																											
			<li class="vcard  c1">
				
				<a href="#auth-1" class="name"><span class="fn">Vincent VanBuren</span></a><sup><a href="#a1">1</a></sup>			</li>
			
																							
				
																											
										
																												
			<li class="vcard last-author">
				
				<a href="#auth-2" class="name"><span class="fn">Hailin Chen</span></a><sup><a href="#a1">1</a></sup>			</li>
			
																			
				
																											
										
			</ul>
	
		
	
	
	

	
	
	

	
							
		
															
					<ul id="author-links" class="cleared">
									<li class="first"><a href="#affil-auth">Affiliations</a></li>
									<li><a href="#contrib-auth">Contributions</a></li>
									<li><a href="#corres-auth">Corresponding author</a></li>
							</ul>
			
	<dl class="citation">
		<dt>Journal name:</dt>	
		<dd class="journal-title">Scientific Reports</dd>
						<dt class="volume">Volume:</dt>
		<dd class="volume">
			2<span>,</span></dd>
							<dt class="seq">Article number:</dt>
							<dd class="page">1011</dd>
	       							 		<dt>DOI:</dt>
		<dd class="doi">doi:10.1038/srep01011</dd>
			</dl>

	<dl class="citation dates">
																							<dt class="received first">Received</dt>
			<dd><time datetime="2012-10-02">
															02 October 2012
												</time></dd>
											 												<dt class="accepted">Accepted</dt>
			<dd><time datetime="2012-12-04">
															04 December 2012
												</time></dd>
									
																<dt class="published-online">Published</dt>
			<dd><time datetime="2012-12-20" pubdate="pubdate">
															20 December 2012
												</time></dd>
								</dl>
	
									
												
							
						
			 
											
												
					
	
															
												
					

 

		


<aside><div class="article-tools">
	<h2 class="hidden">Article tools</h2>
	
								<script type="text/javascript" src="http://content.readcube.com/ping?doi=10.1038/srep01011&format=js"></script>
	
	<ul class="tools">
		 					
						
		 												<li> <h3 id="toggle-download-links">PDF</h3>
						<div id="download-links">
							<ul class="downloading ">
								<li class="download-option articlepdf"><a href="/srep/2012/121220/srep01011/pdf/srep01011.pdf">Download as PDF (3,032 KB)</a>
								</li>
																<li class="download-option readcube">
									<a href="http://readcube.com/view/10.1038/srep01011" class="readcube-view">View interactive PDF in ReadCube</a>
								</li>
							</ul>
						</div>
					</li>
																		<li class="download-citation"><a href="/srep/2012/121220/srep01011/ris/srep01011.ris">Citation</a></li>
							
															<li class="order-reprints"><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=NPGR&amp;publication=Scientific+Reports&amp;title=Managing+biological+complexity+across+orthologs+with+a+visual+knowledgebase+of+documented+biomolecular+interactions&amp;contentID=10.1038%2Fsrep01011&amp;volumeNum=2&amp;issueNum=&amp;numPages=&amp;pageNumbers=pp%24%7BnPage.startPage%7D&amp;orderBeanReset=true&amp;publicationDate=2012-12-20&amp;author=Vincent+VanBuren%2C+Hailin+Chen">Reprints</a></li>
				<li class="rights"><a href="https://s100.copyright.com/AppDispatchServlet?publisherName=NPG&amp;publication=Scientific+Reports&amp;title=Managing+biological+complexity+across+orthologs+with+a+visual+knowledgebase+of+documented+biomolecular+interactions&amp;contentID=10.1038%2Fsrep01011&amp;volumeNum=2&amp;issueNum=&amp;numPages=&amp;pageNumbers=pp%24%7BnPage.startPage%7D&amp;publicationDate=2012-12-20&amp;cc=by&amp;author=Vincent+VanBuren%2C+Hailin+Chen">Rights &amp; permissions</a></li>
					
		
		

					<li class="article-metrics">
															<a href="/srep/2012/121220/srep01011/metrics" >Metrics</a>
												</li>
			</ul>
</div></aside>

					
											
												
						
		</header>
		
		
												
												
					

	
			
		
							
																						
		
										
					
							
			
										
										
																	
			
										
										
																						
			
										
										
																						
			
										
										
																						
			
										
										
																					
														
																						
																					
																					
																					
																					
																												
																																																								
																																																																					
																																										
																												
																																										
																																			
																																										
																					
																					
							
																																																		
				
				
						
																				
									
									<section><div class="section first no-nav no-title first-no-nav">

						<div class="content">
												
								
												
												
					<div class="first-paragraph" id="first-paragraph"><p>The complexity of biomolecular interactions and influences is a major obstacle to their comprehension and elucidation. Visualizing knowledge of biomolecular interactions increases comprehension and facilitates the development of new hypotheses. The rapidly changing landscape of high-content experimental results also presents a challenge for the maintenance of comprehensive knowledgebases. Distributing the responsibility for maintenance of a knowledgebase to a community of subject matter experts is an effective strategy for large, complex and rapidly changing knowledgebases. Cognoscente serves these needs by building visualizations for queries of biomolecular interactions on demand, by managing the complexity of those visualizations, and by crowdsourcing to promote the incorporation of current knowledge from the literature.</p></div>


								
												
					


                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        
								<div class="article-keywords cleared">
			<h2>Subject terms:</h2>
			<ul>
																								<li class="first"><a href="/search/executeSearch?sp-advanced=true&amp;sp-m=0&amp;siteCode=srep&amp;sp-p=all&amp;sp-q-9[SREP]=1&amp;sp-p-2=all&amp;sp-p-3=all&amp;subject=/631/114/129&amp;sp-s=date_descending&amp;sp-c=25&amp;facets=new">Databases</a></li>
																			<li><a href="/search/executeSearch?sp-advanced=true&amp;sp-m=0&amp;siteCode=srep&amp;sp-p=all&amp;sp-q-9[SREP]=1&amp;sp-p-2=all&amp;sp-p-3=all&amp;subject=/631/114/2408&amp;sp-s=date_descending&amp;sp-c=25&amp;facets=new">Network topology</a></li>
																			<li><a href="/search/executeSearch?sp-advanced=true&amp;sp-m=0&amp;siteCode=srep&amp;sp-p=all&amp;sp-q-9[SREP]=1&amp;sp-p-2=all&amp;sp-p-3=all&amp;subject=/631/208/200&amp;sp-s=date_descending&amp;sp-c=25&amp;facets=new">Gene regulation</a></li>
																																				<li class="last"><a href="/search/executeSearch?sp-advanced=true&amp;sp-m=0&amp;siteCode=srep&amp;sp-p=all&amp;sp-q-9[SREP]=1&amp;sp-p-2=all&amp;sp-p-3=all&amp;subject=/631/553/2703&amp;sp-s=date_descending&amp;sp-c=25&amp;facets=new">Complexity</a></li>
							</ul>
		</div>
										
												
					

<div class="figures-at-a-glance">
	
		
	<h2>At a glance</h2>
	<div id="figure-browser-group">
		<div class="figure-browser">
			<ol>
																																																																							
																																												
																																	<li>
																																			
								<figure>									<figcaption><span class="legend cleared">
										Figure 1: Biomolecular interactions manually added to Cognoscente via a web-based submission portal.
									</span></figcaption>

																						
		
																														
			
				
				
	<img src="/srep/2012/121220/srep01011/carousel/srep01011-f1.jpg" alt="Biomolecular interactions manually added to Cognoscente via a web-based submission portal." class="fig" data-media-desc="Figure" data-media-id="figure" style="width:214px;height:200px;" />
																			<div class="description">
										<p>Interactions supplied by NCBI Interaction GeneRIFs were found to be incomplete with respect to documented interactions. We subsequently identified 75 interactions relevant to cardiac development in the literature, and manually added them to Cognoscente via a web-based submission portal.</p>
									</div>
																			<div class="larger">
																							<p class="fig-link down-arrow"><a href="#f1">See larger</a></p>

																					</div>
																	</figure>
	
						</li>
					 																									<li>
																																			
								<figure>									<figcaption><span class="legend cleared">
										Figure 2: Submitting new biomolecular interactions in Cognoscente.
									</span></figcaption>

																						
		
																														
			
				
				
	<img src="/srep/2012/121220/srep01011/carousel/srep01011-f2.jpg" alt="Submitting new biomolecular interactions in Cognoscente." class="fig" data-media-desc="Figure" data-media-id="figure" style="width:184px;height:200px;" />
																			<div class="description">
										<p>User submissions interface for <b>Cognoscente</b> (A). Feedback is provided to the user about their proposed submission. If the submission appears to be redundant with an entry in the knowledgebase, the user receives a warning about the potential redundancy (B). Likewise, users receive a ‘green light’ if the proposed interaction does not already exist (C). In addition to these notifications, the system also retrieves gene information based on the user-supplied gene IDs, and retrieves the entire citation and abstract for the submitted PubMed IDs. These measures are expected to reduce redundancy and errors due in manual submissions.</p>
									</div>
																			<div class="larger">
																							<p class="fig-link down-arrow"><a href="#f2">See larger</a></p>

																					</div>
																	</figure>
	
						</li>
					 																									<li>
																																			
								<figure>									<figcaption><span class="legend cleared">
										Figure 3: The Basic Query interface for Cognoscente allows queries of multiple genes, the option to show orthologous interactions, and an option to prescribe the complexity of the network to draw in the result (A).
									</span></figcaption>

																						
		
																														
			
				
				
	<img src="/srep/2012/121220/srep01011/carousel/srep01011-f3.jpg" alt="The Basic Query interface for Cognoscente allows queries of multiple genes, the option to show orthologous interactions, and an option to prescribe the complexity of the network to draw in the result (A)." class="fig" data-media-desc="Figure" data-media-id="figure" style="width:216px;height:200px;" />
																			<div class="description">
										<p>A list of multiple genes can be divided into groups by separating the entered gene symbols with a forward slash (‘/’). In the results, the queried genes will be indicated as groups by different colored boxes around the network nodes. These groups (up to 3) may be named (B), which will be indicated in an automatically generated legend in the resulting graph. Results may be filtered by interaction type or by annotation status (B).</p>
									</div>
																			<div class="larger">
																							<p class="fig-link down-arrow"><a href="#f3">See larger</a></p>

																					</div>
																	</figure>
	
						</li>
					 																									<li>
																																			
								<figure>									<figcaption><span class="legend cleared">
										Figure 4: Graphical query result for human genes ‘<i>Nkx2-5</i>, <i>Mef2c</i>’ in Cognoscente.
									</span></figcaption>

																						
		
																														
			
				
				
	<img src="/srep/2012/121220/srep01011/carousel/srep01011-f4.jpg" alt="Graphical query result for human genes ‘Nkx2-5, Mef2c’ in Cognoscente." class="fig" data-media-desc="Figure" data-media-id="figure" style="width:246px;height:200px;" />
																			<div class="description">
										<p>This query was parameterized to search across all known orthologs for all primary (<i>radius</i> = 1) biomolecular interactions with either <i>Nkx2-5</i> or <i>Mef2c</i>. <b>Cognoscente</b> automatically generates legends (as shown here) that are inserted into the graphic.</p>
									</div>
																			<div class="larger">
																							<p class="fig-link down-arrow"><a href="#f4">See larger</a></p>

																					</div>
																	</figure>
	
						</li>
					 																									<li>
																																			
								<figure>									<figcaption><span class="legend cleared">
										Figure 5: Partial results for the query “cardiac muscle” using the Gene Lookup tool (<a href="http://vanburenlab.medicine.tamhsc.edu/gene_lookup.html">http://vanburenlab.medicine.tamhsc.edu/gene_lookup.html</a>).
									</span></figcaption>

																						
		
																														
			
				
				
	<img src="/srep/2012/121220/srep01011/carousel/srep01011-f5.jpg" alt="Partial results for the query “cardiac muscle” using the Gene Lookup tool (http://vanburenlab.medicine.tamhsc.edu/gene_lookup.html)." class="fig" data-media-desc="Figure" data-media-id="figure" style="width:220px;height:200px;" />
																			<div class="description">
										<p>Full results include a search against gene descriptions, GO terms, official gene symbols and unofficial gene synonyms. Graphic symbols (as described in the legend) appear next to genes that are supported by StarNet or Cognoscente, respectively. These graphics may be clicked by the user to redirect to the appropriate query page, and automatically populate parameters on the query page for the respective gene.</p>
									</div>
																			<div class="larger">
																							<p class="fig-link down-arrow"><a href="#f5">See larger</a></p>

																					</div>
																	</figure>
	
						</li>
					 																									<li>
																																			
								<figure>									<figcaption><span class="legend cleared">
										Figure 6: Managing complexity with Cognoscente.
									</span></figcaption>

																						
		
																														
			
				
				
	<img src="/srep/2012/121220/srep01011/carousel/srep01011-f6.jpg" alt="Managing complexity with Cognoscente." class="fig" data-media-desc="Figure" data-media-id="figure" style="width:272px;height:200px;" />
																			<div class="description">
										<p>iPSC genes <i>Oct4</i> (<i>Pou5f1</i>), <i>Sox2</i>, <i>c-Myc</i> (<i>Myc</i>), <i>Klf4</i>, <i>Nanog</i>, and <i>Lin28A</i> were queried with <b>Cognoscente</b> using different approaches. In the upper left panel, all primary (<i>radius</i> = 1) interactions are shown across all orthologs. In the upper right panel, only <i>human</i> primary interactions are shown. In the lower left panel, <i>radius</i> = 0 interactions (those between members of the query list) and ‘intermediates’ (those where the interactant has interactions with at least two members of the query list) are shown across all orthologs. Finally, in the lower right panel, only <i>human</i> <i>radius</i> = 0 <i>interactions with intermediates</i> are shown, which eliminates hundreds of nodes as compared with the upper left panel.</p>
									</div>
																			<div class="larger">
																							<p class="fig-link down-arrow"><a href="#f6">See larger</a></p>

																					</div>
																	</figure>
	
						</li>
					 				 			</ol>
		</div>
	</div>			
</div>
								
												
											
							
				
							</div>
																					</div></section>

		
						
																								
												
									<section><div class="section   expanded" id="introduction">

							<h1 class="section-heading toggle">Introduction</h1>
						<div class="content">
																		<nav><ul class="section-nav cleared">

																																																														<li class="current first"><span>Introduction</span></li>	
																																													<li><a href="#results">Results</a></li>
																																													<li><a href="#discussion">Discussion</a></li>
																																													<li><a href="#methods">Methods</a></li>
																																													<li><a href="#references">References</a></li>
																																													<li><a href="#acknowledgments">Acknowledgements</a></li>
																																													<li><a href="#author-information">Author information</a></li>
																																													<li><a href="#supplementary-information">Supplementary information</a></li>
																																													<li class="last"><a href="#comments">Comments</a></li>
																		
					</ul></nav>

								
								
				<p>Imputing functional associations between biomolecules and imputing directionality of regulation for those predictions each require a corpus of existing knowledge as a framework to build upon. Comprehension of the complexity of this corpus of knowledge will be facilitated by effective visualizations of the corresponding biomolecular interaction networks. <b>Cognoscente</b> (<a href="http://vanburenlab.medicine.tamhsc.edu/cognoscente.html">http://vanburenlab.medicine.tamhsc.edu/cognoscente.html</a>) was designed and implemented to serve these roles as a knowledgebase and as an effective visualization tool for systems biology research and education. <b>Cognoscente</b> currently contains over 413,000 documented interactions, with coverage across multiple species. Perl, HTML, <b>GraphViz</b><sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref1" title="Gansner, E. &amp; North, S. An open graph visualization system and its applications to software engineering. Softw. Pract. Exper. 00, 1–5 (1999)." id="ref-link-1">1</a></sup>, and a <b>MySQL</b> database were used in the development of <b>Cognoscente</b>.</p><p><b>Cognoscente</b> was motivated by the need to (1) update the knowledgebase of biomolecular interactions at the user level, and (2) flexibly visualize multi-molecule query results for heterogeneous interaction types across different orthologs. Satisfying these needs provides a strong foundation for developing new hypotheses about regulatory and metabolic pathway topologies. Several existing tools provide functions that are similar to <b>Cognoscente</b>, so we selected several popular alternatives to assess how their feature sets compare with <b>Cognoscente</b> (<a href="/srep/2012/121220/srep01011/full/srep01011.html#t1"> <b>Table 1</b> </a>)<sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref2" title="Breitkreutz, B. J. et al. The BioGRID Interaction Database: 2008 update. Nucleic Acids Res 36, D637–640 (2008)." id="ref-link-2">2</a>, <a href="/srep/2012/121220/srep01011/full/srep01011.html#ref3" title="Jensen, L. J. et al. STRING 8--a global view on proteins and their functional interactions in 630 organisms. Nucleic Acids Res 37, D412–416 (2009)." id="ref-link-3">3</a>, <a href="/srep/2012/121220/srep01011/full/srep01011.html#ref4" title="Kerrien, S. et al. The IntAct molecular interaction database in 2012. Nucleic Acids Res 40, D841–846 (2012)." id="ref-link-4">4</a>, <a href="/srep/2012/121220/srep01011/full/srep01011.html#ref5" title="Keseler, I. M. et al. EcoCyc: a comprehensive view of Escherichia coli biology. Nucleic Acids Res 37, D464–470 (2009)." id="ref-link-5">5</a>, <a href="/srep/2012/121220/srep01011/full/srep01011.html#ref6" title="Keshava Prasad, T. S. et al. Human Protein Reference Database--2009 update. Nucleic Acids Res 37, D767–772 (2009)." id="ref-link-6">6</a>, <a href="/srep/2012/121220/srep01011/full/srep01011.html#ref7" title="Licata, L. et al. MINT, the molecular interaction database: 2012 update. Nucleic Acids Res 40, D857–861 (2012)." id="ref-link-7">7</a>, <a href="/srep/2012/121220/srep01011/full/srep01011.html#ref8" title="NCBI. GeneRIF: Gene Reference Into Function, <http://www.ncbi.nlm.nih.gov/gene/about-generif> (2012)." id="ref-link-8">8</a>, <a href="/srep/2012/121220/srep01011/full/srep01011.html#ref9" title="von Mering, C. et al. STRING: a database of predicted functional associations between proteins. Nucleic Acids Res 31, 258–261 (2003)." id="ref-link-9">9</a>, <a href="/srep/2012/121220/srep01011/full/srep01011.html#ref10" title="Willis, R. C. &amp; Hogue, C. W. Searching, viewing, and visualizing data in the Biomolecular Interaction Network Database (BIND). Curr Protoc Bioinformatics Chapter 8, Unit 8 9 (2006)." id="ref-link-10">10</a>, <a href="/srep/2012/121220/srep01011/full/srep01011.html#ref11" title="Xenarios, I. et al. DIP, the Database of Interacting Proteins: a research tool for studying cellular networks of protein interactions. Nucleic Acids Res 30, 303–305 (2002)." id="ref-link-11">11</a></sup>. All databases assessed had easily traceable documentation for each interaction, and included protein-protein interactions in the database. Most databases, with the exception of <b>BIND</b>, provide an open-access database that can be downloaded as a whole. Most databases, with the exceptions of <b>EcoCyc</b> and <b>HPRD</b>, provide support for multiple organisms. Most databases support web services for interacting with the database contents programmatically, whereas this is a planned feature for <b>Cognoscente</b>. <b>MINT</b>, <b>STRING</b>, <b>IntAct</b>, <b>EcoCyc</b>, <b>DIP</b> and <b>Cognoscente</b> provide built-in visualizations of query results, which we consider among the most important features for facilitating comprehension of query results. <b>BIND</b> supports visualizations via <b>Cytoscape</b><sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref12" title="Shannon, P. et al. Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res 13, 2498–2504 (2003)." id="ref-link-12">12</a>, <a href="/srep/2012/121220/srep01011/full/srep01011.html#ref13" title="Smoot, M. E., Ono, K., Ruscheinski, J., Wang, P. L. &amp; Ideker, T. Cytoscape 2.8: new features for data integration and network visualization. Bioinformatics 27, 431–432 (2011)." id="ref-link-13">13</a></sup>. <b>Cognoscente</b> is among a few other tools that support multiple organisms in the same query, protein-&#x0003E;DNA interactions, and multi-molecule queries. <b>Cognoscente</b> has planned support for small molecule interactants (i.e. pharmacological agents). <b>MINT</b>, <b>STRING</b>, and <b>IntAct</b> provide a prediction (i.e. score) of functional associations, whereas <b>Cognoscente</b> does not currently support this. <b>Cognoscente</b> provides support for multiple edge encodings to visualize different types of interactions in the same display, a crowdsourcing web portal that allows users to submit interactions that are then automatically incorporated in the knowledgebase, and displays orthologs as compound nodes to provide clues about potential orthologous interactions. The main strengths of <b>Cognoscente</b> are that (1) it provides a combined feature set that is superior to any existing database, and (2) that it provides a unique visualization feature for orthologous molecules, and relatively unique support for multiple edge encodings, crowdsourcing, and connectivity parameterization. The current weaknesses of <b>Cognoscente</b> relative to these other tools are (1) that it does not fully support web service interactions with the database, (2) it does not fully support small molecule interactants, and (3) it does not score interactions to predict functional associations. Web services and support for small molecule interactants are currently under development.</p>								
												
					




				


	
	
	
	<div class="table figure cleared" id="t1">
		<figure>							<figcaption><span class="legend">
					Table 1: Features of biomolecular interaction databases				</span></figcaption>

									
			<div class="item-content box">
				
														<div class="description">
													<a href="/srep/2012/121220/srep01011/fig_tab/srep01011_T1.html" class="table-link"><span>Full table</span></a>
												
					</div>
												</div>

																	
						
			
			<nav>				<ul class="figure-nav">
										<li class="index only"><a href="/srep/2012/121220/srep01011/fig_tab/srep01011_ft.html">Figures/tables index</a></li>
									</ul>
			</nav>

			
					</figure>

	</div>
	
		
							</div>
																					</div></section>

		
						
																								
												
									<section><div class="section   expanded" id="results">

							<h1 class="section-heading toggle">Results</h1>
						<div class="content">
																		<nav><ul class="section-nav cleared">

																																																						<li class="first"><a href="#introduction">Introduction</a></li>
																																																					<li class="current"><span>Results</span></li>	
																																													<li><a href="#discussion">Discussion</a></li>
																																													<li><a href="#methods">Methods</a></li>
																																													<li><a href="#references">References</a></li>
																																													<li><a href="#acknowledgments">Acknowledgements</a></li>
																																													<li><a href="#author-information">Author information</a></li>
																																													<li><a href="#supplementary-information">Supplementary information</a></li>
																																													<li class="last"><a href="#comments">Comments</a></li>
																		
					</ul></nav>

								
								
				<h2>Crowdsourcing</h2><p><b>Cognoscente</b> was partly motivated by incomplete coverage of documented biomolecular interactions in existing databases. In the first iteration of <b>Cognoscente</b>, which contained over 270,000 interactions built from Interaction GeneRIFs compiled by NCBI<sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref8" title="NCBI. GeneRIF: Gene Reference Into Function, <http://www.ncbi.nlm.nih.gov/gene/about-generif> (2012)." id="ref-link-14">8</a></sup>, we noted that that at least 75 interactions relevant to early cardiac development were absent. We have thus added 75 interactions relevant to cardiac development that we manually collected from the literature (<a href="/srep/2012/121220/srep01011/full/srep01011.html#f1"> <b>Fig. 1</b> </a>). These interactions were not in the databases above, and were discovered through a manual search of the PubMed database for documented interactions with selected genes. This demonstrates that the NCBI Interaction GeneRIFs do not provide a complete catalogue of documented biomolecular interactions, and thus motivates a mechanism for user submissions. To encourage accurate knowledgebase submissions, users are asked to pre-submit interactions so that the submission software module can check for redundancies and report back to the user with the details about their submissions (<a href="/srep/2012/121220/srep01011/full/srep01011.html#f2"> <b>Fig. 2</b> </a>). This will reduce errors and redundancies in new knowledgebase submissions. Accountability and credit for new submissions is afforded by requiring that each submitter registers for an account. Each submission is tagged with the submitter&#x27;s name, and submitter names are shown in any query results that include that submitted entry.</p>								
												
					


							




		 		 	 	
												
			
			
	
	<div class="figure cleared" id="f1"
			>
		<figure>			<figcaption><span class="legend">
				Figure 1: Biomolecular interactions manually added to Cognoscente via a web-based submission portal.			</span></figcaption>
			
			<div class="item-content box">
				
																			  												
						
						<a href="/srep/2012/121220/srep01011/fig_tab/srep01011_F1.html" class="fig-link">								
		
																									
			
				
	<img src="/srep/2012/121220/srep01011/images_article/srep01011-f1.jpg" alt="Biomolecular interactions manually added to Cognoscente via a web-based submission portal." class="fig" style="width:600px;height:561px;" />
						</a>													
				<div class="description"><p>Interactions supplied by NCBI Interaction GeneRIFs were found to be incomplete with respect to documented interactions. We subsequently identified 75 interactions relevant to cardiac development in the literature, and manually added them to Cognoscente via a web-based submission portal.</p></div>
																					
																		
												
						<ul class="resources">
	
																			<li class="full"><a href="/srep/2012/121220/srep01011/fig_tab/srep01011_F1.html">Full size image (249 KB)</a></li>
												</ul>
													
			</div>
			
																													
								
				<nav>					<ul class="figure-nav">
												<li class="index"><a href="/srep/2012/121220/srep01011/fig_tab/srep01011_ft.html">Figures/tables index</a></li>
						<li class="next down-arrow"><a href="#f2" title="Submitting new biomolecular interactions in Cognoscente.">Next</a></li>					</ul>
				</nav>
				
											
		</figure>
	</div>

									
												
					


							




		 		 	 	
												
			
			
	
	<div class="figure cleared" id="f2"
			>
		<figure>			<figcaption><span class="legend">
				Figure 2: Submitting new biomolecular interactions in Cognoscente.			</span></figcaption>
			
			<div class="item-content box">
				
																			  												
						
						<a href="/srep/2012/121220/srep01011/fig_tab/srep01011_F2.html" class="fig-link">								
		
																									
			
				
	<img src="/srep/2012/121220/srep01011/images_article/srep01011-f2.jpg" alt="Submitting new biomolecular interactions in Cognoscente." class="fig" style="width:600px;height:651px;" />
						</a>													
				<div class="description"><p>User submissions interface for <b>Cognoscente</b> (A). Feedback is provided to the user about their proposed submission. If the submission appears to be redundant with an entry in the knowledgebase, the user receives a warning about the potential redundancy (B). Likewise, users receive a ‘green light’ if the proposed interaction does not already exist (C). In addition to these notifications, the system also retrieves gene information based on the user-supplied gene IDs, and retrieves the entire citation and abstract for the submitted PubMed IDs. These measures are expected to reduce redundancy and errors due in manual submissions.</p></div>
																					
																		
												
						<ul class="resources">
	
																			<li class="full"><a href="/srep/2012/121220/srep01011/fig_tab/srep01011_F2.html">Full size image (428 KB)</a></li>
												</ul>
													
			</div>
			
																																											
								
				<nav>					<ul class="figure-nav">
						<li class="prev up-arrow"><a href="#f1" title="Biomolecular interactions manually added to Cognoscente via a web-based submission portal.">Previous</a></li>						<li class="index"><a href="/srep/2012/121220/srep01011/fig_tab/srep01011_ft.html">Figures/tables index</a></li>
						<li class="next down-arrow"><a href="#f3" title="The Basic Query interface for Cognoscente allows queries of multiple genes, the option to show orthologous interactions, and an option to prescribe the complexity of the network to draw in the result (A).">Next</a></li>					</ul>
				</nav>
				
											
		</figure>
	</div>

	<h2>Search by gene symbol or Gene ID</h2><p><b>Cognoscente</b> queries are conducted using Entrez Gene IDs or Entrez Symbols (<a href="/srep/2012/121220/srep01011/full/srep01011.html#f3"> <b>Fig. 3</b> </a>). Unofficial gene synonyms are permitted in queries, as long as the synonyms unambiguously match the official gene symbols. Queries with Gene Symbols or unofficial synonyms require specifying the organism associated with the respective symbols.</p>								
												
					


							




		 		 	 	
												
			
			
	
	<div class="figure cleared" id="f3"
			>
		<figure>			<figcaption><span class="legend">
				Figure 3: The Basic Query interface for Cognoscente allows queries of multiple genes, the option to show orthologous interactions, and an option to prescribe the complexity of the network to draw in the result (A).			</span></figcaption>
			
			<div class="item-content box">
				
																			  												
						
						<a href="/srep/2012/121220/srep01011/fig_tab/srep01011_F3.html" class="fig-link">								
		
																									
			
				
	<img src="/srep/2012/121220/srep01011/images_article/srep01011-f3.jpg" alt="The Basic Query interface for Cognoscente allows queries of multiple genes, the option to show orthologous interactions, and an option to prescribe the complexity of the network to draw in the result (A)." class="fig" style="width:600px;height:555px;" />
						</a>													
				<div class="description"><p>A list of multiple genes can be divided into groups by separating the entered gene symbols with a forward slash (‘/’). In the results, the queried genes will be indicated as groups by different colored boxes around the network nodes. These groups (up to 3) may be named (B), which will be indicated in an automatically generated legend in the resulting graph. Results may be filtered by interaction type or by annotation status (B).</p></div>
																					
																		
												
						<ul class="resources">
	
																			<li class="full"><a href="/srep/2012/121220/srep01011/fig_tab/srep01011_F3.html">Full size image (488 KB)</a></li>
												</ul>
													
			</div>
			
																																											
								
				<nav>					<ul class="figure-nav">
						<li class="prev up-arrow"><a href="#f2" title="Submitting new biomolecular interactions in Cognoscente.">Previous</a></li>						<li class="index"><a href="/srep/2012/121220/srep01011/fig_tab/srep01011_ft.html">Figures/tables index</a></li>
						<li class="next down-arrow"><a href="#f4" title="Graphical query result for human genes ‘Nkx2-5, Mef2c’ in Cognoscente.">Next</a></li>					</ul>
				</nav>
				
											
		</figure>
	</div>

	<h2>Generation of GraphViz dot files for rendering networks</h2><p><b>GraphViz</b> is software that was developed at AT&amp;T for drawing graphs defined in the dot scripting language. <b>Cognoscente</b> leverages <b>GraphViz</b> by automatically generating dot script files for arbitrary graphs, and runs the scripts in <b>GraphViz</b>, which defines the layout and performs rendering of the graphs (<a href="/srep/2012/121220/srep01011/full/srep01011.html#f4"> <b>Fig. 4</b> </a>).</p>								
												
					


							




		 		 	 	
												
			
			
	
	<div class="figure cleared" id="f4"
			>
		<figure>			<figcaption><span class="legend">
				Figure 4: Graphical query result for human genes ‘<i>Nkx2-5</i>, <i>Mef2c</i>’ in Cognoscente.			</span></figcaption>
			
			<div class="item-content box">
				
																			  												
						
						<a href="/srep/2012/121220/srep01011/fig_tab/srep01011_F4.html" class="fig-link">								
		
																									
			
				
	<img src="/srep/2012/121220/srep01011/images_article/srep01011-f4.jpg" alt="Graphical query result for human genes ‘Nkx2-5, Mef2c’ in Cognoscente." class="fig" style="width:600px;height:488px;" />
						</a>													
				<div class="description"><p>This query was parameterized to search across all known orthologs for all primary (<i>radius</i> = 1) biomolecular interactions with either <i>Nkx2-5</i> or <i>Mef2c</i>. <b>Cognoscente</b> automatically generates legends (as shown here) that are inserted into the graphic.</p></div>
																					
																		
												
						<ul class="resources">
	
																			<li class="full"><a href="/srep/2012/121220/srep01011/fig_tab/srep01011_F4.html">Full size image (316 KB)</a></li>
												</ul>
													
			</div>
			
																																											
								
				<nav>					<ul class="figure-nav">
						<li class="prev up-arrow"><a href="#f3" title="The Basic Query interface for Cognoscente allows queries of multiple genes, the option to show orthologous interactions, and an option to prescribe the complexity of the network to draw in the result (A).">Previous</a></li>						<li class="index"><a href="/srep/2012/121220/srep01011/fig_tab/srep01011_ft.html">Figures/tables index</a></li>
						<li class="next down-arrow"><a href="#f5" title="Partial results for the query “cardiac muscle” using the Gene Lookup tool (http://vanburenlab.medicine.tamhsc.edu/gene_lookup.html).">Next</a></li>					</ul>
				</nav>
				
											
		</figure>
	</div>

	<h2>Hyperlinks</h2><p>Network specifications sent to <b>GraphViz</b> are configured to provide a mapping of nodes and edges for web interactivity, with mouse-over hints and links from the nodes to Entrez Gene and links from the edges to record pages containing interaction details and supporting references. The result page for a query in <b>Cognoscente</b> contains a rendered PNG version of the graph with mapped outlinks, the text of the dot file used to create that rendering (so the user can modify the dot file and create their own rendering with <b>GraphViz</b>, if needed), and a link to a high-quality PDF of the graph.</p><h2>Hypernodes</h2><p>Nodes in <b>Cognoscente</b> are hypernodes, meaning that each representation as a biomolecule (i.e. DNA, RNA, protein) is dependent on the type of edge that connects the node to another node. For example, a black line indicates a protein-protein interaction, and a red arrow indicates a protein-&#x0003E;DNA interaction. A node <i>X</i> that has both a black line connecting it to node <i>Y</i> and a red arrow pointing to it from node <i>Z</i> is thus representing both the protein <i>X</i> and the DNA of gene <i>X</i>, depending on the interaction represented.</p><h2>Orthology</h2><p><b>Cognoscente</b> renders networks that contain nodes that are stacked by orthology. The corpus of knowledge of biomolecular interactions differs in coverage for different model organisms. Organizing orthology as stacked nodes thus presents biomolecular interaction knowledge in a manner that facilitates comparisons between orthologs, and interactions known in one organism may provide clues to complement knowledge in another organism. The default behavior of <b>Cognoscente</b> will return results that use stacked nodes to represent orthology. However, the user can choose to confine results to a single organism.</p><h2>Lookup integration with StarNet</h2><p>There is a lack of knowledge about biomolecular interactions for many genes and gene products. Users of <b>Cognoscente</b> may thus query a gene for which there are no results. To expedite the use of <b>Cognoscente</b>, as well as to provide users a way to look up unfamiliar gene symbols, we created a Gene Lookup Tool (<a href="http://vanburenlab.medicine.tamhsc.edu/gene_lookup.html">http://vanburenlab.medicine.tamhsc.edu/gene_lookup.html</a>). This tool allows a keyword search that simultaneously searches Gene Descriptions, Gene Ontology (GO) Terms, official Gene Symbols, and unofficial Gene Synonyms (<a href="/srep/2012/121220/srep01011/full/srep01011.html#f5"> <b>Fig. 5</b> </a>). Queries with this tool return all Entrez Gene entries with a keyword match, and icons next to each matched entry indicate support by either <b>StarNet</b><sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref14" title="Jupiter, D., Chen, H. &amp; VanBuren, V. STARNET 2: a web-based tool for accelerating discovery of gene regulatory networks using microarray co-expression data. BMC Bioinformatics 10, 332 (2009)." id="ref-link-15">14</a>, <a href="/srep/2012/121220/srep01011/full/srep01011.html#ref15" title="Jupiter, D. C. &amp; VanBuren, V. A visual data mining tool that facilitates reconstruction of transcription regulatory networks. PLoS One 3, e1717 (2008)." id="ref-link-16">15</a></sup> (a yellow star) or <b>Cognoscente</b> (a white ‘C’ on a blue background). Clicking on one of these icons redirects the user to the respective tool and populates the parameters of that tool to query the matched Entrez Gene entry.</p>								
												
					


							




		 		 	 	
												
			
			
	
	<div class="figure cleared" id="f5"
			>
		<figure>			<figcaption><span class="legend">
				Figure 5: Partial results for the query “cardiac muscle” using the Gene Lookup tool (<a href="http://vanburenlab.medicine.tamhsc.edu/gene_lookup.html">http://vanburenlab.medicine.tamhsc.edu/gene_lookup.html</a>).			</span></figcaption>
			
			<div class="item-content box">
				
																			  												
						
						<a href="/srep/2012/121220/srep01011/fig_tab/srep01011_F5.html" class="fig-link">								
		
																									
			
				
	<img src="/srep/2012/121220/srep01011/images_article/srep01011-f5.jpg" alt="Partial results for the query “cardiac muscle” using the Gene Lookup tool (http://vanburenlab.medicine.tamhsc.edu/gene_lookup.html)." class="fig" style="width:600px;height:545px;" />
						</a>													
				<div class="description"><p>Full results include a search against gene descriptions, GO terms, official gene symbols and unofficial gene synonyms. Graphic symbols (as described in the legend) appear next to genes that are supported by StarNet or Cognoscente, respectively. These graphics may be clicked by the user to redirect to the appropriate query page, and automatically populate parameters on the query page for the respective gene.</p></div>
																					
																		
												
						<ul class="resources">
	
																			<li class="full"><a href="/srep/2012/121220/srep01011/fig_tab/srep01011_F5.html">Full size image (371 KB)</a></li>
												</ul>
													
			</div>
			
																																											
								
				<nav>					<ul class="figure-nav">
						<li class="prev up-arrow"><a href="#f4" title="Graphical query result for human genes ‘Nkx2-5, Mef2c’ in Cognoscente.">Previous</a></li>						<li class="index"><a href="/srep/2012/121220/srep01011/fig_tab/srep01011_ft.html">Figures/tables index</a></li>
						<li class="next down-arrow"><a href="#f6" title="Managing complexity with Cognoscente.">Next</a></li>					</ul>
				</nav>
				
											
		</figure>
	</div>

	<h2>Network radius</h2><p>Documented connectivity among genes and gene products produces networks that are scale-free. This means that most nodes have low connectivity, but a few nodes have very high connectivity. To accommodate these variations in complexity, different ‘network radii’ can be drawn with <b>Cognoscente</b>. The four options available are <i>radius</i> = 0, <i>radius</i> = 0 <i>with intermediates</i>, <i>radius</i> = 1, and <i>radius</i> = 2, in order of increasing complexity of the resultant network. Here radius is defined as the prescribed graph distance with respect to each query gene. In <b><i>radius</i> = 0</b> networks, only interactions between members of the query list are shown. In <b><i>radius</i> = 0 <i>with intermediates</i></b> networks, only interactions between members of the query list and intermediates that interact with at least two list members are shown. In <b><i>radius</i> = 1</b> networks, all known interactions with members of the query list are shown. Finally, in <b><i>radius</i> = 2</b> networks, all <b><i>radius</i> = 1</b> interactions and all interactions with <b><i>radius</i> = 1</b> interactants are shown. As we define it, <i>network radius</i> is different from the graph theoretic concept of radius (which applies to distances in an entire graph), as our definition is with respect to the query genes and their documented interactions. For multiple query genes, the graph theoretic radius can thus be higher than the prescribed network radius because multiple graphs (one for each query gene) are combined in a composite graph. Likewise, the graph theoretic radius can be smaller than the prescribed network radius where there are no documented interactions for a query gene.</p><h2>Filtering queries</h2><p>In the ‘Group Names and Filters’ panel of the <b>Cognoscente</b> interface, users can exclude different types of interactions, or exclude entries based on annotation status. For example, a user can configure a query to only return results for protein-protein interactions, or only genetic interactions, or some arbitrary combination of interaction types.</p><h2>Multi-gene searches</h2><p>Queries in <b>Cognoscente</b> may be composed of one or more genes, and results are merged into a single graph. This offers great flexibility and power for composing queries that bridge multiple pathways. One useful application of <b>Cognoscente</b> is that queries of selected gene sets (that may be either functionally or phenotypically related) can be used to reveal interacting intermediates between two genes/gene products of interest for further study.</p><h2>Query groups</h2><p>A comma-separated list of query genes will be treated as a single ‘query group’. However, up to three different query groups may be specified by using a forward-slash (‘/’) separator between query groups. This has the effect of color-coding the query group members in the results, and indicating this color-coding in a legend. The default group names (‘Query Group 1’, ‘Query Group 2’, and ‘Query Group 3’) may be changed in the ‘Group Names and Filters’ panel of the <b>Cognoscente</b> interface.</p><h2>Managing complexity</h2><p>Although visualizations of complex knowledge leverage our ability to comprehend visual complexity, this ability has limitations that vary with the context of the knowledge. Comprehension may thus be aided by exercising some control over the amount of complexity expressed in a visualization. The most obvious way of controlling complexity in Cognoscente visualizations is to query only one or a few genes rather than many genes. Complexity in Cognoscente visualizations is also parameterized in two main ways: (1) by either choosing to query all orthologs or confining the search to a single organism, and (2) by prescribing the network radius to retrieve in the query. This is illustrated with an example query for genes that were found to be important for the specification of inducible pluripotent stem cells (iPSCs). Induction of stem cells from human adult cells was first achieved in 2007<sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref16" title="Takahashi, K. et al. Induction of pluripotent stem cells from adult human fibroblasts by defined factors. Cell 131, 861–872 (2007)." id="ref-link-17">16</a>, <a href="/srep/2012/121220/srep01011/full/srep01011.html#ref17" title="Yu, J. et al. Induced pluripotent stem cell lines derived from human somatic cells. Science 318, 1917–1920 (2007)." id="ref-link-18">17</a></sup>. Yamanaka and colleagues induced transformation into pluripotent stems cells using Oct4, Sox2, Klf4, and c-Myc<sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref16" title="Takahashi, K. et al. Induction of pluripotent stem cells from adult human fibroblasts by defined factors. Cell 131, 861–872 (2007)." id="ref-link-19">16</a></sup>, whereas Thomson and colleagues used Oct4, Sox2, Nanog and Lin28A<sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref17" title="Yu, J. et al. Induced pluripotent stem cell lines derived from human somatic cells. Science 318, 1917–1920 (2007)." id="ref-link-20">17</a></sup>. We queried “Oct4, Sox2, Klf4, Myc/Oct4, Sox2, Nanog, Lin28A” in Cognoscente, and varied the parameterization of complexity in four ways to demonstrate how visual complexity may be controlled in query results (<a href="/srep/2012/121220/srep01011/full/srep01011.html#f6"> <b>Fig. 6</b> </a>). In the first query, all primary (<i>radius</i> = 1) interactants are visualized across orthologs, yielding a relatively complex graph with hundreds of nodes. This complexity is greatly reduced by constraining results to one species (<i>Homo sapiens</i>) and by constraining the visualized interactions to those between members of the query list and to those with interactants that have interactions with at least two members of the query list (our definition of ‘intermediate’). This reduces the number of nodes in the resulting network from several hundred to several dozen, and thus greatly increases comprehensibility. However, the more complex modalities for graphing these query results may also be desirable for their greater completeness, or in contexts where there are a smaller number of documented interactions for a given query.</p>								
												
					


							




		 		 	 	
												
			
			
	
	<div class="figure cleared" id="f6"
			>
		<figure>			<figcaption><span class="legend">
				Figure 6: Managing complexity with Cognoscente.			</span></figcaption>
			
			<div class="item-content box">
				
																			  												
						
						<a href="/srep/2012/121220/srep01011/fig_tab/srep01011_F6.html" class="fig-link">								
		
																									
			
				
	<img src="/srep/2012/121220/srep01011/images_article/srep01011-f6.jpg" alt="Managing complexity with Cognoscente." class="fig" style="width:600px;height:441px;" />
						</a>													
				<div class="description"><p>iPSC genes <i>Oct4</i> (<i>Pou5f1</i>), <i>Sox2</i>, <i>c-Myc</i> (<i>Myc</i>), <i>Klf4</i>, <i>Nanog</i>, and <i>Lin28A</i> were queried with <b>Cognoscente</b> using different approaches. In the upper left panel, all primary (<i>radius</i> = 1) interactions are shown across all orthologs. In the upper right panel, only <i>human</i> primary interactions are shown. In the lower left panel, <i>radius</i> = 0 interactions (those between members of the query list) and ‘intermediates’ (those where the interactant has interactions with at least two members of the query list) are shown across all orthologs. Finally, in the lower right panel, only <i>human</i> <i>radius</i> = 0 <i>interactions with intermediates</i> are shown, which eliminates hundreds of nodes as compared with the upper left panel.</p></div>
																					
																		
												
						<ul class="resources">
	
																			<li class="full"><a href="/srep/2012/121220/srep01011/fig_tab/srep01011_F6.html">Full size image (512 KB)</a></li>
												</ul>
													
			</div>
			
																													
								
				<nav>					<ul class="figure-nav">
						<li class="prev up-arrow"><a href="#f5" title="Partial results for the query “cardiac muscle” using the Gene Lookup tool (http://vanburenlab.medicine.tamhsc.edu/gene_lookup.html).">Previous</a></li>						<li class="index"><a href="/srep/2012/121220/srep01011/fig_tab/srep01011_ft.html">Figures/tables index</a></li>
											</ul>
				</nav>
				
											
		</figure>
	</div>

	<h2>Future software maintenance</h2><p><b>Cognoscente</b> is maintained, upgraded and updated on an ongoing basis. Two main strategies are used to promote the sustainability of the knowledgebase. These are (1) automatically leveraging the Interaction GeneRIFs using a script that identifies new GeneRIFs catalogued by NCBI, processes those interactions, and adds them to the knowledgebase on a weekly schedule, and (2) providing a web portal for user submissions of interactions to complement Interaction GeneRIFs. These strategies were selected because they require very little in the way of centralized maintenance and are low cost, and together work to ensure that the knowledgebase is kept up-to-date. We plan to add web services to allow programmatic interactions with Cognoscente, and add support for drug and target interactions.</p><h2>Data access</h2><p>The <b>Cognoscente</b> web application can be accessed at <a href="http://vanburenlab.medicine.tamhsc.edu/cognoscente.html">http://vanburenlab.medicine.tamhsc.edu/cognoscente.html</a>. <b>Cognoscente</b> usage is freely licensed without warranty. Downloading the <b>Cognoscente</b> interaction table is freely available under the Creative Commons Attribution 3.0 Unported License (CC-BY-3.0) license at <a href="http://vanburenlab.medicine.tamhsc.edu/cognoscente_downloads.shtml">http://vanburenlab.medicine.tamhsc.edu/cognoscente_downloads.shtml</a>. The downloadable interaction table is updated weekly, and an archive of previous versions is maintained at this site.</p>
							</div>
																					</div></section>

		
						
																								
												
									<section><div class="section   expanded" id="discussion">

							<h1 class="section-heading toggle">Discussion</h1>
						<div class="content">
																		<nav><ul class="section-nav cleared">

																																																						<li class="first"><a href="#introduction">Introduction</a></li>
																																													<li><a href="#results">Results</a></li>
																																																					<li class="current"><span>Discussion</span></li>	
																																													<li><a href="#methods">Methods</a></li>
																																													<li><a href="#references">References</a></li>
																																													<li><a href="#acknowledgments">Acknowledgements</a></li>
																																													<li><a href="#author-information">Author information</a></li>
																																													<li><a href="#supplementary-information">Supplementary information</a></li>
																																													<li class="last"><a href="#comments">Comments</a></li>
																		
					</ul></nav>

								
								
				<p>There are four main ways that we envision Cognoscente to be leveraged by biomedical scientists: 1. as a tool for quickly retrieving known interactions across orthologs for a particular gene, 2. to help interpret the results of high-content data approaches such as microarray analysis, 3. to build interactomes for arbitrary lists of genes, and 4. to aid the inference of biomolecular pathways for regulation and signaling. <a href="/srep/2012/121220/srep01011/full/srep01011.html#supplementary-information">Supplementary Figure S1</a> shows an example of using <b>Cognoscente</b> to aid the interpretation of high-content data. Briefly, we retrieved the list of differentially expressed genes and a list of TRAP-ranked regulators from Kwong and colleagues’ recent melanoma study (<a href="/srep/2012/121220/srep01011/full/srep01011.html#supplementary-information">Tables S1</a> and <a href="/srep/2012/121220/srep01011/full/srep01011.html#supplementary-information">S5</a>, respectively, from their publication)<sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref18" title="Kwong, L. N. et al. Oncogenic NRAS signaling differentially regulates survival and proliferation in melanoma. Nat Med 18, 1503–1510 (2012)." id="ref-link-21">18</a></sup>. These genes were queried in <b>Cognoscente</b> in groups labeled ‘Up with respect to control’, ‘Down with respect to control’, and ‘TRAP-ranked regulators’, parameterized with <i>radius</i> = 0 <i>with intermediates</i>. The resultant graph reveals clues about potential regulatory relationships that induce the observed differential expressions (<a href="/srep/2012/121220/srep01011/full/srep01011.html#supplementary-information">Supplementary Fig. S1</a>). <a href="/srep/2012/121220/srep01011/full/srep01011.html#supplementary-information">Supplementary Figures S2 and S3</a> show an example of using <b>Cognoscente</b> to build an arbitrary interactome. The mechanisms by which alcohol consumption inhibits bone healing after fracture are not well defined. We used the Lookup tool provided with <b>Cognoscente</b> to retrieve a list of all human genes annotated with a Gene Ontology (GO) term that contains the word “alcohol”, and another list of all human genes annotated with a GO term containing the word “bone” (<a href="/srep/2012/121220/srep01011/full/srep01011.html#supplementary-information">Supplementary Table S1</a>). These labeled groups were queried in Cognoscente and parameterized with <i>radius</i> = 0 and <i>radius</i> = 0 <i>with intermediates</i>, respectively (<a href="/srep/2012/121220/srep01011/full/srep01011.html#supplementary-information">Supplementary Figs. S2 and S3</a>). The results reveal connections between alcohol metabolism and bone metabolism that may be relevant to the effects of alcohol consumption on bone healing. The construction of arbitrary interactomes from genes implicated in a particular disease or developmental process will provide new hypotheses about intermediates that may also be relevant to the respective developmental process or disease (e.g. <a href="/srep/2012/121220/srep01011/full/srep01011.html#f6"> <b>Fig. 6</b> </a>). These interactomes may in turn provide insights for the inference of relevant pathways.</p><p>We previously developed <b>StarNet</b>, which is a visual database for correlations in pre-selected microarray data<sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref14" title="Jupiter, D., Chen, H. &amp; VanBuren, V. STARNET 2: a web-based tool for accelerating discovery of gene regulatory networks using microarray co-expression data. BMC Bioinformatics 10, 332 (2009)." id="ref-link-22">14</a>, <a href="/srep/2012/121220/srep01011/full/srep01011.html#ref15" title="Jupiter, D. C. &amp; VanBuren, V. A visual data mining tool that facilitates reconstruction of transcription regulatory networks. PLoS One 3, e1717 (2008)." id="ref-link-23">15</a></sup>. <b>StarNet</b> allows queries of a single gene from one of ten organisms, and draws a correlation network radiating from that gene from precompiled microarray data. The future development of <b>Cognoscente</b> will include better integration with <b>StarNet</b><sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref14" title="Jupiter, D., Chen, H. &amp; VanBuren, V. STARNET 2: a web-based tool for accelerating discovery of gene regulatory networks using microarray co-expression data. BMC Bioinformatics 10, 332 (2009)." id="ref-link-24">14</a>, <a href="/srep/2012/121220/srep01011/full/srep01011.html#ref15" title="Jupiter, D. C. &amp; VanBuren, V. A visual data mining tool that facilitates reconstruction of transcription regulatory networks. PLoS One 3, e1717 (2008)." id="ref-link-25">15</a></sup>, including overlay of known correlations in the interaction map, and inferential modeling combining interaction knowledge from <b>Cognoscente</b> and expression correlations from <b>StarNet</b>. This improved integration will allow users to directly combine interaction knowledge and expression correlation data in models that infer causal relationships, thus creating specific well-formed hypotheses from existing knowledge and data.</p>
							</div>
																					</div></section>

		
						
																								
												
									<section><div class="section   expanded" id="methods">

							<h1 class="section-heading toggle">Methods</h1>
						<div class="content">
																		<nav><ul class="section-nav cleared">

																																																						<li class="first"><a href="#introduction">Introduction</a></li>
																																													<li><a href="#results">Results</a></li>
																																													<li><a href="#discussion">Discussion</a></li>
																																																					<li class="current"><span>Methods</span></li>	
																																													<li><a href="#references">References</a></li>
																																													<li><a href="#acknowledgments">Acknowledgements</a></li>
																																													<li><a href="#author-information">Author information</a></li>
																																													<li><a href="#supplementary-information">Supplementary information</a></li>
																																													<li class="last"><a href="#comments">Comments</a></li>
																		
					</ul></nav>

								
								
				<h2>Source knowledge</h2><p><b>Cognoscente</b> was built and is routinely updated by extracting knowledge from NCBI&#x27;s Interaction GeneRIFs<sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref8" title="NCBI. GeneRIF: Gene Reference Into Function, <http://www.ncbi.nlm.nih.gov/gene/about-generif> (2012)." id="ref-link-26">8</a></sup>. The bulk of the interactions collected from NCBI in <b>Cognoscente</b> were originally extracted from HPRD<sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref6" title="Keshava Prasad, T. S. et al. Human Protein Reference Database--2009 update. Nucleic Acids Res 37, D767–772 (2009)." id="ref-link-27">6</a></sup>, BIND<sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref10" title="Willis, R. C. &amp; Hogue, C. W. Searching, viewing, and visualizing data in the Biomolecular Interaction Network Database (BIND). Curr Protoc Bioinformatics Chapter 8, Unit 8 9 (2006)." id="ref-link-28">10</a></sup>, BioGrid<sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref2" title="Breitkreutz, B. J. et al. The BioGRID Interaction Database: 2008 update. Nucleic Acids Res 36, D637–640 (2008)." id="ref-link-29">2</a></sup>, or EcoCyc<sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref5" title="Keseler, I. M. et al. EcoCyc: a comprehensive view of Escherichia coli biology. Nucleic Acids Res 37, D464–470 (2009)." id="ref-link-30">5</a></sup>. All interactions in the <b>Cognoscente</b> knowledge base are supported by one or more PubMed references. Documented knowledge of interactions that are not already extracted by this method may be added at any time by registered users (see <b>Results</b>, <i>Crowdsourcing</i>).</p><h2>Database schema</h2><p>The Cognoscente knowledgebase is presently maintained on the VanBuren Lab web server as a MySQL database. The main database table of interactions has fields based on the table of Interaction GeneRIFs downloaded from NCBI. The database field structure is identical to the fields in the downloadable table of interactions on the Cognoscente web site (see <b>Results</b>, <i>Data Access</i>). Other tables in the knowledgebase include a table for registered <i>users</i>, a <i>history</i> table for tracking user alterations to the database to allow for quality assurance checks and the ability to revert to an earlier state of the database, and the <i>gene_info</i> and <i>homologene</i> tables downloaded from NCBI.</p><h2>Program logic</h2><p>The web interface was encoded with HTML and Javascript, and interacts with a Perl CGI script that calls the main logic script, which is also written in Perl. The main logic script calls directly on <b>MySQL</b> for database queries and on <b>GraphViz</b> to construct visualizations of graphs. Orthologies were defined in <b>Cognoscente</b> using <b>HomoloGene</b><sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref19" title="Wheeler, D. L. et al. Database resources of the National Center for Biotechnology Information. Nucleic Acids Res 29, 11–16 (2001)." id="ref-link-31">19</a></sup>. For queries in <b>Cognoscente</b> that include orthologous genes (the default behavior), a lookup is performed for genes that have the same HomoloGene ID as each queried gene. Orthologous genes are shown in composite (stacked) nodes only when there is a documented interaction for those genes in the knowledgebase and that interaction will appear in the resultant graph. Graph edges in the web-rendered image and <b>Cognoscente</b> ID numbers in the respective table of interactions are linked to record pages for each interaction, which include citations and abstracts extracted from PubMed using <b>RefSense</b><sup><a href="/srep/2012/121220/srep01011/full/srep01011.html#ref20" title="Arvestad, L. RefSense, <https://github.com/arvestad/RefSense> (2012)." id="ref-link-32">20</a></sup>.</p><h2>Drawing graphs</h2><p>Graphs are drawn in <b>Cognoscente</b> using <b>GraphViz</b>. The <i>sfdp</i> algorithm for layouts is used to produce graphs in GIF format coupled with CMAP mapping to provide clickable graphs in the web interface, and to produce PDFs of the graph.</p>
																	
										
										</div>
																					</div></section>

		
						
																								
												
									<section><div class="section   expanded" id="references">

							<h1 class="section-heading toggle">References</h1>
						<div class="content">
																		<nav><ul class="section-nav cleared">

																																																						<li class="first"><a href="#introduction">Introduction</a></li>
																																													<li><a href="#results">Results</a></li>
																																													<li><a href="#discussion">Discussion</a></li>
																																													<li><a href="#methods">Methods</a></li>
																																																					<li class="current"><span>References</span></li>	
																																													<li><a href="#acknowledgments">Acknowledgements</a></li>
																																													<li><a href="#author-information">Author information</a></li>
																																													<li><a href="#supplementary-information">Supplementary information</a></li>
																																													<li class="last"><a href="#comments">Comments</a></li>
																		
					</ul></nav>

								
								
												
												
					

	
<ol class="references">
														
		<li id="ref1"><span class="vcard author"><span class="fn">Gansner, E.</span></span> &amp; <span class="vcard author"><span class="fn">North, S.</span></span> <span class="title">An open graph visualization system and its applications to software engineering</span>. <span class="source-title">Softw. Pract. Exper.</span> <span class="volume">00</span>, <span class="start-page">1</span>–<span class="end-page">5</span> (<span class="year">1999</span>).
				</li>
		
								
														
		<li id="ref2"><span class="vcard author"><span class="fn">Breitkreutz, B. J.</span></span> <i>et al.</i> <span class="title">The BioGRID Interaction Database: 2008 update</span>. <span class="source-title">Nucleic Acids Res</span> <span class="volume">36</span>, <span class="start-page">D637</span>–<span class="end-page">640</span> (<span class="year">2008</span>).
					<ul class="cleared">
																																				<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXhtVSrsLk%3D&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=96ada44d4a9439f90ac7071356b00484">CAS</a></li>
																															 																												<li>	
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000252545400114&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=18000002&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="http://dx.doi.org/10.1093/nar/gkm1001">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1093/nar/gkm1001&amp;rft_id=info:pmid/18000002&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Breitkreutz&amp;rft.aufirst=B. J.&amp;rft.jtitle=Nucleic Acids Res&amp;rft.volume=36&amp;rft.spage=D637&amp;rft.epage=640&amp;rft.date=2008&amp;rft.atitle=The BioGRID Interaction Database: 2008 update&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1093/nar/gkm1001&amp;id=pmid:18000002&amp;genre=article&amp;aulast=Breitkreutz&amp;aufirst=B. J.&amp;title=Nucleic Acids Res&amp;volume=36&amp;spage=D637&amp;epage=640&amp;date=2008&amp;atitle=The BioGRID Interaction Database: 2008 update&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref3"><span class="vcard author"><span class="fn">Jensen, L. J.</span></span> <i>et al.</i> <span class="title">STRING 8--a global view on proteins and their functional interactions in 630 organisms</span>. <span class="source-title">Nucleic Acids Res</span> <span class="volume">37</span>, <span class="start-page">D412</span>–<span class="end-page">416</span> (<span class="year">2009</span>).
					<ul class="cleared">
																																				<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXhsFejt7jL&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=7c07e1cb4ac8ba71d03555aff14e3944">CAS</a></li>
																				 																												<li>	
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000261906200075&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=18940858&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="http://dx.doi.org/10.1093/nar/gkn760">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1093/nar/gkn760&amp;rft_id=info:pmid/18940858&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Jensen&amp;rft.aufirst=L. J.&amp;rft.jtitle=Nucleic Acids Res&amp;rft.volume=37&amp;rft.spage=D412&amp;rft.epage=416&amp;rft.date=2009&amp;rft.atitle=STRING 8--a global view on proteins and their functional interactions in 630 organisms&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1093/nar/gkn760&amp;id=pmid:18940858&amp;genre=article&amp;aulast=Jensen&amp;aufirst=L. J.&amp;title=Nucleic Acids Res&amp;volume=37&amp;spage=D412&amp;epage=416&amp;date=2009&amp;atitle=STRING 8--a global view on proteins and their functional interactions in 630 organisms&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref4"><span class="vcard author"><span class="fn">Kerrien, S.</span></span> <i>et al.</i> <span class="title">The IntAct molecular interaction database in 2012</span>. <span class="source-title">Nucleic Acids Res</span> <span class="volume">40</span>, <span class="start-page">D841</span>–<span class="end-page">846</span> (<span class="year">2012</span>).
					<ul class="cleared">
																																	<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC3MXhs12htbvI&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=5dd56e74909559806425dd4581ea9083">CAS</a></li>
																				 																												<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=22121220&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="http://dx.doi.org/10.1093/nar/gkr1088">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1093/nar/gkr1088&amp;rft_id=info:pmid/22121220&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Kerrien&amp;rft.aufirst=S.&amp;rft.jtitle=Nucleic Acids Res&amp;rft.volume=40&amp;rft.spage=D841&amp;rft.epage=846&amp;rft.date=2012&amp;rft.atitle=The IntAct molecular interaction database in 2012&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1093/nar/gkr1088&amp;id=pmid:22121220&amp;genre=article&amp;aulast=Kerrien&amp;aufirst=S.&amp;title=Nucleic Acids Res&amp;volume=40&amp;spage=D841&amp;epage=846&amp;date=2012&amp;atitle=The IntAct molecular interaction database in 2012&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref5"><span class="vcard author"><span class="fn">Keseler, I. M.</span></span> <i>et al.</i> <span class="title">EcoCyc: a comprehensive view of Escherichia coli biology</span>. <span class="source-title">Nucleic Acids Res</span> <span class="volume">37</span>, <span class="start-page">D464</span>–<span class="end-page">470</span> (<span class="year">2009</span>).
					<ul class="cleared">
																																				<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXhsFejtLvO&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=be1fb35f6ac864adeb1d853e3ee5b931">CAS</a></li>
																				 																												<li>	
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000261906200084&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=18974181&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="http://dx.doi.org/10.1093/nar/gkn751">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1093/nar/gkn751&amp;rft_id=info:pmid/18974181&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Keseler&amp;rft.aufirst=I. M.&amp;rft.jtitle=Nucleic Acids Res&amp;rft.volume=37&amp;rft.spage=D464&amp;rft.epage=470&amp;rft.date=2009&amp;rft.atitle=EcoCyc: a comprehensive view of Escherichia coli biology&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1093/nar/gkn751&amp;id=pmid:18974181&amp;genre=article&amp;aulast=Keseler&amp;aufirst=I. M.&amp;title=Nucleic Acids Res&amp;volume=37&amp;spage=D464&amp;epage=470&amp;date=2009&amp;atitle=EcoCyc: a comprehensive view of Escherichia coli biology&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref6"><span class="vcard author"><span class="fn">Keshava Prasad, T. S.</span></span> <i>et al.</i> <span class="title">Human Protein Reference Database--2009 update</span>. <span class="source-title">Nucleic Acids Res</span> <span class="volume">37</span>, <span class="start-page">D767</span>–<span class="end-page">772</span> (<span class="year">2009</span>).
					<ul class="cleared">
																																				<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXhsFejtLvL&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=98d644b15fadc0d5d0b179177d91a405">CAS</a></li>
																				 																												<li>	
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000261906200134&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=18988627&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="http://dx.doi.org/10.1093/nar/gkn892">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1093/nar/gkn892&amp;rft_id=info:pmid/18988627&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Keshava Prasad&amp;rft.aufirst=T. S.&amp;rft.jtitle=Nucleic Acids Res&amp;rft.volume=37&amp;rft.spage=D767&amp;rft.epage=772&amp;rft.date=2009&amp;rft.atitle=Human Protein Reference Database--2009 update&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1093/nar/gkn892&amp;id=pmid:18988627&amp;genre=article&amp;aulast=Keshava Prasad&amp;aufirst=T. S.&amp;title=Nucleic Acids Res&amp;volume=37&amp;spage=D767&amp;epage=772&amp;date=2009&amp;atitle=Human Protein Reference Database--2009 update&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref7"><span class="vcard author"><span class="fn">Licata, L.</span></span> <i>et al.</i> <span class="title">MINT, the molecular interaction database: 2012 update</span>. <span class="source-title">Nucleic Acids Res</span> <span class="volume">40</span>, <span class="start-page">D857</span>–<span class="end-page">861</span> (<span class="year">2012</span>).
					<ul class="cleared">
																																	<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC3MXhs12hurbE&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=3d47b64d3f7b05788e8c7bc96950455e">CAS</a></li>
																				 																												<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=22096227&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="http://dx.doi.org/10.1093/nar/gkr930">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1093/nar/gkr930&amp;rft_id=info:pmid/22096227&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Licata&amp;rft.aufirst=L.&amp;rft.jtitle=Nucleic Acids Res&amp;rft.volume=40&amp;rft.spage=D857&amp;rft.epage=861&amp;rft.date=2012&amp;rft.atitle=MINT, the molecular interaction database: 2012 update&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1093/nar/gkr930&amp;id=pmid:22096227&amp;genre=article&amp;aulast=Licata&amp;aufirst=L.&amp;title=Nucleic Acids Res&amp;volume=40&amp;spage=D857&amp;epage=861&amp;date=2012&amp;atitle=MINT, the molecular interaction database: 2012 update&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref8">NCBI. <cite class="book">GeneRIF: Gene Reference Into Function</cite>, &#60;<a href="http://www.ncbi.nlm.nih.gov/gene/about-generif">http://www.ncbi.nlm.nih.gov/gene/about-generif</a>&#x0003E; (2012).
				</li>
		
								
														
		<li id="ref9"><span class="vcard author"><span class="fn">von Mering, C.</span></span> <i>et al.</i> <span class="title">STRING: a database of predicted functional associations between proteins</span>. <span class="source-title">Nucleic Acids Res</span> <span class="volume">31</span>, <span class="start-page">258</span>–<span class="end-page">261</span> (<span class="year">2003</span>).
					<ul class="cleared">
																																				<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3sXhvFSmsrg%3D&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=320c624961d292a081b7f65b3190ff37">CAS</a></li>
																				 																												<li>	
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000181079700060&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=12519996&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="http://dx.doi.org/10.1093/nar/gkg034">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1093/nar/gkg034&amp;rft_id=info:pmid/12519996&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=von Mering&amp;rft.aufirst=C.&amp;rft.jtitle=Nucleic Acids Res&amp;rft.volume=31&amp;rft.spage=258&amp;rft.epage=261&amp;rft.date=2003&amp;rft.atitle=STRING: a database of predicted functional associations between proteins&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1093/nar/gkg034&amp;id=pmid:12519996&amp;genre=article&amp;aulast=von Mering&amp;aufirst=C.&amp;title=Nucleic Acids Res&amp;volume=31&amp;spage=258&amp;epage=261&amp;date=2003&amp;atitle=STRING: a database of predicted functional associations between proteins&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref10"><span class="vcard author"><span class="fn">Willis, R. C.</span></span> &amp; <span class="vcard author"><span class="fn">Hogue, C. W.</span></span> <span class="title">Searching, viewing, and visualizing data in the Biomolecular Interaction Network Database (BIND)</span>. <span class="source-title">Curr Protoc Bioinformatics</span> <b>Chapter</b> <span class="volume">8</span>, Unit 8 9 (<span class="year">2006).</span>
				</li>
		
								
														
		<li id="ref11"><span class="vcard author"><span class="fn">Xenarios, I.</span></span> <i>et al.</i> <span class="title">DIP, the Database of Interacting Proteins: a research tool for studying cellular networks of protein interactions</span>. <span class="source-title">Nucleic Acids Res</span> <span class="volume">30</span>, <span class="start-page">303</span>–<span class="end-page">305</span> (<span class="year">2002</span>).
					<ul class="cleared">
																																				<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD38Xht12rs78%3D&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=85a176f497f0afe8bd632c9208b2eaf9">CAS</a></li>
																				 																												<li>	
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000173077100081&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=11752321&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="http://dx.doi.org/10.1093/NAR/30.1.303">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1093/NAR/30.1.303&amp;rft_id=info:pmid/11752321&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Xenarios&amp;rft.aufirst=I.&amp;rft.jtitle=Nucleic Acids Res&amp;rft.volume=30&amp;rft.spage=303&amp;rft.epage=305&amp;rft.date=2002&amp;rft.atitle=DIP, the Database of Interacting Proteins: a research tool for studying cellular networks of protein interactions&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1093/NAR/30.1.303&amp;id=pmid:11752321&amp;genre=article&amp;aulast=Xenarios&amp;aufirst=I.&amp;title=Nucleic Acids Res&amp;volume=30&amp;spage=303&amp;epage=305&amp;date=2002&amp;atitle=DIP, the Database of Interacting Proteins: a research tool for studying cellular networks of protein interactions&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref12"><span class="vcard author"><span class="fn">Shannon, P.</span></span> <i>et al.</i> <span class="title">Cytoscape: a software environment for integrated models of biomolecular interaction networks</span>. <span class="source-title">Genome Res</span> <span class="volume">13</span>, <span class="start-page">2498</span>–<span class="end-page">2504</span> (<span class="year">2003</span>).
					<ul class="cleared">
																																				<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3sXovFWrtr4%3D&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=242d576989075dadcc612118b29c74f2">CAS</a></li>
																				 																												<li>	
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000186357000016&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=14597658&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="http://dx.doi.org/10.1101/gr.1239303">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1101/gr.1239303&amp;rft_id=info:pmid/14597658&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Shannon&amp;rft.aufirst=P.&amp;rft.jtitle=Genome Res&amp;rft.volume=13&amp;rft.spage=2498&amp;rft.epage=2504&amp;rft.date=2003&amp;rft.atitle=Cytoscape: a software environment for integrated models of biomolecular interaction networks&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1101/gr.1239303&amp;id=pmid:14597658&amp;genre=article&amp;aulast=Shannon&amp;aufirst=P.&amp;title=Genome Res&amp;volume=13&amp;spage=2498&amp;epage=2504&amp;date=2003&amp;atitle=Cytoscape: a software environment for integrated models of biomolecular interaction networks&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref13"><span class="vcard author"><span class="fn">Smoot, M. E.</span></span>, <span class="vcard author"><span class="fn">Ono, K.</span></span>, <span class="vcard author"><span class="fn">Ruscheinski, J.</span></span>, <span class="vcard author"><span class="fn">Wang, P. L.</span></span> &amp; <span class="vcard author"><span class="fn">Ideker, T.</span></span> <span class="title">Cytoscape 2.8: new features for data integration and network visualization</span>. <span class="source-title">Bioinformatics</span> <span class="volume">27</span>, <span class="start-page">431</span>–<span class="end-page">432</span> (<span class="year">2011</span>).
					<ul class="cleared">
																																							<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC3MXhs1GisL0%3D&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=0cbaec96c74142653e77a5a734100e68">CAS</a></li>
																				 																												<li>	
											<a href="http://adsabs.harvard.edu/abs/2011qbi4.conf..451S">ADS</a></li>
																<li>	
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000286991300023&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=21149340&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="http://dx.doi.org/10.1093/bioinformatics/btq675">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1093/bioinformatics/btq675&amp;rft_id=info:pmid/21149340&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Smoot&amp;rft.aufirst=M. E.&amp;rft.jtitle=Bioinformatics&amp;rft.volume=27&amp;rft.spage=431&amp;rft.epage=432&amp;rft.date=2011&amp;rft.atitle=Cytoscape 2.8: new features for data integration and network visualization&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1093/bioinformatics/btq675&amp;id=pmid:21149340&amp;genre=article&amp;aulast=Smoot&amp;aufirst=M. E.&amp;title=Bioinformatics&amp;volume=27&amp;spage=431&amp;epage=432&amp;date=2011&amp;atitle=Cytoscape 2.8: new features for data integration and network visualization&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref14"><span class="vcard author"><span class="fn">Jupiter, D.</span></span>, <span class="vcard author"><span class="fn">Chen, H.</span></span> &amp; <span class="vcard author"><span class="fn">VanBuren, V.</span></span> <span class="title">STARNET 2: a web-based tool for accelerating discovery of gene regulatory networks using microarray co-expression data</span>. <span class="source-title">BMC Bioinformatics</span> <span class="volume">10</span>, <span class="start-page">332</span> (<span class="year">2009</span>).
					<ul class="cleared">
																																	<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1MXhtlWqtb3L&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=435f60548a0181df3fd06e1a189ff766">CAS</a></li>
																				 																												<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=19828039&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="http://dx.doi.org/10.1186/1471-2105-10-332">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1186/1471-2105-10-332&amp;rft_id=info:pmid/19828039&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Jupiter&amp;rft.aufirst=D.&amp;rft.jtitle=BMC Bioinformatics&amp;rft.volume=10&amp;rft.spage=332&amp;rft.epage=&amp;rft.date=2009&amp;rft.atitle=STARNET 2: a web-based tool for accelerating discovery of gene regulatory networks using microarray co-expression data&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1186/1471-2105-10-332&amp;id=pmid:19828039&amp;genre=article&amp;aulast=Jupiter&amp;aufirst=D.&amp;title=BMC Bioinformatics&amp;volume=10&amp;spage=332&amp;epage=&amp;date=2009&amp;atitle=STARNET 2: a web-based tool for accelerating discovery of gene regulatory networks using microarray co-expression data&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref15"><span class="vcard author"><span class="fn">Jupiter, D. C.</span></span> &amp; <span class="vcard author"><span class="fn">VanBuren, V.</span></span> <span class="title">A visual data mining tool that facilitates reconstruction of transcription regulatory networks</span>. <span class="source-title">PLoS One</span> <span class="volume">3</span>, <span class="start-page">e1717</span> (<span class="year">2008</span>).
					<ul class="cleared">
																																				<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD1cXksFensLc%3D&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=be3cf15d18e82fa51b3abbc9997c1023">CAS</a></li>
																				 																												<li>	
											<a href="http://adsabs.harvard.edu/abs/2008PLoSO...3.1717J">ADS</a></li>
																<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=18320038&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="http://dx.doi.org/10.1371/journal.pone.0001717">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1371/journal.pone.0001717&amp;rft_id=info:pmid/18320038&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Jupiter&amp;rft.aufirst=D. C.&amp;rft.jtitle=PLoS One&amp;rft.volume=3&amp;rft.spage=e1717&amp;rft.epage=&amp;rft.date=2008&amp;rft.atitle=A visual data mining tool that facilitates reconstruction of transcription regulatory networks&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1371/journal.pone.0001717&amp;id=pmid:18320038&amp;genre=article&amp;aulast=Jupiter&amp;aufirst=D. C.&amp;title=PLoS One&amp;volume=3&amp;spage=e1717&amp;epage=&amp;date=2008&amp;atitle=A visual data mining tool that facilitates reconstruction of transcription regulatory networks&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref16"><span class="vcard author"><span class="fn">Takahashi, K.</span></span> <i>et al.</i> <span class="title">Induction of pluripotent stem cells from adult human fibroblasts by defined factors</span>. <span class="source-title">Cell</span> <span class="volume">131</span>, <span class="start-page">861</span>–<span class="end-page">872</span> (<span class="year">2007</span>).
					<ul class="cleared">
																																				<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXhsVCntbbK&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=1a8d6696a45b6d9c01093bcf9056829f">CAS</a></li>
																				 																												<li>	
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000251800900010&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=18035408&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="http://dx.doi.org/10.1016/j.cell.2007.11.019">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1016/j.cell.2007.11.019&amp;rft_id=info:pmid/18035408&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Takahashi&amp;rft.aufirst=K.&amp;rft.jtitle=Cell&amp;rft.volume=131&amp;rft.spage=861&amp;rft.epage=872&amp;rft.date=2007&amp;rft.atitle=Induction of pluripotent stem cells from adult human fibroblasts by defined factors&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1016/j.cell.2007.11.019&amp;id=pmid:18035408&amp;genre=article&amp;aulast=Takahashi&amp;aufirst=K.&amp;title=Cell&amp;volume=131&amp;spage=861&amp;epage=872&amp;date=2007&amp;atitle=Induction of pluripotent stem cells from adult human fibroblasts by defined factors&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref17"><span class="vcard author"><span class="fn">Yu, J.</span></span> <i>et al.</i> <span class="title">Induced pluripotent stem cell lines derived from human somatic cells</span>. <span class="source-title">Science</span> <span class="volume">318</span>, <span class="start-page">1917</span>–<span class="end-page">1920</span> (<span class="year">2007</span>).
					<ul class="cleared">
																																				<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2sXhsVGjsLbN&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=f02e2761d838d7a359242184fe2bedb4">CAS</a></li>
																				 																												<li>	
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000251786600057&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=18029452&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="http://dx.doi.org/10.1126/science.1151526">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1126/science.1151526&amp;rft_id=info:pmid/18029452&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Yu&amp;rft.aufirst=J.&amp;rft.jtitle=Science&amp;rft.volume=318&amp;rft.spage=1917&amp;rft.epage=1920&amp;rft.date=2007&amp;rft.atitle=Induced pluripotent stem cell lines derived from human somatic cells&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1126/science.1151526&amp;id=pmid:18029452&amp;genre=article&amp;aulast=Yu&amp;aufirst=J.&amp;title=Science&amp;volume=318&amp;spage=1917&amp;epage=1920&amp;date=2007&amp;atitle=Induced pluripotent stem cell lines derived from human somatic cells&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref18"><span class="vcard author"><span class="fn">Kwong, L. N.</span></span> <i>et al.</i> <span class="title">Oncogenic NRAS signaling differentially regulates survival and proliferation in melanoma</span>. <span class="source-title">Nat Med</span> <span class="volume">18</span>, <span class="start-page">1503</span>–<span class="end-page">1510</span> (<span class="year">2012</span>).
					<ul class="cleared">
																																	<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BC38XhtlCqtbrI&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=7ca95179cbb5fd2b94d090ec0a6f53c1">CAS</a></li>
																				 																												<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=22983396&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="/doifinder/10.1038/nm.2941">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1038/nm.2941&amp;rft_id=info:pmid/22983396&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Kwong&amp;rft.aufirst=L. N.&amp;rft.jtitle=Nat Med&amp;rft.volume=18&amp;rft.spage=1503&amp;rft.epage=1510&amp;rft.date=2012&amp;rft.atitle=Oncogenic NRAS signaling differentially regulates survival and proliferation in melanoma&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1038/nm.2941&amp;id=pmid:22983396&amp;genre=article&amp;aulast=Kwong&amp;aufirst=L. N.&amp;title=Nat Med&amp;volume=18&amp;spage=1503&amp;epage=1510&amp;date=2012&amp;atitle=Oncogenic NRAS signaling differentially regulates survival and proliferation in melanoma&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref19"><span class="vcard author"><span class="fn">Wheeler, D. L.</span></span> <i>et al.</i> <span class="title">Database resources of the National Center for Biotechnology Information</span>. <span class="source-title">Nucleic Acids Res</span> <span class="volume">29</span>, <span class="start-page">11</span>–<span class="end-page">16</span> (<span class="year">2001</span>).
					<ul class="cleared">
																																							<li>	
																																																																																																																																																																										<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3MXjtlWns7c%3D&amp;pissn={printIssn}&amp;pyear=2012&amp;md5=1ffcb9d492d5f15cfb7030616dc665cb">CAS</a></li>
																				 																												<li>	
											<a href="http://adsabs.harvard.edu/abs/2001scgo.book.....W">ADS</a></li>
																<li>	
																																																																																																																																							<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?amp;GWVersion=2&amp;amp;SrcAuth=Nature&amp;amp;SrcApp=Nature&amp;amp;DestLinkType=FullRecord&amp;amp;KeyUT=000166360300002&amp;amp;DestApp=WOS_CPL">ISI</a></li>
																<li>	
																																																																																																														<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;amp;cmd=Retrieve&amp;amp;db=PubMed&amp;amp;list_uids=11125038&amp;amp;dopt=Abstract">PubMed</a></li>
																<li>	
											<a href="http://dx.doi.org/10.1093/nar/29.1.11">Article</a></li>
										</ul>
			<span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_id=info:doi/10.1093/nar/29.1.11&amp;rft_id=info:pmid/11125038&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.genre=article&amp;rft.aulast=Wheeler&amp;rft.aufirst=D. L.&amp;rft.jtitle=Nucleic Acids Res&amp;rft.volume=29&amp;rft.spage=11&amp;rft.epage=16&amp;rft.date=2001&amp;rft.atitle=Database resources of the National Center for Biotechnology Information&amp;rfr_id=info:sid/nature.com:Nature.com&amp;id=doi:10.1093/nar/29.1.11&amp;id=pmid:11125038&amp;genre=article&amp;aulast=Wheeler&amp;aufirst=D. L.&amp;title=Nucleic Acids Res&amp;volume=29&amp;spage=11&amp;epage=16&amp;date=2001&amp;atitle=Database resources of the National Center for Biotechnology Information&amp;sid=nature:Nature"></span>
				</li>
		
								
														
		<li id="ref20"><span class="vcard author"><span class="fn">Arvestad, L.</span></span> <cite class="book">RefSense</cite>, &#60;<a href="http://https://github.com/arvestad/RefSense">https://github.com/arvestad/RefSense</a>&#x0003E; (2012).
				</li>
		
								
	</ol>
<p class="download-refs"><a href="/srep/2012/121220/srep01011/ris/srep01011refs.ris">Download references</a></p>
	
							</div>
																					</div></section>

		
						
																								
												
									<section><div class="section   expanded" id="acknowledgments">

							<h1 class="section-heading toggle">Acknowledgements</h1>
						<div class="content">
																		<nav><ul class="section-nav cleared">

																																																						<li class="first"><a href="#introduction">Introduction</a></li>
																																													<li><a href="#results">Results</a></li>
																																													<li><a href="#discussion">Discussion</a></li>
																																													<li><a href="#methods">Methods</a></li>
																																													<li><a href="#references">References</a></li>
																																																					<li class="current"><span>Acknowledgements</span></li>	
																																													<li><a href="#author-information">Author information</a></li>
																																													<li><a href="#supplementary-information">Supplementary information</a></li>
																																													<li class="last"><a href="#comments">Comments</a></li>
																		
					</ul></nav>

								
								
				<p>This work was funded by a National Scientist Development Grant from the American Heart Association (AHA SDG 0630263N, PI: VV), and by start-up funds to VV from the Department of Systems Biology and Translational Medicine, and the Dean of the College of Medicine at Texas A&amp;M HSC College of Medicine.</p>
							</div>
																					</div></section>

		
						
																								
												
									<section><div class="section   expanded" id="author-information">

							<h1 class="section-heading toggle">Author information</h1>
						<div class="content">
																		<nav><ul class="section-nav cleared">

																																																						<li class="first"><a href="#introduction">Introduction</a></li>
																																													<li><a href="#results">Results</a></li>
																																													<li><a href="#discussion">Discussion</a></li>
																																													<li><a href="#methods">Methods</a></li>
																																													<li><a href="#references">References</a></li>
																																													<li><a href="#acknowledgments">Acknowledgements</a></li>
																																																					<li class="current"><span>Author information</span></li>	
																																													<li><a href="#supplementary-information">Supplementary information</a></li>
																																													<li class="last"><a href="#comments">Comments</a></li>
																		
					</ul></nav>

								
								
												
												
					



											
								
	<div id="author-affiliations">
				
		<h2 id="affil-auth">Affiliations</h2>
		<ol class="affiliations">
			
																<li id="a1">
						<h3>Texas A&amp;M HSC College of Medicine, Systems Biology and Translational Medicine</h3>
						 							
																							
																																																																		
																																																								
																					<ul class="affiliation-authors">
																	<li>Vincent VanBuren &amp;</li>
																	<li>Hailin Chen</li>
															</ul>
																		</li>
																																								
																																		</ol>
	</div>



	<div id="author-contributions">
		<h2 id="contrib-auth">Contributions</h2>
		<p>V.V conceived of, designed and implemented the Cognoscente knowledgebase, and wrote the manuscript. H.C identified documented interactions for genes with a role in heart development that were not included in the Interaction GeneRIF file, added those interactions to the knowledgebase, provided extensive testing of the knowledgebase, contributed to building Table 1 and provided useful comments on the manuscript.</p>
	</div>

 
	<div id="competing-financial-interests">
		<h2 id="cfi">Competing financial interests</h2>

								
												<p class="conflict">The authors declare no competing financial interests.</p>
													</div>

	<div id="corresponding-authors">
		<h2 id="corres-auth">Corresponding author</h2>
		<p class="correspondence">Correspondence to: </p>
		<ul class="correspondence">
					<li id="c1"><a href="/srep/foxtrot/svc/authoremailform?doi=10.1038/srep01011&amp;file=/srep/2012/121220/srep01011/full/srep01011.html&amp;title=Managing+biological+complexity+across+orthologs+with+a+visual+knowledgebase+of+documented+biomolecular+interactions&amp;author=Vincent+VanBuren" class="contact">Vincent VanBuren</a></li>
							</ul>
	</div>



	<div id="author-extra-details" >
		<h2>Author Details</h2>
		<ul><li id="auth-1" class="author-details cleared"><h3>Vincent VanBuren</h3><p class="email"><a href="/srep/foxtrot/svc/authoremailform?doi=10.1038/srep01011&amp;file=/srep/2012/121220/srep01011/full/srep01011.html&amp;title=Managing+biological+complexity+across+orthologs+with+a+visual+knowledgebase+of+documented+biomolecular+interactions&amp;author=Vincent+VanBuren" class="contact">Contact Vincent VanBuren</a></p><div class="author-search"><h4>Search for this author in:</h4><ul><li class="first"><a href="/search/executeSearch?sp-advanced=true&amp;sp-q-2=Vincent+VanBuren&amp;sp-p-2=all">NPG journals</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Vincent+VanBuren%5Bauthor%5D">PubMed</a></li><li><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Vincent+VanBuren&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en">Google Scholar</a></li></ul></div></li><li id="auth-2" class="author-details cleared"><h3>Hailin Chen</h3><div class="author-search"><h4>Search for this author in:</h4><ul><li class="first"><a href="/search/executeSearch?sp-advanced=true&amp;sp-q-2=Hailin+Chen&amp;sp-p-2=all">NPG journals</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Hailin+Chen%5Bauthor%5D">PubMed</a></li><li><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Hailin+Chen&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en">Google Scholar</a></li></ul></div></li></ul>
	</div>

	
							</div>
																					</div></section>

		
						
																								
												
									<section><div class="section   expanded" id="supplementary-information">

							<h1 class="section-heading toggle">Supplementary information</h1>
						<div class="content">
																		<nav><ul class="section-nav cleared">

																																																						<li class="first"><a href="#introduction">Introduction</a></li>
																																													<li><a href="#results">Results</a></li>
																																													<li><a href="#discussion">Discussion</a></li>
																																													<li><a href="#methods">Methods</a></li>
																																													<li><a href="#references">References</a></li>
																																													<li><a href="#acknowledgments">Acknowledgements</a></li>
																																													<li><a href="#author-information">Author information</a></li>
																																																					<li class="current"><span>Supplementary information</span></li>	
																																													<li class="last"><a href="#comments">Comments</a></li>
																		
					</ul></nav>

								
								
														
												
							
														

<div class="box supp-info">
	
	
					
					
						
					
		<h2 class="pdf">PDF files</h2>
		<ol>
					<li><dl><dt><a href="/srep/2012/121220/srep01011/extref/srep01011-s1.pdf">Supplementary Information (282K)</a></dt><dd><p>Supplementary Figures</p></dd></dl></li>
				</ol>
					
		<h2 class="excel">Excel files</h2>
		<ol>
					<li><dl><dt><a href="/srep/2012/121220/srep01011/extref/srep01011-s2.xls">Supplementary Information (57.0K)</a></dt><dd><p>Supplementary Table S1</p></dd></dl></li>
				</ol>
	</div>
	
							</div>
																				<footer><div class="article-footer">
					
					</div></footer>

										</div></section>

		
						
																								
												
									<section><div class="section   expanded" id="comments">

							<h1 class="section-heading toggle">Comments</h1>
						<div class="content">
																		<nav><ul class="section-nav cleared">

																																																						<li class="first"><a href="#introduction">Introduction</a></li>
																																													<li><a href="#results">Results</a></li>
																																													<li><a href="#discussion">Discussion</a></li>
																																													<li><a href="#methods">Methods</a></li>
																																													<li><a href="#references">References</a></li>
																																													<li><a href="#acknowledgments">Acknowledgements</a></li>
																																													<li><a href="#author-information">Author information</a></li>
																																													<li><a href="#supplementary-information">Supplementary information</a></li>
																																																					<li class="current last"><span>Comments</span></li>	
																		
					</ul></nav>

								
								
												
												
					


		










		



			<p>There are currently no comments.</p>
	
	<p class="comment-rss"><a href="/rss/feed?doi=10.1038/srep01011">Subscribe to comments</a></p>

		
				
			<h2>Add your comment</h2>
			<p>This is a public forum. Please keep to our <a href="/info/community-guidelines.html">Community Guidelines</a>. You can be controversial, but please don't get personal or offensive and do keep it brief. Remember our threads are for feedback and discussion - not for publishing papers, press releases or advertisements. If you find something abusive or inappropriate or which does not otherwise comply with our <a href="/info/tandc.html">Terms</a> or <a href="/info/community-guidelines.html">Community Guidelines</a>, please select the relevant 'Report this comment' link.</p>
	
			
				<div id="comment-help">
					<h3>How do I enter formatted text?</h3>
					<ul>
						<li><b>*Bold*</b> and <i>_italic_</i> are achieved like this.</li>
						<li><sub>~Subscript~</sub> and <sup>^superscript^</sup> are achieved like this.</li>
						<li>To hyperlink a word or phrase, place it in quotation marks followed by a colon and the URL e.g. <kbd>"text to link":http://example.com/url/to/link/to</kbd></li>
						<li>An unordered list can be made be prefixing each list item with an asterisk (*) and a space. An ordered list can be made be prefixing each list item with a hash symbol (#) and a space. Make sure you precede your list with a blank line.</li>
					</ul>
				</div>

				<form action="/post/comment" id="comment-form" method="post" class="cleared">
					<fieldset>
						<label for="comment-body" class="hidden">Comment body:</label>
						<textarea id="comment-body" name="message" class="box"></textarea>

						<input type="hidden" name="product" value="srep" /> 
						<input type="hidden" name="subject" value="" /> 
						<input type="hidden" name="recordId" value="2796558" /> 
						<input type="hidden" name="doi" value="10.1038/srep01011" />
						<input type="hidden" name="target" value="/srep/2012/121220/srep01011/full/srep01011.html" />
					
						<span class="comment-agreed">I have read and agreed to the <a href="/info/community-guidelines.html">Community Guidelines</a></span>

																					
												
					
<div class="captcha-box">
	<h2>Please confirm the words below</h2>
    <p>In order to reduce spamming, this process ensures you are a real person and not an automated program.</p>
	<label class="enter">Enter Words:</label>
	<div class="captcha-tool">
		
		<!--[if (IE 6)]>
			<iframe src="http://api.recaptcha.net/noscript?k=6LdH4r0SAAAAAJkZvUwkI7zPCKoguxJpcdYdGz2R"></iframe>
		    <textarea name="recaptcha_challenge_field" rows="3" cols="40"></textarea>
		    <input type="hidden" name="recaptcha_response_field" value="manual_challenge" />
		
		<![endif]-->
		
		<!--[if !(IE 6)]><!-->  
		<script type="text/javascript" src="http://api.recaptcha.net/challenge?k=6LdH4r0SAAAAAJkZvUwkI7zPCKoguxJpcdYdGz2R"></script>
		<noscript>
		    <iframe src="http://api.recaptcha.net/noscript?k=6LdH4r0SAAAAAJkZvUwkI7zPCKoguxJpcdYdGz2R"></iframe>
		    <textarea name="recaptcha_challenge_field" rows="3" cols="40"></textarea>
		    <input type="hidden" name="recaptcha_response_field" value="manual_challenge" />
		</noscript>
		<!--<![endif]-->				
	</div>
</div>												
						<input type="submit" id="comment-submit-button" name="submit" value="submit" class="submit" />
						<input type="submit" id="comment-preview-button" name="preview" value="preview" class="submit" /> 
					</fieldset> 
				</form>

				<h2>Preview of your comment</h2>
				<ol class="comments ugc" id="comment-preview">
					<li class="box cleared">
						<p class="datetime">Not yet posted</p>
						<dl>
							<dt><cite class="vcard"><span class="fn">Cameron Neylon</span></cite> said:</dt>
							<dd><p>Your comment&hellip;</p></dd>
						</dl>
					</li>
				</ol>
						
	
							</div>
																					</div></section>

		
				
					<footer><div class="article-footer">
							<p class="license">
					<a href="http://creativecommons.org/licenses/by/3.0/" class="cc-license by" rel="license"><img src="/view/images/icon_by.png"/></a>
					This work is licensed under a Creative Commons Attribution 3.0 Unported License. To view a copy of this license, visit  <a href="http://creativecommons.org/licenses/by/3.0/">http://creativecommons.org/licenses/by/3.0/</a>
				</p>
						</div></footer>

				
	
	
										
												
					<section>
<h1 class="hidden">Additional data</h1>
<div class="additional-data">

	

<script type="text/javascript" src="http://admin.brightcove.com/js/BrightcoveExperiences.js"></script>
	
	
	
</div>
</section>	
	</article>
</div>
<div id="extranav" role="complementary" class="col">
													
												
															




						
											
												
																						
												
							
 




	

								


						
 
<nav><div id="quick-nav" class="module cleared">
	<ul class="journal-nav">
					<li class="home"><a href="/srep/index.html">Journal home</a></li>
					<li class="archive"><a href="/search/executeSearch?sp-advanced=true&sp-m=0&siteCode=srep&sp-p=all&sp-q-9[SREP]=1&sp-p-2=all&sp-p-3=all&subject=&sp-s=date_descending&sp-c=25">Browse archive</a></li>
					<li class="authors"><a href="/srep/authors/index.html">For authors</a></li>
			</ul>
	<ul class="journal-notifications">
					<li class="submit"><a href="http://mts-srep.nature.com/cgi-bin/main.plex">Submit manuscript</a></li>
					<li class="e-alert"><a href="/srep/alerts/index.html ">E-alert sign up</a></li>
					<li class="rss"><a href="/srep/newsfeeds.html">RSS feed</a></li>
			</ul>
	
	</div></nav>
													
										
															
												
					

	
													
												
																		
												
						
					
		
							
	
																		
						

		
		
				
			
		
		
			
						
							
																											
																		
																										
						
	
	
	
	
		
		<div class="ad adunit-sidebar-1 
					3-1-rectangle
		">
		
					<script type="text/javascript" src="//ad.doubleclick.net/N285/adj/scientific_reports/article;pos=right;artid=srep01011;type=af;submeta=%2F631%2F114%2F129;submeta=%2F631%2F553%2F2703;submeta=%2F631%2F114%2F2408;submeta=%2F631%2F208%2F200;abr=!webtv;ptile=1;sz=300x100;ord=4592047608988775?"> </script>
			<noscript>
					<a href="//ad.doubleclick.net/N285/jump/scientific_reports/article;pos=right;artid=srep01011;type=af;submeta=%2F631%2F114%2F129;submeta=%2F631%2F553%2F2703;submeta=%2F631%2F114%2F2408;submeta=%2F631%2F208%2F200;abr=!webtv;ptile=1;sz=300x100;ord=4592047608988775?"><img src="//ad.doubleclick.net/N285/ad/scientific_reports/article;pos=right;artid=srep01011;type=af;submeta=%2F631%2F114%2F129;submeta=%2F631%2F553%2F2703;submeta=%2F631%2F114%2F2408;submeta=%2F631%2F208%2F200;abr=!webtv;ptile=1;sz=300x100;ord=4592047608988775?" alt="Advertisment" /></a>
					</noscript>
			</div>
														
										
															
										
															
												
					

														
										
															
												
					
													
												
																		
												
					


													
												
					

													
												
						
						
		
													
												
					    		
		    
				
																																										
		<div id="nature-jobs-box" class="module box no-padding">

			<h2 class="module-header">Science jobs from <span class="nature">nature</span><span class="hidden"> </span><span class="jobs">jobs</span></h2>

			<div class="box-padding">
				<ul class="jobs-list">
									<li>
						<h3><a href="http://www.nature.com/naturejobs/science/jobs/304811-Group-Leader-Computational-Biology-and-BioInformatics">Group Leader, Computational Biology and BioInformatics</a></h3>
						<ul>
							<li>Brigham and Women's Hospital & Dana-Farber Cancer Institute</li>
													</ul>
					</li>
									<li>
						<h3><a href="http://www.nature.com/naturejobs/science/jobs/298729-Tenure-Track-Faculty-Positions">Tenure Track Faculty Positions</a></h3>
						<ul>
							<li>Texas Tech University Health Sciences Center</li>
													</ul>
					</li>
									<li>
						<h3><a href="http://www.nature.com/naturejobs/science/jobs/294922-Faculty-position-Virology">Faculty position, Virology</a></h3>
						<ul>
							<li>University of Rochester School of Medicine and Dentistry</li>
													</ul>
					</li>
									<li>
						<h3><a href="http://www.nature.com/naturejobs/science/jobs/299576-Gastroenterologist">Gastroenterologist</a></h3>
						<ul>
							<li>University of Maryland and Baltimore Veterans Affairs Medical Center</li>
													</ul>
					</li>
									<li>
						<h3><a href="http://www.nature.com/naturejobs/science/jobs/305180-Non-Clinical-Lecturer-Senior-Lecturer-Reader-in-Virology">Non-Clinical Lecturer / Senior Lecturer / Reader in Virology</a></h3>
						<ul>
							<li>University College of London</li>
													</ul>
					</li>
								</ul>
			</div>
	
			<ul class="box-footer cleared">
				<li class="right-arrow post first"><a href="/naturejobs/science/jobs/new">Post a free job</a></li>
				<li class="right-arrow more last"><a href="/naturejobs/science">More science jobs</a></li>
			</ul>
		</div>
	
														
												
					


													
												
					
<div id="open-innovation-box" class="module box no-padding" role="secondary">
	<aside>		<div class="box-padding">
			<hgroup>
				<h2>Nature.com Open innovation Pavillion</h2>
			</hgroup>
										<ul class="challenge-list">
								   					<li>
						<h3><a href="/openinnovation?challengeId=9933318">  				    
						Propose a Biomarker that Could Predict Potential for Plexiform Neurofibroma Growth</a></h3>
						<img src="http://www.nature.com/openinnovation/challenge/images/9933318.jpg" alt="" />
						<dl class="cleared">
							<dt class="deadline">Deadline:</dt>
							<dd class="deadline">Feb 13 2013</dd>
							<dt class="reward">Reward:</dt>
							<dd class="reward">$10,000 USD</dd>
						</dl>
						<p><p> This Challenge seeks creative ways to identify which plexiform neurofibroma tumors will undergo significant growth in indivi&hellip;</p></p>
					</li>
													   					<li>
						<h3><a href="/openinnovation?challengeId=9933183">  				    
						Seeking Creative Use Cases for Thomson Reuters Web of Knowledge</a></h3>
						<img src="http://www.nature.com/openinnovation/challenge/images/9933183.jpg" alt="" />
						<dl class="cleared">
							<dt class="deadline">Deadline:</dt>
							<dd class="deadline">Feb 24 2013</dd>
							<dt class="reward">Reward:</dt>
							<dd class="reward">$10,000 USD</dd>
						</dl>
						<p><p> This Challenge seeks use cases for Thomson Reuters <em>Web of Knowledge&nbsp;</em>content, tools, and APIs (Application Programming Interf&hellip;</p></p>
					</li>
													</ul>
						
		</div>
	</aside>
	
	<ul class="box-footer cleared">
		<li class="powered first"><span>Powered by:</span><a href="http://www.innocentive.com"><img src="/view/images/innocentive_logo_small.png" alt="innocentive" class="innocentive-logo"/></a></li>
		<li class="right-arrow last"><a href="/openinnovation?moreChallenges=true">View all</a></li>
	</ul>
	
</div>




													
												
					
			



	


		



	


		



	


				
		<div class="module">
			<h2 class="tab-group-heading">Top content</h2>
			<div id="top-content" class="tab-group">
				
								 					
											<div class="box tab-box first active" id="top-content-emailed">
							<h3 class="tab">Emailed</h3>
							
							<div class="tab-content">
								<div class="box-padding">
									<ol class="article-list">

																												<li>
											<article>
												<div class="standard-teaser media-none">
													<h1><a href="http://www.nature.com/doifinder/10.1038/srep01174">The rich club phenomenon in the classroom</a></h1>
													<p class="publication"><span class="journal-title">Scientific Reports</span> <time datetime="2013-01-30" pubdate><span class="time">30 January 2013</span></time></p>
												</div>
											</article>
										</li>
																			<li>
											<article>
												<div class="standard-teaser media-none">
													<h1><a href="http://www.nature.com/doifinder/10.1038/srep01140">The natural ocean acidification and fertilization event caused by the submarine eruption of El Hierro</a></h1>
													<p class="publication"><span class="journal-title">Scientific Reports</span> <time datetime="2013-01-25" pubdate><span class="time">25 January 2013</span></time></p>
												</div>
											</article>
										</li>
																			<li>
											<article>
												<div class="standard-teaser media-none">
													<h1><a href="http://www.nature.com/doifinder/10.1038/srep01067">Effect of correlations on network controllability</a></h1>
													<p class="publication"><span class="journal-title">Scientific Reports</span> <time datetime="2013-01-15" pubdate><span class="time">15 January 2013</span></time></p>
												</div>
											</article>
										</li>
																			<li>
											<article>
												<div class="standard-teaser media-none">
													<h1><a href="http://www.nature.com/doifinder/10.1038/srep01199">Estimation of autistic children by metallomics analysis</a></h1>
													<p class="publication"><span class="journal-title">Scientific Reports</span> <time datetime="2013-02-04" pubdate><span class="time">04 February 2013</span></time></p>
												</div>
											</article>
										</li>
																			<li>
											<article>
												<div class="standard-teaser media-none">
													<h1><a href="http://www.nature.com/doifinder/10.1038/srep01098">Flexible packaging of solid-state integrated circuit chips with elastomeric microfluidics</a></h1>
													<p class="publication"><span class="journal-title">Scientific Reports</span> <time datetime="2013-01-22" pubdate><span class="time">22 January 2013</span></time></p>
												</div>
											</article>
										</li>
									
									</ol>
								</div>
								<p class="box-footer right-arrow"><a href="/srep/most.html?type=emailed">View all</a></p>
							</div>
						</div>	
									 					
											<div class="box tab-box" id="top-content-read">
							<h3 class="tab">Read</h3>
							
							<div class="tab-content">
								<div class="box-padding">
									<ol class="article-list">

																												<li>
											<article>
												<div class="standard-teaser media-none">
													<h1><a href="http://www.nature.com/doifinder/10.1038/srep01065">A serum &ldquo;sweet-doughnut&rdquo; protein facilitates fibrosis evaluation and therapy assessment in patients with viral hepatitis</a></h1>
													<p class="publication"><span class="journal-title">Scientific Reports</span> <time datetime="2013-01-15" pubdate><span class="time">15 January 2013</span></time></p>
												</div>
											</article>
										</li>
																			<li>
											<article>
												<div class="standard-teaser media-none">
													<h1><a href="http://www.nature.com/doifinder/10.1038/srep01073">Emergent dynamics of laboratory insect swarms</a></h1>
													<p class="publication"><span class="journal-title">Scientific Reports</span> <time datetime="2013-01-15" pubdate><span class="time">15 January 2013</span></time></p>
												</div>
											</article>
										</li>
																			<li>
											<article>
												<div class="standard-teaser media-none">
													<h1><a href="http://www.nature.com/doifinder/10.1038/srep01090">hDbr1 is a nucleocytoplasmic shuttling protein with a protein phosphatase-like motif essential for debranching activity</a></h1>
													<p class="publication"><span class="journal-title">Scientific Reports</span> <time datetime="2013-01-21" pubdate><span class="time">21 January 2013</span></time></p>
												</div>
											</article>
										</li>
																			<li>
											<article>
												<div class="standard-teaser media-none">
													<h1><a href="http://www.nature.com/doifinder/10.1038/srep01121">Equilibrious Strand Exchange Promoted by DNA Conformational Switching</a></h1>
													<p class="publication"><span class="journal-title">Scientific Reports</span> <time datetime="2013-01-24" pubdate><span class="time">24 January 2013</span></time></p>
												</div>
											</article>
										</li>
																			<li>
											<article>
												<div class="standard-teaser media-none">
													<h1><a href="http://www.nature.com/doifinder/10.1038/srep01120">Mass production of chemicals from biomass-derived oil by directly atmospheric distillation coupled with co-pyrolysis</a></h1>
													<p class="publication"><span class="journal-title">Scientific Reports</span> <time datetime="2013-01-24" pubdate><span class="time">24 January 2013</span></time></p>
												</div>
											</article>
										</li>
									
									</ol>
								</div>
								<p class="box-footer right-arrow"><a href="/srep/most.html?type=read">View all</a></p>
							</div>
						</div>	
												</div>
		</div>
						
					
	
							
	
																		
						

		
		
				
			
		
		
			
						
							
																											
																		
																										
						
	
	
	
	
		
		<div class="ad adunit-sidebar-2 
					medium-rectangle
		">
		
					<script type="text/javascript" src="//ad.doubleclick.net/N285/adj/scientific_reports/article;pos=right;artid=srep01011;type=af;submeta=%2F631%2F114%2F129;submeta=%2F631%2F553%2F2703;submeta=%2F631%2F114%2F2408;submeta=%2F631%2F208%2F200;abr=!webtv;ptile=2;sz=300x250;ord=4592047608988775?"> </script>
			<noscript>
					<a href="//ad.doubleclick.net/N285/jump/scientific_reports/article;pos=right;artid=srep01011;type=af;submeta=%2F631%2F114%2F129;submeta=%2F631%2F553%2F2703;submeta=%2F631%2F114%2F2408;submeta=%2F631%2F208%2F200;abr=!webtv;ptile=2;sz=300x250;ord=4592047608988775?"><img src="//ad.doubleclick.net/N285/ad/scientific_reports/article;pos=right;artid=srep01011;type=af;submeta=%2F631%2F114%2F129;submeta=%2F631%2F553%2F2703;submeta=%2F631%2F114%2F2408;submeta=%2F631%2F208%2F200;abr=!webtv;ptile=2;sz=300x250;ord=4592047608988775?" alt="Advertisment" /></a>
					</noscript>
			</div>
													
												
					
		
					
	<div id="nature-events-box" class="module box no-padding">

		<h2 class="module-header">Science events from <span class="nature">nature</span><span class="hidden"> </span><span class="events">events</span></h2>

				<div class="box-padding">
			<ul class="events-list">
									<li>
						<h3 class="title"><a href="http://www.nature.com/natureevents/science/events/18975-European_Calcified_Tissue_Society_50th_Anniversary_Congress">European Calcified Tissue Society 50th Anniversary Congress</a></h3>
						<p><strong><time datetime="2013-05-18">18 May 2013</time>
 &mdash; <time datetime="2013-05-21">21 May 2013</time>
</strong></p>
						<p>Lisbon, Portugal</p>
					</li>
							</ul>
		</div>
				
		<ul class="box-footer cleared">
			<li class="right-arrow post first"><a href="/natureevents/science/events/new">Post a free event</a></li>
			<li class="right-arrow more last"><a href="/natureevents/science">More science events</a></li>
		</ul>
	</div>
					
					
	
							
	
																		
						

		
		
				
			
		
		
			
						
							
																											
																		
																										
						
	
	
	
	
		
		<div class="ad adunit-sidebar-3 
					wide-skyscraper
		">
		
					<script type="text/javascript" src="//ad.doubleclick.net/N285/adj/scientific_reports/article;pos=right;artid=srep01011;type=af;submeta=%2F631%2F114%2F129;submeta=%2F631%2F553%2F2703;submeta=%2F631%2F114%2F2408;submeta=%2F631%2F208%2F200;abr=!webtv;ptile=3;sz=160x600;ord=4592047608988775?"> </script>
			<noscript>
					<a href="//ad.doubleclick.net/N285/jump/scientific_reports/article;pos=right;artid=srep01011;type=af;submeta=%2F631%2F114%2F129;submeta=%2F631%2F553%2F2703;submeta=%2F631%2F114%2F2408;submeta=%2F631%2F208%2F200;abr=!webtv;ptile=3;sz=160x600;ord=4592047608988775?"><img src="//ad.doubleclick.net/N285/ad/scientific_reports/article;pos=right;artid=srep01011;type=af;submeta=%2F631%2F114%2F129;submeta=%2F631%2F553%2F2703;submeta=%2F631%2F114%2F2408;submeta=%2F631%2F208%2F200;abr=!webtv;ptile=3;sz=160x600;ord=4592047608988775?" alt="Advertisment" /></a>
					</noscript>
			</div>
			
</div>

	
			</div>
			
			<div id="constrain-footer" class="cleared">
				<footer>												
												
					<div id="footer" class="constrain">
		<div class="inner">
		
		
					<dl class="clear ">
				<dt>Scientific Reports</dt>
												<dd class="issn"><abbr title="International Standard Serial Number">ISSN</abbr> (online)<span class="hidden">:</span> 2045-2322</dd>															</dl>
		
		<div class="footer-main">
			
			<ul class="footer-links first">
							<li>	
											<a href="/npg">About <abbr title="Nature Publishing Group">NPG</abbr></a></li>
							<li>	
											<a href="/npg_/contact">Contact <abbr title="Nature Publishing Group">NPG</abbr></a></li>
							<li>	
											<a href="/info/accessibility_statement.html">Accessibility statement</a></li>
							<li>	
											<a href="/help">Help</a></li>
						</ul>

			<ul class="footer-links">
							<li>	
											<a href="/info/privacy.html">Privacy policy</a></li>
							<li>	
											<a href="/info/cookies.html">Use of cookies</a></li>
							<li>	
											<a href="/info/legal_notice.html">Legal notice</a></li>
							<li>	
											<a href="/info/tandc.html">Terms</a></li>
						</ul>

			<ul class="footer-links last">
							<li>	
											<a href="/naturejobs">Nature<span class="hidden"> </span>jobs</a></li>
							<li>	
											<a href="http://www.natureasia.com">Nature Asia</a></li>
							<li>	
											<a href="/scitable">Nature Education</a></li>
							<li>	
											<a href="/webfeeds">RSS web feeds</a></li>
						</ul>

						
			<div id="mobile-css-test"></div>

											
												
					




			<form method="get" action="/search/executeSearch" id="footer-search" class="search-form" role="search">
			<label for="footer-keyword">Search:</label>
			<input type="hidden" name="sp-c" value="25"/>
			<input type="hidden" name="sp-s" value=""/>
															 									 						<input type="hidden" name="sp-q-1" value="SREP"/>
						 			 				
			<input type="search" id="footer-keyword" name="sp-q" value="" class="txt" maxlength="200"/>
			<button type="submit" id="footer-search-submit">Go</button>
					</form>
		
			<p class="copyright">
				<small><a href="/info/copyright_statement.html">&#169; 2012 Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.</a></small>
				<small>partner of <a href="/info/partners.html">AGORA, HINARI, OARE, INASP, ORCID, CrossRef and COUNTER</a></small>
			</p>
		</div>
	</div>
</div>

					</footer>

			</div>
		</div>
		
										
												
					


			
																																											
		
				<script src="/view/scripts/jquery/jquery-1.7.2.min.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/plugins/jquery.baseline.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/plugins/jquery.hitch.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/plugins/jquery.linkify.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/plugins/jquery.colorfade.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/plugins/jquery.scrollintoview.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/plugins/jquery.ellipsis.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/plugins/jquery.swapclasses.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/plugins/jquery.preview.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/plugins/jquery.grow.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/plugins/jquery.collision.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/plugins/jquery.initplaceholders.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.configuration.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.css.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.stringutils.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.cookie.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.broadcaster.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.pagemanager.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.popup.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.truncator.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.bitstate.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/icb/global.icb.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.slideshow.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.authorlists.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.tabgroup.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.popup-this.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.zoomer.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.simplecarousel.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.ads.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.mobile.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.draggablepopup.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.message.js" type="text/javascript"></script>
																																														
		
				<script src="/view/srep/scripts/site.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.deepdyve.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/article/global.article.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/article/global.highlighter.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/article/global.figures.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/article/global.articlesetup.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/article/global.init.js" type="text/javascript"></script>
																																														
		
				<script src="http://content.readcube.com/assets/readcube_purchase.js" type="text/javascript"></script>
																																														
		
				<script src="/view/scripts/global.recaptcha.js" type="text/javascript"></script>
																					
												
					

<!-- webtrends Version: 8.6.0 -->
<script src="/super/scripts/webtrends/webtrends.js" type="text/javascript"></script> 
<script type="text/javascript"><!--//--><![CDATA[//><!-- 
var _tag=new WebTrends(); 
_tag.dcsGetId(); 
//--><!]]></script> 
<script type="text/javascript"><!--//--><![CDATA[//><!--
_tag.WT.z_cg_type = _tag.getContentType(_tag.WT.cg_n);
_tag.dcsCollect(); 
//--><!]]></script>
<noscript>
	<div><img alt="" id="DCSIMG" width="1" height="1" src="//statse.webtrendslive.com/dcs0zztfg00000s969s37qoal_2f6z/njs.gif?dcsuri=/nojavascript&amp;WT.js=No&amp;WT.tv=8.6.0" /></div>
</noscript>
	<script type="text/javascript"><!--//--><![CDATA[//><!-- 
	(function(b){b(function(){var d=function(h){var g=[];for(var c in h){g.push(c);g.push(h[c])}_tag.dcsMultiTrack.apply(_tag,g)};var a=function(){var c={};c["DCS.dcsuri"]=this.pathname?this.pathname.indexOf("/")!==0?"/"+this.pathname:this.pathname:"/";c["DCS.dcsqry"]=this.search?this.search.substring(1):"";c["WT.ti"]="Download:"+(this.hostname?this.hostname.split(":")[0]:"")+c["DCS.dcsuri"]+(c["DCS.dcsqry"]?"?"+c["DCS.dcsqry"]:"");c["WT.dl"]="20";setTimeout(function(){d(c)},20)};b("a").each(function(){var c=this.href;if(c.indexOf(".pdf")!=-1&&!c.match(/\/journal\/v[0-9]+\/n[0-9]+\/pdf\//i)){b(this).bind("click",a)}else{if(c.indexOf("/rss")!=-1&&c.match(/\/rss\/?(\?.*)?$/)){b(this).bind("click",a)}}});b('div[class*="adunit-"] a').bind("click",function(){var c=b(this).parents('div[class*="adunit-"]:first').attr("class");if(c){var f=/(^|\s)adunit-([^\s]+)/.exec(c);_tag.dcsMultiTrack("WT.ac",(f)?f[2]:"")}})})})(jQuery);
	//--><!]]></script>
<!-- end webtrends Version: 8.6.0 -->																																																																																																			<script type="text/javascript"><!--//--><![CDATA[//><!-- 
	com.nature.Configuration.add({'siteName': 'srep', 'isLoggedIn': 'yes', 'mobileSettings': 'false', 'webtrendsEnabled': 'true', 'authorLimit': '19', 'articleLevel': '0', 'articleDoi': '10.1038/srep01011', 'nextDoi': '${configItem.value}', 'prevDoi': '${configItem.value}', 'typeText': 'Article', 'mopRegister': '/register/srep', 'mopSubscribe': 'https://secure.nature.com/subscribe/srep', 'mopLogin': '/foxtrot/svc/login', 'mopTitle': 'nature.com'});
	//--><!]]></script>
						
	</body>
</html>